<!DOCTYPE html>
<html lang="en" class="no-js firefox"><head><meta class="foundation-mq-small"><meta class="foundation-mq-small-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-large"><meta class="foundation-mq-large-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xxlarge"><meta class="foundation-data-attribute-namespace">
    <!-- charset must appear in the first 1024 bytes of the document -->
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
        <title>Leveraging large language models for literature-driven 
prioritization of protein binding pockets | Bioinformatics | Oxford 
Academic</title>
        
                


        <script async="" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/643701de45aa460012e1032e.js"></script><script async="" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/google-analytics_analytics.js"></script><script type="text/javascript" defer="defer" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/trendmd.min.js" data-trendmdconfig="{&quot;element&quot;:&quot;#trendmd-suggestions&quot;}"></script>




    <script src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/jquery.min.js" type="text/javascript"></script>
<script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/Themes/Silver/app/js/jquery.3.7.1.min.js" type="text/javascript">\x3C/script>')</script>
<script src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/jquery-migrate-1.4.1.min.js" type="text/javascript"></script>

    <script type="text/javascript" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/sharethis.js" async="async"></script>


    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=10">

    
    <meta http-equiv="X-UA-Compatible" content="IE=Edge">
    <!-- Turn off telephone number detection. -->
    <meta name="format-detection" content="telephone=no">

<!-- Bookmark Icons -->
  <link rel="apple-touch-icon" sizes="180x180" href="https://oup.silverchair-cdn.com/UI/app/img/v-638969377196181265/apple-touch-icon.png">
  <link rel="icon" type="image/png" href="https://oup.silverchair-cdn.com/UI/app/img/v-638969377197081303/favicon-32x32.png" sizes="32x32">
  <link rel="icon" type="image/png" href="https://oup.silverchair-cdn.com/UI/app/img/v-638969377196981285/favicon-16x16.png" sizes="16x16">
  <link rel="mask-icon" href="https://oup.silverchair-cdn.com/UI/app/img/v-638969377198281311/safari-pinned-tab.svg" color="#001C54">
  <link rel="icon" href="https://oup.silverchair-cdn.com/UI/app/img/v-638969377197381314/favicon.ico">
  <link rel="manifest" href="https://oup.silverchair-cdn.com/UI/app/img/v-638969377197681285/manifest.json">
  <meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-638969377196181265/browserconfig.xml">
  <meta name="theme-color" content="#002f65">


    




<link rel="stylesheet" type="text/css" href="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/icons.css">

<link rel="stylesheet" type="text/css" href="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/site.min.css">


    <link rel="stylesheet" href="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/css.css" as="style" onload="this.onload=null;this.rel='stylesheet'">


        <link href="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/global.css" rel="stylesheet" type="text/css">
            <link href="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/Site.css" rel="stylesheet" type="text/css">



<script> var dataLayer = [{"full_title":"Leveraging large language models for literature-driven prioritization of protein binding pockets","short_title":"Leveraging large language models for literature-driven prioritization of protein binding pockets","authors":"Roman Stratiichuk,Mykola Melnychenko,Ihor Koleiev,Taras Voitsitskyi,Vladyslav Husak,Nazar Shevchuk,Zakhar Ostrovsky,Volodymyr Bdzhola,Semen Yesylevskyy,Serhii Starosyla,Alan Nafiiev","issue_and_volume":"Volume 41 | Issue 8","type":"research-article","online_publication_date":"2025-08-07","access_type":"Open Access","license_type":"cc-by","event_type":"full-text","discipline_ot_level_1":"Science and Mathematics","discipline_ot_level_2":"Biological Sciences","supplier_tag":"SC_Journals","object_type":"Article","taxonomy":"taxId%3a39%7ctaxLabel%3aAcademicSubjects%7cnodeId%3aSCI01060%7cnodeLabel%3aBioinformatics+and+Computational+Biology%7cnodeLevel%3a3","siteid":"bioinformatics","authzrequired":"false","doi":"10.1093/bioinformatics/btaf449"}]; </script>
            <script>
                (function (w, d, s, l, i) {
                    w[l] = w[l] || []; w[l].push({
                        'gtm.start':
                            new Date().getTime(), event: 'gtm.js'
                    }); var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : '';
                        j.async = true; j.src =
                        'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
            </script>

    

        <script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: 0,
                isIndividualLoggedIn: 0
            };

            App.CurrentSubdomain = 'bioinformatics';
            App.SiteURL = 'academic.oup.com/bioinformatics';
        </script>
    
    
    <link href="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/chartist.min.css" onload="this.onload=null;this.removeAttribute('media');" rel="stylesheet" type="text/css">
    <script type="application/ld+json">
        {"@context":"https://schema.org","@type":"ScholarlyArticle","@id":"https://academic.oup.com/bioinformatics/article/41/8/btaf449/8225722","name":"Leveraging large language models for literature-driven prioritization of protein binding pockets","datePublished":"2025-08-07","isPartOf":{"@id":"https://academic.oup.com/bioinformatics/issue/41/8","@type":"PublicationIssue","issueNumber":"8","datePublished":"2025-08-02","isPartOf":{"@id":"https://academic.oup.com/bioinformatics/bioinformatics","@type":"Periodical","name":"Bioinformatics","issn":["1367-4811"]}},"url":"https://dx.doi.org/10.1093/bioinformatics/btaf449","inLanguage":"en","copyrightHolder":"Oxford University Press","copyrightYear":"2025","publisher":"Oxford University Press","author":[{"name":"Stratiichuk, Roman","affiliation":"Receptor.AI Inc. , London N1 7GU,  United Kingdom    Department of Biophysics and Medical Informatics, Educational and Scientific Centre “Іnstitute of Biology and Medicine”, Taras Shevchenko Kyiv National University , Kyiv 01601,  Ukraine","@type":"Person","sameAs":"https://orcid.org/0009-0001-6632-7642"},{"name":"Melnychenko, Mykola","affiliation":"Receptor.AI Inc. , London N1 7GU,  United Kingdom","@type":"Person","sameAs":"https://orcid.org/0009-0006-5671-9809"},{"name":"Koleiev, Ihor","affiliation":"Receptor.AI Inc. , London N1 7GU,  United Kingdom    Department of Physics of Biological Systems, Institute of Physics of The National Academy of Sciences of Ukraine , Kyiv 03038,  Ukraine","@type":"Person","sameAs":"https://orcid.org/0009-0001-4620-3175"},{"name":"Voitsitskyi, Taras","affiliation":"Receptor.AI Inc. , London N1 7GU,  United Kingdom    Department of Physics of Biological Systems, Institute of Physics of The National Academy of Sciences of Ukraine , Kyiv 03038,  Ukraine","@type":"Person","sameAs":"https://orcid.org/0000-0003-3127-3688"},{"name":"Husak, Vladyslav","affiliation":"Receptor.AI Inc. , London N1 7GU,  United Kingdom    Department of Cellular, Computational and Integrative Biology, The University of Trento , Povo, Trento 38123,  Italy","@type":"Person","sameAs":"https://orcid.org/0009-0000-7398-2330"},{"name":"Shevchuk, Nazar","affiliation":"Receptor.AI Inc. , London N1 7GU,  United Kingdom","@type":"Person","sameAs":"https://orcid.org/0009-0003-8474-9376"},{"name":"Ostrovsky, Zakhar","affiliation":"Receptor.AI Inc. , London N1 7GU,  United Kingdom","@type":"Person","sameAs":"https://orcid.org/0009-0003-4644-3587"},{"name":"Bdzhola, Volodymyr","affiliation":"Institute of Molecular Biology and Genetics of The National Academy of Sciences of Ukraine , Kyiv 03143,  Ukraine","@type":"Person","sameAs":"https://orcid.org/0000-0003-0315-450X"},{"name":"Yesylevskyy, Semen","affiliation":"Receptor.AI Inc. , London N1 7GU,  United Kingdom    Department of Physics of Biological Systems, Institute of Physics of The National Academy of Sciences of Ukraine , Kyiv 03038,  Ukraine    Institute of Organic Chemistry and Biochemistry, Czech Academy of Sciences , Prague 6 CZ-166 10,  Czech Republic    Department of Physical Chemistry, Faculty of Science, Palacký University Olomouc , Olomouc 771 46,  Czech Republic","@type":"Person","sameAs":"https://orcid.org/0000-0002-6748-8931"},{"name":"Starosyla, Serhii","affiliation":"Receptor.AI Inc. , London N1 7GU,  United Kingdom","@type":"Person","sameAs":"https://orcid.org/0000-0002-5103-0635"},{"name":"Nafiiev, Alan","affiliation":"Receptor.AI Inc. , London N1 7GU,  United Kingdom","@type":"Person","sameAs":"https://orcid.org/0009-0004-8604-377X"}],"description":"AbstractMotivation. Accurately identifying and prioritizing protein binding pockets is a foundational element of small-molecule drug discovery. Defining th","pageStart":"btaf449","siteName":"OUP Academic","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/m_btaf449f11.jpeg?Expires=1825961844&Signature=xDtH5yuuCvJe3SxZx~7sBw6td0Whbx76NquUY7a1zhJcIjLRFSczRAvM-Krn3jHBtPTdbQMyr01ypLo3pTx8qtQqsCwJEyha~mxb5WPOu8vwwR4wLE0g7P~W76RD0TkV8nxpOS0ob1ccSu6Qx0YAjj7rDU0F15j1gmoll-ML18xLEKKYXqpYecfBK0O7K6xswciG9NUvUgDbnOZJE2vnNr8lLxxKo3otCJ-s5T9taaiRPV0JJj2KCVOdnW1XJD4T2gBJ5L6fC6Yf3xX1x-Fzqznr9-~MJmAOuBN2~d3F3OAbQOYafRtNbZSuKQj7k5RS0wa435emwRPNW4UTcssugw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Leveraging large language models for literature-driven prioritization of protein binding pockets","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/m_btaf449f11.jpeg?Expires=1825961844&Signature=xDtH5yuuCvJe3SxZx~7sBw6td0Whbx76NquUY7a1zhJcIjLRFSczRAvM-Krn3jHBtPTdbQMyr01ypLo3pTx8qtQqsCwJEyha~mxb5WPOu8vwwR4wLE0g7P~W76RD0TkV8nxpOS0ob1ccSu6Qx0YAjj7rDU0F15j1gmoll-ML18xLEKKYXqpYecfBK0O7K6xswciG9NUvUgDbnOZJE2vnNr8lLxxKo3otCJ-s5T9taaiRPV0JJj2KCVOdnW1XJD4T2gBJ5L6fC6Yf3xX1x-Fzqznr9-~MJmAOuBN2~d3F3OAbQOYafRtNbZSuKQj7k5RS0wa435emwRPNW4UTcssugw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","image:alt":"featured image"}
    </script>
<meta property="og:site_name" content="OUP Academic">
<meta property="og:title" content="Leveraging large language models for literature-driven prioritization of protein binding pockets">
<meta property="og:description" content="AbstractMotivation. Accurately identifying and prioritizing protein binding pockets is a foundational element of small-molecule drug discovery. Defining th">
<meta property="og:type" content="article">
<meta property="og:url" content="https://dx.doi.org/10.1093/bioinformatics/btaf449">
<meta property="og:updated_time" content="">
<meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/m_btaf449f11.jpeg?Expires=1825961844&amp;Signature=xDtH5yuuCvJe3SxZx~7sBw6td0Whbx76NquUY7a1zhJcIjLRFSczRAvM-Krn3jHBtPTdbQMyr01ypLo3pTx8qtQqsCwJEyha~mxb5WPOu8vwwR4wLE0g7P~W76RD0TkV8nxpOS0ob1ccSu6Qx0YAjj7rDU0F15j1gmoll-ML18xLEKKYXqpYecfBK0O7K6xswciG9NUvUgDbnOZJE2vnNr8lLxxKo3otCJ-s5T9taaiRPV0JJj2KCVOdnW1XJD4T2gBJ5L6fC6Yf3xX1x-Fzqznr9-~MJmAOuBN2~d3F3OAbQOYafRtNbZSuKQj7k5RS0wa435emwRPNW4UTcssugw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
<meta property="og:image:url" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/m_btaf449f11.jpeg?Expires=1825961844&amp;Signature=xDtH5yuuCvJe3SxZx~7sBw6td0Whbx76NquUY7a1zhJcIjLRFSczRAvM-Krn3jHBtPTdbQMyr01ypLo3pTx8qtQqsCwJEyha~mxb5WPOu8vwwR4wLE0g7P~W76RD0TkV8nxpOS0ob1ccSu6Qx0YAjj7rDU0F15j1gmoll-ML18xLEKKYXqpYecfBK0O7K6xswciG9NUvUgDbnOZJE2vnNr8lLxxKo3otCJ-s5T9taaiRPV0JJj2KCVOdnW1XJD4T2gBJ5L6fC6Yf3xX1x-Fzqznr9-~MJmAOuBN2~d3F3OAbQOYafRtNbZSuKQj7k5RS0wa435emwRPNW4UTcssugw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
<meta property="og:image:secure_url" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/m_btaf449f11.jpeg?Expires=1825961844&amp;Signature=xDtH5yuuCvJe3SxZx~7sBw6td0Whbx76NquUY7a1zhJcIjLRFSczRAvM-Krn3jHBtPTdbQMyr01ypLo3pTx8qtQqsCwJEyha~mxb5WPOu8vwwR4wLE0g7P~W76RD0TkV8nxpOS0ob1ccSu6Qx0YAjj7rDU0F15j1gmoll-ML18xLEKKYXqpYecfBK0O7K6xswciG9NUvUgDbnOZJE2vnNr8lLxxKo3otCJ-s5T9taaiRPV0JJj2KCVOdnW1XJD4T2gBJ5L6fC6Yf3xX1x-Fzqznr9-~MJmAOuBN2~d3F3OAbQOYafRtNbZSuKQj7k5RS0wa435emwRPNW4UTcssugw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
<meta property="og:image:alt" content="featured image">
<meta name="twitter:card" content="summary_large_image">
<meta name="citation_author" content="Stratiichuk, Roman"><meta name="citation_author_institution" content="Receptor.AI Inc., London N1 7GU, United Kingdom"><meta name="citation_author_institution" content="Department of Biophysics and Medical Informatics, Educational and Scientific Centre “Іnstitute of Biology and Medicine”, Taras Shevchenko Kyiv National University, Kyiv 01601, Ukraine"><meta name="citation_author" content="Melnychenko, Mykola"><meta name="citation_author_institution" content="Receptor.AI Inc., London N1 7GU, United Kingdom"><meta name="citation_author" content="Koleiev, Ihor"><meta name="citation_author_institution" content="Receptor.AI Inc., London N1 7GU, United Kingdom"><meta name="citation_author_institution" content="Department of Physics of Biological Systems, Institute of Physics of The National Academy of Sciences of Ukraine, Kyiv 03038, Ukraine"><meta name="citation_author" content="Voitsitskyi, Taras"><meta name="citation_author_institution" content="Receptor.AI Inc., London N1 7GU, United Kingdom"><meta name="citation_author_institution" content="Department of Physics of Biological Systems, Institute of Physics of The National Academy of Sciences of Ukraine, Kyiv 03038, Ukraine"><meta name="citation_author" content="Husak, Vladyslav"><meta name="citation_author_institution" content="Receptor.AI Inc., London N1 7GU, United Kingdom"><meta name="citation_author_institution" content="Department of Cellular, Computational and Integrative Biology, The University of Trento, Povo, Trento 38123, Italy"><meta name="citation_author" content="Shevchuk, Nazar"><meta name="citation_author_institution" content="Receptor.AI Inc., London N1 7GU, United Kingdom"><meta name="citation_author" content="Ostrovsky, Zakhar"><meta name="citation_author_institution" content="Receptor.AI Inc., London N1 7GU, United Kingdom"><meta name="citation_author" content="Bdzhola, Volodymyr"><meta name="citation_author_institution" content="Institute of Molecular Biology and Genetics of The National Academy of Sciences of Ukraine, Kyiv 03143, Ukraine"><meta name="citation_author" content="Yesylevskyy, Semen"><meta name="citation_author_institution" content="Receptor.AI Inc., London N1 7GU, United Kingdom"><meta name="citation_author_institution" content="Department of Physics of Biological Systems, Institute of Physics of The National Academy of Sciences of Ukraine, Kyiv 03038, Ukraine"><meta name="citation_author_institution" content="Institute of Organic Chemistry and Biochemistry, Czech Academy of Sciences, Prague 6 CZ-166 10, Czech Republic"><meta name="citation_author_institution" content="Department of Physical Chemistry, Faculty of Science, Palacký University Olomouc, Olomouc 771 46, Czech Republic"><meta name="citation_author" content="Starosyla, Serhii"><meta name="citation_author_institution" content="Receptor.AI Inc., London N1 7GU, United Kingdom"><meta name="citation_author" content="Nafiiev, Alan"><meta name="citation_author_institution" content="Receptor.AI Inc., London N1 7GU, United Kingdom"><meta name="citation_title" content="Leveraging large language models for literature-driven prioritization of protein binding pockets"><meta name="citation_doi" content="10.1093/bioinformatics/btaf449"><meta name="citation_journal_title" content="Bioinformatics"><meta name="citation_journal_abbrev" content="Bioinformatics"><meta name="citation_volume" content="41"><meta name="citation_issue" content="8"><meta name="citation_publication_date" content="2025/08/02"><meta name="citation_publisher" content="Oxford Academic"><meta name="citation_reference" content="citation_title=DeepPocket: ligand binding site detection and segmentation using 3D convolutional neural networks; citation_author=Aggarwal  R; citation_author=Gupta  A; citation_author=Chelur  V;  citation_journal_title=J Chem Inf Model;  citation_year=2022;  citation_volume=62;  citation_pages=5069-79; "><meta name="citation_reference" content="citation_title=Structural basis of Nav1.7 inhibition by an isoform-selective small-molecule antagonist; citation_author=Ahuja  S; citation_author=Mukund  S; citation_author=Deng  L;  citation_journal_title=Science;  citation_year=2015;  citation_volume=350;  citation_pages=aac5464"><meta name="citation_reference" content="citation_title=In silico fragment-based discovery of CIB1-directed anti-tumor agents by FRASE-Bot; citation_author=An  Y; citation_author=Lim  J; citation_author=Glavatskikh  M;  citation_journal_title=Nat Commun;  citation_year=2024;  citation_volume=15;  citation_pages=5564"><meta name="citation_reference" content="citation_title=Predicting protein ligand binding sites by combining evolutionary sequence conservation and 3D structure; citation_author=Capra  JA; citation_author=Laskowski  RA; citation_author=Thornton  JM;  citation_journal_title=PLoS Comput Biol;  citation_year=2009;  citation_volume=5;  citation_pages=e1000585"><meta name="citation_reference" content="citation_title=POVME 2.0: an enhanced tool for determining pocket shape and volume characteristics; citation_author=Durrant  JD; citation_author=Votapka  L; citation_author=Sørensen  J;  citation_journal_title=J Chem Theory Comput;  citation_year=2014;  citation_volume=10;  citation_pages=5047-56; "><meta name="citation_reference" content="citation_title=Improving accuracy and efficiency of blind protein‐ligand docking by focusing on predicted binding sites; citation_author=Ghersi  D; citation_author=Sanchez  R.;  citation_journal_title=Proteins;  citation_year=2009;  citation_volume=74;  citation_pages=417-24; "><meta name="citation_reference" content="citation_title=Binding site detection remastered: enabling fast, robust, and reliable binding site detection and descriptor calculation with DoGSite3; citation_author=Graef  J; citation_author=Ehrt  C; citation_author=Rarey  M.;  citation_journal_title=J Chem Inf Model;  citation_year=2023;  citation_volume=63;  citation_pages=3128-37; "><meta name="citation_reference" content="citation_title=PUResNetV2.0: a deep learning model leveraging sparse representation for improved ligand binding site prediction; citation_author=Jeevan  K; citation_author=Palistha  S; citation_author=Tayara  H;  citation_journal_title=J Cheminform;  citation_year=2024;  citation_volume=16;  citation_pages=66"><meta name="citation_reference" content="citation_title=DeepSite: protein-Binding site predictor using 3D-convolutional neural networks; citation_author=Jiménez  J; citation_author=Doerr  S; citation_author=Martínez-Rosell  G;  citation_journal_title=Bioinformatics;  citation_year=2017;  citation_volume=33;  citation_pages=3036-42; "><meta name="citation_reference" content="citation_title=PUResNet: prediction of protein–ligand binding sites using deep residual neural network; citation_author=Kandel  J; citation_author=Tayara  H; citation_author=Chong  KT.;  citation_journal_title=J Cheminform;  citation_year=2021;  citation_volume=13;  citation_pages=65"><meta name="citation_reference" content="citation_title=Shared structural mechanisms of general anaesthetics and benzodiazepines; citation_author=Kim  JJ; citation_author=Gharpure  A; citation_author=Teng  J;  citation_journal_title=Nature;  citation_year=2020;  citation_volume=585;  citation_pages=303-8; "><meta name="citation_reference" content="citation_title=P2Rank: machine learning based tool for rapid and accurate prediction of ligand binding sites from protein structure; citation_author=Krivák  R; citation_author=Hoksza  D.;  citation_journal_title=J Cheminform;  citation_year=2018;  citation_volume=10;  citation_pages=39"><meta name="citation_reference" content="citation_title=Muscarinic acetylcholine receptors: novel opportunities for drug development; citation_author=Kruse  AC; citation_author=Kobilka  BK; citation_author=Gautam  D;  citation_journal_title=Nat Rev Drug Discov;  citation_year=2014;  citation_volume=13;  citation_pages=549-60; "><meta name="citation_reference" content="citation_title=Fpocket: an open source platform for ligand pocket detection; citation_author=Le Guilloux  V; citation_author=Schmidtke  P; citation_author=Tuffery  P.;  citation_journal_title=BMC Bioinformatics;  citation_year=2009;  citation_volume=10;  citation_pages=168"><meta name="citation_reference" content="citation_title=Anatomy of protein pockets and cavities: measurement of binding site geometry and implications for ligand design; citation_author=Liang  J; citation_author=Edelsbrunner  H; citation_author=Woodward  C.;  citation_journal_title=Protein Sci;  citation_year=1998;  citation_volume=7;  citation_pages=1884-97; "><meta name="citation_reference" content="citation_title=CB-Dock2: improved protein–ligand blind docking by integrating cavity detection, docking and homologous template fitting; citation_author=Liu  Y; citation_author=Yang  X; citation_author=Gan  J;  citation_journal_title=Nucleic Acids Res;  citation_year=2022;  citation_volume=50;  citation_pages=W159-64; "><meta name="citation_reference" content="citation_title=Insights into the evolution of divergent nucleotide-binding mechanisms among pseudokinases revealed by crystal structures of human and mouse MLKL; citation_author=Murphy  JM; citation_author=Lucet  IS; citation_author=Hildebrand  JM;  citation_journal_title=Biochem J;  citation_year=2014;  citation_volume=457;  citation_pages=369-77; "><meta name="citation_reference" content="citation_title=CASTp 3.0: computed atlas of surface topography of proteins; citation_author=Tian  W; citation_author=Chen  C; citation_author=Lei  X;  citation_journal_title=Nucleic Acids Res;  citation_year=2018;  citation_volume=46;  citation_pages=W363-7; "><meta name="citation_reference" content="citation_title=Q-BioLiP: a comprehensive resource for quaternary structure-based protein–ligand interactions; citation_author=Wei  H; citation_author=Wang  W; citation_author=Peng  Z;  citation_journal_title=Genomics Proteomics Bioinform;  citation_year=2024;  citation_volume=22;  citation_pages=qzae001"><meta name="citation_reference" content="Wei J, Wang X, Schuurmans D &nbsp; et al Chain-of-thought prompting elicits reasoning in large language models. arXiv, arXiv:2201.11903, 2023 , preprint: not peer reviewed. "><meta name="citation_reference" content="citation_title=Protein-Ligand binding residue prediction enhancement through hybrid deep heterogeneous learning of sequence and structure data; citation_author=Xia  C-Q; citation_author=Pan  X; citation_author=Shen  H-B.;  citation_journal_title=Bioinformatics;  citation_year=2020;  citation_volume=36;  citation_pages=3018-27; "><meta name="citation_reference" content="Yao S, Yu D, Zhao J &nbsp; et al Tree of thoughts: deliberate problem solving with large language models. arXiv, arXiv:2305.10601, 2023 , preprint: not peer reviewed. "><meta name="citation_reference" content="Yesylevskyy S. MolAR: memory‐safe library for analysis of MD simulations written in rust. J Comput Chem 2025; 46 :e27536. 10.1002/jcc.27536 "><meta name="citation_reference" content="citation_title=BioLiP2: an updated structure database for biologically relevant Ligand-Protein interactions; citation_author=Zhang  C; citation_author=Zhang  X; citation_author=Freddolino  L;  citation_journal_title=Nucleic Acids Res;  citation_year=2024;  citation_volume=52;  citation_pages=D404-12; "><meta name="citation_reference" content="citation_title=A point cloud graph neural network for protein–ligand binding site prediction; citation_author=Zhao  Y; citation_author=He  S; citation_author=Xing  Y;  citation_journal_title=Int J Mol Sci;  citation_year=2024;  citation_volume=25;  citation_pages=9280"><meta name="citation_fulltext_world_readable" content=""><meta name="citation_pdf_url" content="https://academic.oup.com/bioinformatics/article-pdf/41/8/btaf449/63978887/btaf449.pdf"><meta name="description" content="AbstractMotivation. Accurately identifying and prioritizing protein binding pockets is a foundational element of small-molecule drug discovery. Defining th"><meta name="citation_xml_url" content="https://academic.oup.com/bioinformatics/article-xml/doi/10.1093/bioinformatics/btaf449/8225722">


    <link rel="canonical" href="https://academic.oup.com/bioinformatics/article/41/8/btaf449/8225722">

    <meta name="citation_fulltext_world_readable">






    <meta name="publish_image" content="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/bioinformatics/bioinformatics_title-205396838.svg">



    <meta name="product_code" content="J_BIOINFORMATICS_1996_01_9999">
    <meta name="product_code" content="OPEN_ACCESS">
    <meta name="product_code" content="j_ALLJOURNALS">
    <meta name="product_code" content="J_8225722">
    <meta name="product_code" content="I_165440">





    



        <script>
            var SCM = SCM || {};
            SCM.pubGradeAdsEnabled = true;
            SCM.pubGradeJSLibrary = 'https://cdn.pbgrd.com/core-oup-new.js';
        </script>
    <script async="async" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/googletagservices_gpt.js"></script>
    <script>
        var googletag = googletag || {};
        googletag.cmd = googletag.cmd || [];
    </script>
        
            <script type="text/javascript">
            var gptAdSlots = [];
            googletag.cmd.push(function() {
                    
                    var mapping_ad1 = googletag.sizeMapping()
                        .addSize([1024, 0], [[970, 90], [728, 90]])
                        .addSize([768, 0], [728, 90])
                        .addSize([0, 0], [320, 50])
                        .build();
                    gptAdSlots["ad1"] = googletag.defineSlot('/116097782/bioinformatics_Supplement_Ad1',
                            [[970, 90], [728, 90], [320, 50]], 'adBlockHeader')
                        .defineSizeMapping(mapping_ad1)
                        .addService(googletag.pubads())
                        .setTargeting("ad_position", "ad1");
                    

                    
                    var mapping_ad2 = googletag.sizeMapping()
                        .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
                        .build();
                    gptAdSlots["ad2"] = googletag.defineSlot('/116097782/bioinformatics_Supplement_Ad2',
                            [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop')
                        .defineSizeMapping(mapping_ad2)
                        .addService(googletag.pubads())
                        .setTargeting("ad_position", "ad2");
                    

                    
                    var mapping_ad3 = googletag.sizeMapping()
                        .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
                        .build();
                    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/bioinformatics_Supplement_Ad3',
                        [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom')
                        .defineSizeMapping(mapping_ad3)
                        .addService(googletag.pubads())
                        .setTargeting("ad_position", "ad3");
                    

                    
                var mapping_ad4 = googletag.sizeMapping()
                        .addSize([0,0], [320, 50])
                        .addSize([768, 0], [728, 90])
                        .build();
                    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/bioinformatics_Supplement_Ad4',
                        [728, 90], 'adBlockFooter')
                        .defineSizeMapping(mapping_ad4)
                        .addService(googletag.pubads())
                        .setTargeting("ad_position", "ad4");
                    

                    
                var mapping_ad6 = googletag.sizeMapping()
                        .addSize([1024, 0], [[970, 90], [728, 90]])
                        .addSize([768, 0], [728, 90])
                        .addSize([0, 0], [320, 50])
                        .build();
                    gptAdSlots["ad6"] = googletag.defineSlot('/116097782/bioinformatics_Supplement_Ad6',
                        [[728, 90], [970, 90]], 'adBlockStickyFooter')
                        .defineSizeMapping(mapping_ad6)
                        .addService(googletag.pubads())
                        .setTargeting("ad_position", "ad6");
                    

                    
                    gptAdSlots["adInterstitial"] = googletag.defineOutOfPageSlot('/116097782/bioinformatics_Interstitial_Ad',
                        googletag.enums.OutOfPageFormat.INTERSTITIAL)
                        .addService(googletag.pubads())
                        .setTargeting("ad_position", "ad5");
                    

                googletag.pubads().addEventListener('slotRenderEnded', function (event) {
                    if (!event.isEmpty) {
                        $('.js-' + event.slot.getSlotElementId()).each(function () {
                            if ($(this).find('iframe').length) {
                                $(this).removeClass('hide');
                            }
                        });
                    }
                });

                googletag.pubads().addEventListener('impressionViewable', function (event) {
                    if (!event.isEmpty) {
                        $('.js-' + event.slot.getSlotElementId()).each(function () {
                            var $adblockDiv = $(this).find('.js-adblock');
                            var $adText = $(this).find('.js-adblock-advertisement-text');
                            if ($adblockDiv && $adblockDiv.is(':visible') && $adblockDiv.find('*').length > 1) {
                                $adText.removeClass('hide');
                                App.CenterAdBlock.Init($adblockDiv, $adText);
                            }
                            else {
                                $adText.addClass('hide');
                            }

                            //Initialize logic for Sticky Footer Ad
                            var $stickyFooterDiv = $(this).parents('.js-sticky-footer-ad');
                            if ($stickyFooterDiv && $stickyFooterDiv.is(':visible') && $stickyFooterDiv.find('*').length > 1) {
                                App.StickyFooterAd.Init();
                            }
                        });
                    }
                });

                        googletag.pubads().setTargeting("authentication",  "not-authenticated");
                        googletag.pubads().setTargeting("doi",  "10.1093/bioinformatics/btaf449");
                        googletag.pubads().setTargeting("url",  "bioinformatics/article/41/8/btaf449/8225722");
                        googletag.pubads().setTargeting("article_category",  );
                        googletag.pubads().setTargeting("article_type",  "research-article");
                        googletag.pubads().setTargeting("page_type",  "secondary");
                        googletag.pubads().setTargeting("jnlsdoi",  "10.1093/bioinformatics/btaf449");
                        googletag.pubads().setTargeting("jnlsurl",  "bioinformatics/article/41/8/btaf449/8225722");
                        googletag.pubads().setTargeting("jnlspage",  "article");


                googletag.pubads().enableSingleRequest();
                googletag.pubads().disableInitialLoad();
                googletag.pubads().collapseEmptyDivs();
                googletag.enableServices();
            });
            </script>
        
        <script id="altmetric-embed-js" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/altmetric_badges-10c8bb92f52500b3055c97099c9c67f45c4cef824fe9.js"></script><style></style><style type="text/css">.CtxtMenu_InfoClose {  top:.2em; right:.2em;}
.CtxtMenu_InfoContent {  overflow:auto; text-align:left; font-size:80%;  padding:.4em .6em; border:1px inset; margin:1em 0px;  max-height:20em; max-width:30em; background-color:#EEEEEE;  white-space:normal;}
.CtxtMenu_Info.CtxtMenu_MousePost {outline:none;}
.CtxtMenu_Info {  position:fixed; left:50%; width:auto; text-align:center;  border:3px outset; padding:1em 2em; background-color:#DDDDDD;  color:black;  cursor:default; font-family:message-box; font-size:120%;  font-style:normal; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 15px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius:15px;               /* Safari and Chrome */  -moz-border-radius:15px;                  /* Firefox */  -khtml-border-radius:15px;                /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */  filter:progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color="gray", Positive="true"); /* IE */}
</style><style type="text/css">.CtxtMenu_MenuClose {  position:absolute;  cursor:pointer;  display:inline-block;  border:2px solid #AAA;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  font-family: "Courier New", Courier;  font-size:24px;  color:#F0F0F0}
.CtxtMenu_MenuClose span {  display:block; background-color:#AAA; border:1.5px solid;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  line-height:0;  padding:8px 0 6px     /* may need to be browser-specific */}
.CtxtMenu_MenuClose:hover {  color:white!important;  border:2px solid #CCC!important}
.CtxtMenu_MenuClose:hover span {  background-color:#CCC!important}
.CtxtMenu_MenuClose:hover:focus {  outline:none}
</style><style type="text/css">.CtxtMenu_Menu {  position:absolute;  background-color:white;  color:black;  width:auto; padding:5px 0px;  border:1px solid #CCCCCC; margin:0; cursor:default;  font: menu; text-align:left; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 5px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius: 5px;             /* Safari and Chrome */  -moz-border-radius: 5px;                /* Firefox */  -khtml-border-radius: 5px;              /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */}
.CtxtMenu_MenuItem {  padding: 1px 2em;  background:transparent;}
.CtxtMenu_MenuArrow {  position:absolute; right:.5em; padding-top:.25em; color:#666666;  font-family: null; font-size: .75em}
.CtxtMenu_MenuActive .CtxtMenu_MenuArrow {color:white}
.CtxtMenu_MenuArrow.CtxtMenu_RTL {left:.5em; right:auto}
.CtxtMenu_MenuCheck {  position:absolute; left:.7em;  font-family: null}
.CtxtMenu_MenuCheck.CtxtMenu_RTL { right:.7em; left:auto }
.CtxtMenu_MenuRadioCheck {  position:absolute; left: .7em;}
.CtxtMenu_MenuRadioCheck.CtxtMenu_RTL {  right: .7em; left:auto}
.CtxtMenu_MenuInputBox {  padding-left: 1em; right:.5em; color:#666666;  font-family: null;}
.CtxtMenu_MenuInputBox.CtxtMenu_RTL {  left: .1em;}
.CtxtMenu_MenuComboBox {  left:.1em; padding-bottom:.5em;}
.CtxtMenu_MenuSlider {  left: .1em;}
.CtxtMenu_SliderValue {  position:absolute; right:.1em; padding-top:.25em; color:#333333;  font-size: .75em}
.CtxtMenu_SliderBar {  outline: none; background: #d3d3d3}
.CtxtMenu_MenuLabel {  padding: 1px 2em 3px 1.33em;  font-style:italic}
.CtxtMenu_MenuRule {  border-top: 1px solid #DDDDDD;  margin: 4px 3px;}
.CtxtMenu_MenuDisabled {  color:GrayText}
.CtxtMenu_MenuActive {  background-color: #606872;  color: white;}
.CtxtMenu_MenuDisabled:focus {  background-color: #E8E8E8}
.CtxtMenu_MenuLabel:focus {  background-color: #E8E8E8}
.CtxtMenu_ContextMenu:focus {  outline:none}
.CtxtMenu_ContextMenu .CtxtMenu_MenuItem:focus {  outline:none}
.CtxtMenu_SelectionMenu {  position:relative; float:left;  border-bottom: none; -webkit-box-shadow:none; -webkit-border-radius:0px; }
.CtxtMenu_SelectionItem {  padding-right: 1em;}
.CtxtMenu_Selection {  right: 40%; width:50%; }
.CtxtMenu_SelectionBox {  padding: 0em; max-height:20em; max-width: none;  background-color:#FFFFFF;}
.CtxtMenu_SelectionDivider {  clear: both; border-top: 2px solid #000000;}
.CtxtMenu_Menu .CtxtMenu_MenuClose {  top:-10px; left:-10px}
</style><style id="MJX-CHTML-styles">
mjx-container[jax="CHTML"] {
  line-height: 0;
}

mjx-container [space="1"] {
  margin-left: .111em;
}

mjx-container [space="2"] {
  margin-left: .167em;
}

mjx-container [space="3"] {
  margin-left: .222em;
}

mjx-container [space="4"] {
  margin-left: .278em;
}

mjx-container [space="5"] {
  margin-left: .333em;
}

mjx-container [rspace="1"] {
  margin-right: .111em;
}

mjx-container [rspace="2"] {
  margin-right: .167em;
}

mjx-container [rspace="3"] {
  margin-right: .222em;
}

mjx-container [rspace="4"] {
  margin-right: .278em;
}

mjx-container [rspace="5"] {
  margin-right: .333em;
}

mjx-container [size="s"] {
  font-size: 70.7%;
}

mjx-container [size="ss"] {
  font-size: 50%;
}

mjx-container [size="Tn"] {
  font-size: 60%;
}

mjx-container [size="sm"] {
  font-size: 85%;
}

mjx-container [size="lg"] {
  font-size: 120%;
}

mjx-container [size="Lg"] {
  font-size: 144%;
}

mjx-container [size="LG"] {
  font-size: 173%;
}

mjx-container [size="hg"] {
  font-size: 207%;
}

mjx-container [size="HG"] {
  font-size: 249%;
}

mjx-container [width="full"] {
  width: 100%;
}

mjx-box {
  display: inline-block;
}

mjx-block {
  display: block;
}

mjx-itable {
  display: inline-table;
}

mjx-row {
  display: table-row;
}

mjx-row > * {
  display: table-cell;
}

mjx-mtext {
  display: inline-block;
  text-align: left;
}

mjx-mstyle {
  display: inline-block;
}

mjx-merror {
  display: inline-block;
  color: red;
  background-color: yellow;
}

mjx-mphantom {
  visibility: hidden;
}

_::-webkit-full-page-media, _:future, :root mjx-container {
  will-change: opacity;
}

mjx-assistive-mml {
  position: absolute !important;
  top: 0px;
  left: 0px;
  clip: rect(1px, 1px, 1px, 1px);
  padding: 1px 0px 0px 0px !important;
  border: 0px !important;
  display: block !important;
  width: auto !important;
  overflow: hidden !important;
  -webkit-touch-callout: none;
  -webkit-user-select: none;
  -khtml-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

mjx-assistive-mml[display="block"] {
  width: 100% !important;
}

mjx-math {
  display: inline-block;
  text-align: left;
  line-height: 0;
  text-indent: 0;
  font-style: normal;
  font-weight: normal;
  font-size: 100%;
  font-size-adjust: none;
  letter-spacing: normal;
  border-collapse: collapse;
  word-wrap: normal;
  word-spacing: normal;
  white-space: nowrap;
  direction: ltr;
  padding: 1px 0;
}

mjx-container[jax="CHTML"][display="true"] {
  display: block;
  text-align: center;
  margin: 1em 0;
}

mjx-container[jax="CHTML"][display="true"][width="full"] {
  display: flex;
}

mjx-container[jax="CHTML"][display="true"] mjx-math {
  padding: 0;
}

mjx-container[jax="CHTML"][justify="left"] {
  text-align: left;
}

mjx-container[jax="CHTML"][justify="right"] {
  text-align: right;
}

mjx-c {
  display: inline-block;
}

mjx-utext {
  display: inline-block;
  padding: .75em 0 .2em 0;
}

mjx-mi {
  display: inline-block;
  text-align: left;
}

mjx-mo {
  display: inline-block;
  text-align: left;
}

mjx-stretchy-h {
  display: inline-table;
  width: 100%;
}

mjx-stretchy-h > * {
  display: table-cell;
  width: 0;
}

mjx-stretchy-h > * > mjx-c {
  display: inline-block;
  transform: scalex(1.0000001);
}

mjx-stretchy-h > * > mjx-c::before {
  display: inline-block;
  width: initial;
}

mjx-stretchy-h > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: clip visible;
  width: 100%;
}

mjx-stretchy-h > mjx-ext > mjx-c::before {
  transform: scalex(500);
}

mjx-stretchy-h > mjx-ext > mjx-c {
  width: 0;
}

mjx-stretchy-h > mjx-beg > mjx-c {
  margin-right: -.1em;
}

mjx-stretchy-h > mjx-end > mjx-c {
  margin-left: -.1em;
}

mjx-stretchy-v {
  display: inline-block;
}

mjx-stretchy-v > * {
  display: block;
}

mjx-stretchy-v > mjx-beg {
  height: 0;
}

mjx-stretchy-v > mjx-end > mjx-c {
  display: block;
}

mjx-stretchy-v > * > mjx-c {
  transform: scaley(1.0000001);
  transform-origin: left center;
  overflow: hidden;
}

mjx-stretchy-v > mjx-ext {
  display: block;
  height: 100%;
  box-sizing: border-box;
  border: 0px solid transparent;
  /* IE */ overflow: hidden;
  /* others */ overflow: visible clip;
}

mjx-stretchy-v > mjx-ext > mjx-c::before {
  width: initial;
  box-sizing: border-box;
}

mjx-stretchy-v > mjx-ext > mjx-c {
  transform: scaleY(500) translateY(.075em);
  overflow: visible;
}

mjx-mark {
  display: inline-block;
  height: 0px;
}

mjx-mrow {
  display: inline-block;
  text-align: left;
}

mjx-mfrac {
  display: inline-block;
  text-align: left;
}

mjx-frac {
  display: inline-block;
  vertical-align: 0.17em;
  padding: 0 .22em;
}

mjx-frac[type="d"] {
  vertical-align: .04em;
}

mjx-frac[delims] {
  padding: 0 .1em;
}

mjx-frac[atop] {
  padding: 0 .12em;
}

mjx-frac[atop][delims] {
  padding: 0;
}

mjx-dtable {
  display: inline-table;
  width: 100%;
}

mjx-dtable > * {
  font-size: 2000%;
}

mjx-dbox {
  display: block;
  font-size: 5%;
}

mjx-num {
  display: block;
  text-align: center;
}

mjx-den {
  display: block;
  text-align: center;
}

mjx-mfrac[bevelled] > mjx-num {
  display: inline-block;
}

mjx-mfrac[bevelled] > mjx-den {
  display: inline-block;
}

mjx-den[align="right"], mjx-num[align="right"] {
  text-align: right;
}

mjx-den[align="left"], mjx-num[align="left"] {
  text-align: left;
}

mjx-nstrut {
  display: inline-block;
  height: .054em;
  width: 0;
  vertical-align: -.054em;
}

mjx-nstrut[type="d"] {
  height: .217em;
  vertical-align: -.217em;
}

mjx-dstrut {
  display: inline-block;
  height: .505em;
  width: 0;
}

mjx-dstrut[type="d"] {
  height: .726em;
}

mjx-line {
  display: block;
  box-sizing: border-box;
  min-height: 1px;
  height: .06em;
  border-top: .06em solid;
  margin: .06em -.1em;
  overflow: hidden;
}

mjx-line[type="d"] {
  margin: .18em -.1em;
}

mjx-msub {
  display: inline-block;
  text-align: left;
}

mjx-mn {
  display: inline-block;
  text-align: left;
}

mjx-c::before {
  display: block;
  width: 0;
}

.MJX-TEX {
  font-family: MJXZERO, MJXTEX;
}

.TEX-B {
  font-family: MJXZERO, MJXTEX-B;
}

.TEX-I {
  font-family: MJXZERO, MJXTEX-I;
}

.TEX-MI {
  font-family: MJXZERO, MJXTEX-MI;
}

.TEX-BI {
  font-family: MJXZERO, MJXTEX-BI;
}

.TEX-S1 {
  font-family: MJXZERO, MJXTEX-S1;
}

.TEX-S2 {
  font-family: MJXZERO, MJXTEX-S2;
}

.TEX-S3 {
  font-family: MJXZERO, MJXTEX-S3;
}

.TEX-S4 {
  font-family: MJXZERO, MJXTEX-S4;
}

.TEX-A {
  font-family: MJXZERO, MJXTEX-A;
}

.TEX-C {
  font-family: MJXZERO, MJXTEX-C;
}

.TEX-CB {
  font-family: MJXZERO, MJXTEX-CB;
}

.TEX-FR {
  font-family: MJXZERO, MJXTEX-FR;
}

.TEX-FRB {
  font-family: MJXZERO, MJXTEX-FRB;
}

.TEX-SS {
  font-family: MJXZERO, MJXTEX-SS;
}

.TEX-SSB {
  font-family: MJXZERO, MJXTEX-SSB;
}

.TEX-SSI {
  font-family: MJXZERO, MJXTEX-SSI;
}

.TEX-SC {
  font-family: MJXZERO, MJXTEX-SC;
}

.TEX-T {
  font-family: MJXZERO, MJXTEX-T;
}

.TEX-V {
  font-family: MJXZERO, MJXTEX-V;
}

.TEX-VB {
  font-family: MJXZERO, MJXTEX-VB;
}

mjx-stretchy-v mjx-c, mjx-stretchy-h mjx-c {
  font-family: MJXZERO, MJXTEX-S1, MJXTEX-S4, MJXTEX, MJXTEX-A ! important;
}

@font-face /* 0 */ {
  font-family: MJXZERO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_Zero.woff") format("woff");
}

@font-face /* 1 */ {
  font-family: MJXTEX;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_Main-Regular.woff") format("woff");
}

@font-face /* 2 */ {
  font-family: MJXTEX-B;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_Main-Bold.woff") format("woff");
}

@font-face /* 3 */ {
  font-family: MJXTEX-I;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_Math-Italic.woff") format("woff");
}

@font-face /* 4 */ {
  font-family: MJXTEX-MI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_Main-Italic.woff") format("woff");
}

@font-face /* 5 */ {
  font-family: MJXTEX-BI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_Math-BoldItalic.woff") format("woff");
}

@font-face /* 6 */ {
  font-family: MJXTEX-S1;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_Size1-Regular.woff") format("woff");
}

@font-face /* 7 */ {
  font-family: MJXTEX-S2;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_Size2-Regular.woff") format("woff");
}

@font-face /* 8 */ {
  font-family: MJXTEX-S3;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_Size3-Regular.woff") format("woff");
}

@font-face /* 9 */ {
  font-family: MJXTEX-S4;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_Size4-Regular.woff") format("woff");
}

@font-face /* 10 */ {
  font-family: MJXTEX-A;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_AMS-Regular.woff") format("woff");
}

@font-face /* 11 */ {
  font-family: MJXTEX-C;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_Calligraphic-Regular.woff") format("woff");
}

@font-face /* 12 */ {
  font-family: MJXTEX-CB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_Calligraphic-Bold.woff") format("woff");
}

@font-face /* 13 */ {
  font-family: MJXTEX-FR;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_Fraktur-Regular.woff") format("woff");
}

@font-face /* 14 */ {
  font-family: MJXTEX-FRB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_Fraktur-Bold.woff") format("woff");
}

@font-face /* 15 */ {
  font-family: MJXTEX-SS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_SansSerif-Regular.woff") format("woff");
}

@font-face /* 16 */ {
  font-family: MJXTEX-SSB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_SansSerif-Bold.woff") format("woff");
}

@font-face /* 17 */ {
  font-family: MJXTEX-SSI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_SansSerif-Italic.woff") format("woff");
}

@font-face /* 18 */ {
  font-family: MJXTEX-SC;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_Script-Regular.woff") format("woff");
}

@font-face /* 19 */ {
  font-family: MJXTEX-T;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_Typewriter-Regular.woff") format("woff");
}

@font-face /* 20 */ {
  font-family: MJXTEX-V;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_Vector-Regular.woff") format("woff");
}

@font-face /* 21 */ {
  font-family: MJXTEX-VB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax@3/es5/output/chtml/fonts/woff-v2/MathJax_Vector-Bold.woff") format("woff");
}

mjx-c.mjx-c50::before {
  padding: 0.683em 0.681em 0 0;
  content: "P";
}

mjx-c.mjx-c6F::before {
  padding: 0.448em 0.5em 0.01em 0;
  content: "o";
}

mjx-c.mjx-c63::before {
  padding: 0.448em 0.444em 0.011em 0;
  content: "c";
}

mjx-c.mjx-c6B::before {
  padding: 0.694em 0.528em 0 0;
  content: "k";
}

mjx-c.mjx-c65::before {
  padding: 0.448em 0.444em 0.011em 0;
  content: "e";
}

mjx-c.mjx-c74::before {
  padding: 0.615em 0.389em 0.01em 0;
  content: "t";
}

mjx-c.mjx-cA0::before {
  padding: 0 0.25em 0 0;
  content: "\A0";
}

mjx-c.mjx-c52::before {
  padding: 0.683em 0.736em 0.022em 0;
  content: "R";
}

mjx-c.mjx-c61::before {
  padding: 0.448em 0.5em 0.011em 0;
  content: "a";
}

mjx-c.mjx-c6C::before {
  padding: 0.694em 0.278em 0 0;
  content: "l";
}

mjx-c.mjx-c3D::before {
  padding: 0.583em 0.778em 0.082em 0;
  content: "=";
}

mjx-c.mjx-c1D45B.TEX-I::before {
  padding: 0.442em 0.6em 0.011em 0;
  content: "n";
}

mjx-c.mjx-c6D::before {
  padding: 0.442em 0.833em 0 0;
  content: "m";
}

mjx-c.mjx-c68::before {
  padding: 0.694em 0.556em 0 0;
  content: "h";
}

mjx-c.mjx-c64::before {
  padding: 0.694em 0.556em 0.011em 0;
  content: "d";
}

mjx-c.mjx-c6E::before {
  padding: 0.442em 0.556em 0 0;
  content: "n";
}

mjx-c.mjx-c70::before {
  padding: 0.442em 0.556em 0.194em 0;
  content: "p";
}

mjx-c.mjx-c73::before {
  padding: 0.448em 0.394em 0.011em 0;
  content: "s";
}

mjx-c.mjx-c53::before {
  padding: 0.705em 0.556em 0.022em 0;
  content: "S";
}

mjx-c.mjx-c69::before {
  padding: 0.669em 0.278em 0 0;
  content: "i";
}

mjx-c.mjx-c66::before {
  padding: 0.705em 0.372em 0 0;
  content: "f";
}

mjx-c.mjx-c79::before {
  padding: 0.431em 0.528em 0.204em 0;
  content: "y";
}

mjx-c.mjx-c31::before {
  padding: 0.666em 0.5em 0 0;
  content: "1";
}

mjx-c.mjx-c2212::before {
  padding: 0.583em 0.778em 0.082em 0;
  content: "\2212";
}

mjx-c.mjx-c78::before {
  padding: 0.431em 0.528em 0 0;
  content: "x";
}

mjx-c.mjx-c72::before {
  padding: 0.442em 0.392em 0 0;
  content: "r";
}
</style><link rel="stylesheet" href="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/badge.css"></head><body data-sitestyletemplate="Journal" theme-bioinformatics="" class="off-canvas pg_Article pg_article" data-sitename="bioinformatics" style="cursor: default;"><input type="hidden" class="hfInterstitial" data-interstitiallinks="bioinformatics/issue,bioinformatics/advance-articles,bioinformatics/advance-article,bioinformatics/supplements,bioinformatics/article,bioinformatics/article-abstract,bioinformatics/pages" data-subdomain="bioinformatics">
    
    



    


    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&discipline_ot_level_1=Science and Mathematics&discipline_ot_level_2=Biological Sciences&supplier_tag=SC_Journals&object_type=Article&taxonomy=taxId%3a39%7ctaxLabel%3aAcademicSubjects%7cnodeId%3aSCI01060%7cnodeLabel%3aBioinformatics+and+Computational+Biology%7cnodeLevel%3a3&siteid=bioinformatics&authzrequired=false&doi=10.1093/bioinformatics/btaf449';
    </script>




    <script src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/f.txt" async=""></script>



            <noscript>
                <iframe  src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"
                        height="0" width="0" style="display:none;visibility:hidden"></iframe>
            </noscript>
            <a href="#skipNav" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" type="hidden" value="5139"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" type="hidden" value="7000"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" type="hidden" value="default"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" type="hidden" value="sticky">
    






<div class="master-container js-master-container">
<section class="master-header row js-master-header vt-site-page-header">
    <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        

    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div class="js-adBlock-parent-wrap adblock-parent-wrap">
    <div class="adBlockHeader-wrap js-adBlockHeader hide">
        <div id="adBlockHeader" class="js-adblock at-adblock" data-lazy-load-margin="150">
            <script>
                googletag.cmd.push(function () {
                    googletag.display('adBlockHeader');
                });
            </script>
        </div>
            <div class="advertisement-text at-adblock js-adblock-advertisement-text hide">Advertisement</div>
            </div>
</div> 
    </div>

    </div>

    <div class="oup-header sigma ">
        <div class="center-inner-row">

<div class="oup-header-logo">

        <a href="https://academic.oup.com/" data-google-interstitial="false">
            <img src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/oxford-academic-logo.svg" alt="Oxford Academic" class="oup-header-image at-oup-header-image ">
        </a>

</div>                <div class="widget widget-CustomNavLinks widget-instance-CustomNavLinksDeskTop">
            <div class="custom-nav-links-box">
            <div class="custom-nav-link">
                <a href="https://academic.oup.com/journals" data-google-interstitial="false">Journals</a>
            </div>
            <div class="custom-nav-link">
                <a href="https://academic.oup.com/books" data-google-interstitial="false">Books</a>
            </div>

    </div>
 
    </div>
            <ul class="oup-header-menu account-menu sigma-account-menu ">

                
                <li class="oup-header-menu-item mobile">
                    <a href="javascript:;" class="mobile-dropdown-toggle mobile-search-toggle" data-google-interstitial="false">
                        <i class="icon-menu_search"><span class="screenreader-text">Search Menu</span></i>
                    </a>
                </li>
                
        <li class="oup-header-menu-item mobile">
            <a href="javascript:;" class="mobile-dropdown-toggle navbar-ai-assistant-link js-navbar-ai-assistant-link" data-google-interstitial="false">
                <i class="icon-spark"><span class="screenreader-text">AI Discovery Assistant</span></i>
            </a>
        </li>        


                    <li class="oup-header-menu-item mobile info-icon-menu-item">
        <a href="https://academic.oup.com/pages/information" target="" class="at-info-button sigma-info-wrapper" data-google-interstitial="false">
            <img class="sigma-info-icon" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/i.svg" alt="Information">
        </a>
    </li>
    <li class="oup-header-menu-item mobile account-icon-menu-item">
        <a href="javascript:;" class="account-button js-account-button at-account-button " role="button" data-turnawayparams="journal%3dbioinformatics" aria-controls="login-box-panel" aria-expanded="false" data-google-interstitial="false">
            <img class="sigma-account-icon" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/account.svg" alt="Account">
        </a>
    </li>


                
                <li class="oup-header-menu-item mobile">
                    <a href="javascript:;" class="mobile-dropdown-toggle mobile-nav-toggle" aria-controls="main-nav-menu" aria-expanded="false" data-google-interstitial="false">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </a>
                </li>

                

                    <li class="oup-header-menu-item desktop info-icon-menu-item">
        <a href="https://academic.oup.com/pages/information" target="" class="at-info-button sigma-info-wrapper" data-google-interstitial="false">
            <img class="sigma-info-icon" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/i.svg" alt="Information">
        </a>
    </li>
    <li class="oup-header-menu-item desktop account-icon-menu-item">
        <a href="javascript:;" class="account-button js-account-button at-account-button sigma-logo-wrapper" role="button" data-turnawayparams="journal%3dbioinformatics" aria-controls="login-box-panel" aria-expanded="false" data-google-interstitial="false">
            <img class="sigma-account-icon" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/account.svg" alt="Account">
        </a>
    </li>
        <li class="oup-header-menu-item desktop account-icon-menu-item">
    <div class="widget widget-SeamlessAccess widget-instance-SitePageHeader">
            <a href="javascript:;" class="js-shibboleth-action seamless-access-button seamless-button-header button-call-to-action at-institutional-sign-in" rel="nofollow" data-action-type="login-discovery" data-entity-id="" data-google-interstitial="false">
        <span class="seamless-access-text">Sign in through your institution</span>
    </a>

 
    </div>

        </li>


                

            </ul>

            <div id="login-box-panel" class="login-box-placeholder js-login-box-placeholder hide">
                <div class="spinner"></div>
            </div>
        </div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">


<div class="navbar-search">
    <search aria-label="Site search">
        <div class="mobile-microsite-search">
                <label for="SitePageHeader-mobile-navbar-search-filter" class="screenreader-text js-mobile-navbar-search-filter-label">
                    Navbar Search Filter
                </label>
                <select class="mobile-navbar-search-filter js-mobile-navbar-search-filter at-navbar-search-filter" id="SitePageHeader-mobile-navbar-search-filter">

<option class="navbar-search-filter-option at-navbar-search-filter-option" value="" selected="selected">Bioinformatics</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Issue">This issue</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Parent">Bioinformatics Journals</option>
                        <optgroup class="navbar-search-optgroup" label="Search across Oxford Academic">
<option class="navbar-search-filter-option at-navbar-search-filter-option" value="AcademicSubjects/SCI01060">Bioinformatics and Computational Biology</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Books">Books</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Journals">Journals</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Umbrella">Oxford Academic</option>                        </optgroup>
                </select>

            <label for="SitePageHeader-mobile-microsite-search-term" class="screenreader-text js-mobile-microsite-search-term-label">
                Mobile Enter search term
            </label>
            <input class="mobile-search-input mobile-microsite-search-term js-mobile-microsite-search-term at-microsite-search-term " type="search" maxlength="255" placeholder="Search" id="SitePageHeader-mobile-microsite-search-term">

            <a href="javascript:;" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search" data-google-interstitial="false">
                <span class="screenreader-text">Search</span>
            </a>

        </div>
    </search>
</div>


        </div>
    </div>
   
    <div class="dropdown-panel mobile-nav-dropdown">


    <ul class="site-menu site-menu-lvl-0 at-site-menu" id="main-nav-menu">
        <li class="site-menu-item site-menu-lvl-0 at-site-menu-item">

                <a href="https://academic.oup.com/bioinformatics/issue" class="nav-link">
                    Issues
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0 at-site-menu-item">

                <a href="https://academic.oup.com/bioinformatics/advance-articles" class="nav-link">
                    Advance articles
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0 at-site-menu-item">

                <a href="javascript:;" class="nav-link js-nav-dropdown at-nav-dropdown" role="button" aria-expanded="false" aria-controls="submenu-1757657" data-google-interstitial="false">
                    Submit
                    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i>
                </a>
                <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                <ul class="site-menu site-menu-lvl-1 at-site-menu" id="submenu-1757657">
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="https://academic.oup.com/bioinformatics/pages/author-guidelines" class="nav-link">
                    Author Guidelines
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="http://mc.manuscriptcentral.com/bioinformatics" class="nav-link" data-google-interstitial="false">
                    Submission Site
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="https://academic.oup.com/bioinformatics/pages/open-access" class="nav-link">
                    Open Access
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="https://academic.oup.com/bioinformatics/pages/why-publish" class="nav-link">
                    Why publish with this journal?
                </a>
            
        </li>
            </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0 at-site-menu-item">

                <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link" data-google-interstitial="false">
                    Alerts
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0 at-site-menu-item">

                <a href="javascript:;" class="nav-link js-nav-dropdown at-nav-dropdown" role="button" aria-expanded="false" aria-controls="submenu-1757658" data-google-interstitial="false">
                    About
                    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i>
                </a>
                <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                <ul class="site-menu site-menu-lvl-1 at-site-menu" id="submenu-1757658">
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="https://academic.oup.com/bioinformatics/pages/About" class="nav-link">
                    About Bioinformatics
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?t730=&amp;t732=470053&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=" class="nav-link" data-google-interstitial="false">
                    Journals Career Network
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="https://academic.oup.com/bioinformatics/pages/Editorial_Board" class="nav-link">
                    Editorial Board
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="https://academic.oup.com/bioinformatics/pages/author-resource-centre" class="nav-link">
                    Author Resource Centre
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="https://academic.oup.com/advertising-and-corporate-services/pages/bioinformatics-media-kit" class="nav-link" data-google-interstitial="false">
                    Advertising and Corporate Services
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link" data-google-interstitial="false">
                    Self-Archiving Policy
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link" data-google-interstitial="false">
                    Dispatch Dates
                </a>
            
        </li>
            </ul>

        </li>
                <li class="site-menu-item site-menu-lvl-0 at-site-menu-item" id="site-menu-item-custom">
            <a href="https://academic.oup.com/journals" class="nav-link" data-google-interstitial="false">Journals on Oxford Academic</a>
        </li>
        <li class="site-menu-item site-menu-lvl-0 at-site-menu-item" id="site-menu-item-custom">
            <a href="https://academic.oup.com/books" class="nav-link" data-google-interstitial="false">Books on Oxford Academic</a>
        </li>
    </ul>
    </div>

</div>

    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <div class="site-parent-link-wrap">
                    <a href="https://academic.oup.com/bioinformaticsjournals" class="site-parent-link" data-google-interstitial="false">Bioinformatics Journals</a>
                </div>
                            <a href="https://academic.oup.com/bioinformatics" class="journal-logo-container">
                    <img id="logo-Bioinformatics" class="journal-logo" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/bioinformatics_title-205396838.svg" alt="Bioinformatics">
                </a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.iscb.org/" target="" class="society-logo-container" data-google-interstitial="false">
                                <img id="logo-InternationalSocietyforComputationalBiology" class="society-logo" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/bioinformatics_h11820233911.svg" alt="International Society for Computational Biology">
                            </a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

    <ul class="site-menu site-menu-lvl-0 at-site-menu" id="main-nav-menu">
        <li class="site-menu-item site-menu-lvl-0 at-site-menu-item">

                <a href="https://academic.oup.com/bioinformatics/issue" class="nav-link">
                    Issues
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0 at-site-menu-item">

                <a href="https://academic.oup.com/bioinformatics/advance-articles" class="nav-link">
                    Advance articles
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0 at-site-menu-item">

                <a href="javascript:;" class="nav-link js-nav-dropdown at-nav-dropdown" role="button" aria-expanded="false" aria-controls="submenu-1757657" data-google-interstitial="false">
                    Submit
                    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i>
                </a>
                <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                <ul class="site-menu site-menu-lvl-1 at-site-menu" id="submenu-1757657">
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="https://academic.oup.com/bioinformatics/pages/author-guidelines" class="nav-link">
                    Author Guidelines
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="http://mc.manuscriptcentral.com/bioinformatics" class="nav-link" data-google-interstitial="false">
                    Submission Site
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="https://academic.oup.com/bioinformatics/pages/open-access" class="nav-link">
                    Open Access
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="https://academic.oup.com/bioinformatics/pages/why-publish" class="nav-link">
                    Why publish with this journal?
                </a>
            
        </li>
            </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0 at-site-menu-item">

                <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link" data-google-interstitial="false">
                    Alerts
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0 at-site-menu-item">

                <a href="javascript:;" class="nav-link js-nav-dropdown at-nav-dropdown" role="button" aria-expanded="false" aria-controls="submenu-1757658" data-google-interstitial="false">
                    About
                    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i>
                </a>
                <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                <ul class="site-menu site-menu-lvl-1 at-site-menu" id="submenu-1757658">
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="https://academic.oup.com/bioinformatics/pages/About" class="nav-link">
                    About Bioinformatics
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?t730=&amp;t732=470053&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=" class="nav-link" data-google-interstitial="false">
                    Journals Career Network
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="https://academic.oup.com/bioinformatics/pages/Editorial_Board" class="nav-link">
                    Editorial Board
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="https://academic.oup.com/bioinformatics/pages/author-resource-centre" class="nav-link">
                    Author Resource Centre
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="https://academic.oup.com/advertising-and-corporate-services/pages/bioinformatics-media-kit" class="nav-link" data-google-interstitial="false">
                    Advertising and Corporate Services
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link" data-google-interstitial="false">
                    Self-Archiving Policy
                </a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1 at-site-menu-item">

                <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link" data-google-interstitial="false">
                    Dispatch Dates
                </a>
            
        </li>
            </ul>

        </li>
            </ul>
        </nav>
            <div class="navbar-right-container">
                <div class="navbar-search-container js-navbar-search-container">
                    <a href="javascript:;" class="navbar-search-close js_close-navsearch" data-google-interstitial="false">Close</a>


<div class="navbar-search">
    <search aria-label="Site search">
        <div class="microsite-search">
                <label for="SitePageHeader-navbar-search-filter" class="screenreader-text js-navbar-search-filter-label">
                    Navbar Search Filter
                </label>
                <select class="navbar-search-filter js-navbar-search-filter at-navbar-search-filter" id="SitePageHeader-navbar-search-filter">

<option class="navbar-search-filter-option at-navbar-search-filter-option" value="" selected="selected">Bioinformatics</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Issue">This issue</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Parent">Bioinformatics Journals</option>
                        <optgroup class="navbar-search-optgroup" label="Search across Oxford Academic">
<option class="navbar-search-filter-option at-navbar-search-filter-option" value="AcademicSubjects/SCI01060">Bioinformatics and Computational Biology</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Books">Books</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Journals">Journals</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Umbrella">Oxford Academic</option>                        </optgroup>
                </select>

            <label for="SitePageHeader-microsite-search-term" class="screenreader-text js-microsite-search-term-label">
                Enter search term
            </label>
            <input class="navbar-search-input microsite-search-term js-microsite-search-term at-microsite-search-term " type="search" maxlength="255" placeholder="Search" id="SitePageHeader-microsite-search-term">

            <a href="javascript:;" class="microsite-search-icon navbar-search-submit icon-menu_search" data-google-interstitial="false">
                <span class="screenreader-text">Search</span>
            </a>

        </div>
    </search>
</div>

<input id="hfCurrentBookSearch" name="hfCurrentBookSearch" type="hidden" value=""><input id="hfCurrentBookScope" name="hfCurrentBookScope" type="hidden" value="CurrentBook"><input id="hfBookSiteScope" name="hfBookSiteScope" type="hidden" value="Books"><input id="hfSeriesScope" name="hfSeriesScope" type="hidden" value="taxWithOr"><input id="hfParentSiteName" name="hfParentSiteName" type="hidden" value="Bioinformatics Journals"><input id="hfParentSiteUrl" name="hfParentSiteUrl" type="hidden" value="academic.oup.com/bioinformaticsjournals"><input id="hfSiteID" name="hfSiteID" type="hidden" value="5139"><input id="hfParentSiteID" name="hfParentSiteID" type="hidden" value="6390"><input id="hfJournalSiteScope" name="hfJournalSiteScope" type="hidden" value="Journals"><input id="hfParentSiteScope" name="hfParentSiteScope" type="hidden" value="Parent"><input id="hfDefaultSearchURL" name="hfDefaultSearchURL" type="hidden" value="search-results?page=1&amp;q="><input id="hfIssueSearch" name="hfIssueSearch" type="hidden" value="&amp;fl_IssueID=165440"><input id="hfIssueSiteScope" name="hfIssueSiteScope" type="hidden" value="Issue"><input id="hfUmbrellaScope" name="hfUmbrellaScope" type="hidden" value="Umbrella"><input id="hfUmbrellaSiteUrl" name="hfUmbrellaSiteUrl" type="hidden" value="academic.oup.com"><input id="hfUmbrellaSiteId" name="hfUmbrellaSiteId" type="hidden" value="191"><input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q="><input id="hfTaggedCollectionScope" name="hfTaggedCollectionScope" type="hidden" value=""><input id="hfApplyLatestEditionSearchFacet" name="hfApplyLatestEditionSearchFacet" type="hidden" value="False"><input id="hfIsLatestBookEdition" name="hfIsLatestBookEdition" type="hidden" value="False">
                    <div class="navbar-search-advanced"><a href="https://academic.oup.com/bioinformatics/advanced-search?page=1&amp;fl_SiteID=5139&amp;SearchSourceType=1" class="advanced-search js-advanced-search" data-google-interstitial="false">Advanced Search</a></div>
                </div>
                
                <div class="navbar-search-collapsed"><a href="javascript:;" class="icon-menu_search js_expand-navsearch" data-google-interstitial="false"><span class="screenreader-text">Search Menu</span></a></div>

        <div class="navbar-menu navbar-ai-assistant-container site-menu-item">
            <a href="javascript:;" class="navbar-ai-assistant-link js-navbar-ai-assistant-link" data-google-interstitial="false">
                <span aria-label="AI Discovery Assistant" class="navbar-ai-assistant-span">
                    <i class="icon-spark navbar-icon-spark"></i>
                    AI Discovery Assistant
                </span>
            </a>
                <div class="ai-onboarding-discovery-tooltip hidden js-navbar-ai-assistant-tooltip" data-discoverymode="discovery">
                    <div class="tooltiptext">
                        Discover the most relevant content quickly with our AI Discovery Assistant
                    </div>
                    <i class="icon-spark navbar-icon-spark"></i>
                </div>
        </div>

            </div>
            </div>
</div>
    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="">
    <input type="hidden" id="hfSolrJournalName" value="">
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100">
    <input type="hidden" id="hfJournalShortName" value="">
    <input type="hidden" id="hfSearchPlaceholder" value="">
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="">
    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/bioinformatics">
    <input type="hidden" name="RedirectSiteUrl" id="RedirectSiteUrl" value="httpszazjzjacademiczwoupzwcom">
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="bioinformatics"> 
    </div>

</section>
        <div class="widget widget-SitewideBanner widget-instance-" style="">
        

 
    </div>


    <div id="main" class="content-main js-main ui-base">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    <div id="skipNav" tabindex="-1"></div>
                    



<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col stuck" data-client-solrsearch-event-registered="true">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav" style="top: 55px;">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <a class="article-issue-link" href="https://academic.oup.com/bioinformatics/issue/41/8">
        <div class="article-issue-img">
                <img id="issueImage" class="fb-featured-image" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/m_bioinfo_41_8cover.jpeg" alt="Bioinformatics Cover Image for Volume 41, Issue 8">
        </div>
        <div class="article-issue-info">
            <div class="volume-issue__wrap">

                        <div class="volume trailing-comma">Volume 41</div>
                        <div class="issue">Issue 8</div>



            </div>
            <div class="ii-pub-date">
August 2025            </div>
            
        </div>
    </a>
</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<h3 class="contents-title">Article Contents</h3>
<ul class="jumplink-list js-jumplink-list">
        <li class="section-jump-link head-1" link-destination="529947872">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#529947872">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="529947875">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#529947875">1 Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="529947888">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo active" href="#529947888">2 Materials and methods</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="529947932">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#529947932">3 Results</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="529947953">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#529947953">4 Discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="529947965">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#529947965">5 Conclusion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="529947967">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#529947967">Author contributions</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="529947969">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#529947969">Supplementary data</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="529947972">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#529947972">Funding</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="529947974">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#529947974">Data availability</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="529947976">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#529947976">References</a>
            </div>
        </li>
<li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li></ul>
 
    </div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="https://academic.oup.com/bioinformatics/article/41/8/btaf419/8211672">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="https://academic.oup.com/bioinformatics/article/41/8/btaf409/8205670">Next &gt;</a>
        </li>
</ul> 
    </div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar stuck"><div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
    <li class="toolbar-item item-pdf js-item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="8225722" href="https://academic.oup.com/bioinformatics/article-pdf/41/8/btaf449/63978887/btaf449.pdf">
            <img src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view at-split-view" target="" data-google-interstitial="false">
                                    <i class="icon-menu-split"></i>
                                    Split View
                                </a>
                            </li>

                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="at-views-dropdown drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true" data-google-interstitial="false">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown js-dropdown-content" data-dropdown-content="" aria-label="submenu">
                                <div class="arrow-up" style="display: block;"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content" style="display: none;">
                                    <a href="javascript:;" data-google-interstitial="false"><span>Article contents</span></a>
                                </li>
                                <li class="at-figures-tables article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;" data-google-interstitial="false"><span>Figures &amp; tables</span></a>
                                </li>
                                
                                
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;" data-google-interstitial="false"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
                        </li>


                        <li class="toolbar-item item-cite js-item-cite">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        <a href="#" class="js-cite-button at-CiteButton" data-reveal-id="getCitation" data-reveal="">
    <i class="icon-read-more"></i>
    <span>Cite</span>
</a>

<div id="getCitation" class="reveal-modal js-citation-modal" data-reveal="">
    <h3 class="modal-title">Cite</h3>
        <div class="oxford-citation-text">
            <p>Roman Stratiichuk, Mykola Melnychenko, Ihor Koleiev, 
Taras Voitsitskyi, Vladyslav Husak, Nazar Shevchuk, Zakhar Ostrovsky, 
Volodymyr Bdzhola, Semen Yesylevskyy, Serhii Starosyla, Alan Nafiiev,  
Leveraging large language models for literature-driven prioritization of
 protein binding pockets, <em>Bioinformatics</em>, Volume 41, Issue 8, August 2025, btaf449, <a href="https://doi.org/10.1093/bioinformatics/btaf449" data-google-interstitial="false">https://doi.org/10.1093/bioinformatics/btaf449</a></p>
        </div>

    <div class="citation-download-wrap">
        <form action="/Citation/Download" method="get" id="citationModal">
            <input type="hidden" name="resourceId" value="8225722">
            <input type="hidden" name="resourceType" value="3">
            <label for="sticky-selectFormat" class="hide js-citation-format-label">Select Format</label>
            <select required="" name="citationFormat" class="citation-download-format js-citation-format" id="sticky-select-format">
                <option selected="selected" disabled="disabled">Select format</option>
                        <option value="0">.ris (Mendeley, Papers, Zotero)</option>
                        <option value="1">.enw (EndNote)</option>
                        <option value="2">.bibtex (BibTex)</option>
                        <option value="3">.txt (Medlars, RefWorks)</option>

            </select>
            <button class="btn citation-download-link disabled" type="submit">Download citation</button>
        </form>
    </div>

    <a class="close-reveal-modal" href="javascript:;" data-google-interstitial="false"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-OUP_Get_Permissions">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1367-4811&amp;title=Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets&amp;publicationDate=2025-08-07&amp;volumeNum=41&amp;issueNum=8&amp;author=Stratiichuk%2C%20Roman%3B%20Melnychenko%2C%20Mykola&amp;startPage=1&amp;endPage=11&amp;contentId=10.1093%2Fbioinformatics%2Fbtaf449&amp;oa=CC%20BY&amp;copyright=%C2%A9%20The%20Author%28s%29%202025.%20Published%20by%20Oxford%20University%20Press.&amp;orderBeanReset=True" id="PermissionsLink" class="" target="" rel="nofollow" data-google-interstitial="false">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>

    <li class="toolbar-item item-with-dropdown item-share">
        <a href="javascript:;" class="drop-trigger js-toolbar-dropdown at-ShareButton" data-dropdown="ShareDrop" data-google-interstitial="false">
            <i class="icon-menu_share"><span class="screenreader-text">Share Icon</span></i>
            <span class="toolbar-label">
                <span class="toolbar-text">Share</span>
                <i class="arrow-icon icon-general-arrow-filled-down js-toolbar-arrow-icon"></i>
            </span>
        </a>
        <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown js-dropdown-content" data-dropdown-content="">

    <li>
        <a class="st-custom-button addthis_button_bluesky js-share-link" data-network="bluesky" data-title="Leveraging large language models for literature-driven prioritization of protein binding pockets" data-url=" https://academic.oup.com/bioinformatics/article/41/8/btaf449/8225722" data-email-subject="Leveraging large language models for literature-driven prioritization of protein binding pockets" href="javascript:;" rel="nofollow" data-google-interstitial="false"><span>Bluesky</span></a>
    </li>
    <li>
        <a class="st-custom-button addthis_button_facebook js-share-link" data-network="facebook" data-title="Leveraging large language models for literature-driven prioritization of protein binding pockets" data-url="https://academic.oup.com/bioinformatics/article/41/8/btaf449/8225722" data-email-subject="Leveraging large language models for literature-driven prioritization of protein binding pockets" href="javascript:;" rel="nofollow" data-google-interstitial="false"><span>Facebook</span></a>
    </li>
    <li>
        <a class="st-custom-button addthis_button_twitter js-share-link" data-network="twitter" data-title="Leveraging large language models for literature-driven prioritization of protein binding pockets" data-url="https://academic.oup.com/bioinformatics/article/41/8/btaf449/8225722" data-email-subject="Leveraging large language models for literature-driven prioritization of protein binding pockets" href="javascript:;" rel="nofollow" data-google-interstitial="false"><span>X</span></a>
    </li>
    <li>
        <a class="st-custom-button addthis_button_linkedin js-share-link" data-network="linkedin" data-title="Leveraging large language models for literature-driven prioritization of protein binding pockets" data-url="https://academic.oup.com/bioinformatics/article/41/8/btaf449/8225722" data-email-subject="Leveraging large language models for literature-driven prioritization of protein binding pockets" href="javascript:;" rel="nofollow" data-google-interstitial="false"><span>LinkedIn</span></a>
    </li>
    <li>
        <a class="st-custom-button addthis_button_email js-share-link" data-network="email" data-title="Leveraging large language models for literature-driven prioritization of protein binding pockets" data-url="https://academic.oup.com/bioinformatics/article/41/8/btaf449/8225722" data-email-subject="Leveraging large language models for literature-driven prioritization of protein binding pockets" href="javascript:;" rel="nofollow" data-google-interstitial="false"><span>Email</span></a>
    </li>



        </ul>
    </li>


                    </ul>
                    <div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        



<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">


<div class="navbar-search">
    <search aria-label="Site search">
        
    </search>
</div>


        </div>
    </div>
   
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>


<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">
        </nav>
            <div class="navbar-right-container">
                <div class="navbar-search-container js-navbar-search-container">
                    <a href="javascript:;" class="navbar-search-close js_close-navsearch" data-google-interstitial="false">Close</a>


<div class="navbar-search">
    <search aria-label="Site search">
        <div class="microsite-search">
                <label for="sticky-navbar-search-filter" class="screenreader-text js-navbar-search-filter-label">
                    Navbar Search Filter
                </label>
                <select class="navbar-search-filter js-navbar-search-filter at-navbar-search-filter" id="sticky-navbar-search-filter">

<option class="navbar-search-filter-option at-navbar-search-filter-option" value="" selected="selected">Bioinformatics</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Issue">This issue</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Parent">Bioinformatics Journals</option>
                        <optgroup class="navbar-search-optgroup" label="Search across Oxford Academic">
<option class="navbar-search-filter-option at-navbar-search-filter-option" value="AcademicSubjects/SCI01060">Bioinformatics and Computational Biology</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Books">Books</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Journals">Journals</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Umbrella">Oxford Academic</option>                        </optgroup>
                </select>

            <label for="sticky-microsite-search-term" class="screenreader-text js-microsite-search-term-label">
                Enter search term
            </label>
            <input class="navbar-search-input microsite-search-term js-microsite-search-term at-microsite-search-term " type="search" maxlength="255" placeholder="Search" id="sticky-microsite-search-term">

            <a href="javascript:;" class="microsite-search-icon navbar-search-submit icon-menu_search" data-google-interstitial="false">
                <span class="screenreader-text">Search</span>
            </a>

        </div>
    </search>
</div>


                    <div class="navbar-search-advanced"><a href="https://academic.oup.com/bioinformatics/advanced-search?page=1&amp;fl_SiteID=5139&amp;SearchSourceType=1" class="advanced-search js-advanced-search" data-google-interstitial="false">Advanced Search</a></div>
                </div>
                
                <div class="navbar-search-collapsed"><a href="javascript:;" class="icon-menu_search js_expand-navsearch" data-google-interstitial="false"><span class="screenreader-text">Search Menu</span></a></div>

        <div class="navbar-menu navbar-ai-assistant-container site-menu-item">
            <a href="javascript:;" class="navbar-ai-assistant-link js-navbar-ai-assistant-link" data-google-interstitial="false">
                <span aria-label="AI Discovery Assistant" class="navbar-ai-assistant-span">
                    <i class="icon-spark navbar-icon-spark"></i>
                    AI Discovery Assistant
                </span>
            </a>
                <div class="ai-onboarding-discovery-tooltip hidden js-navbar-ai-assistant-tooltip" data-discoverymode="discovery">
                    <div class="tooltiptext">
                        Discover the most relevant content quickly with our AI Discovery Assistant
                    </div>
                    <i class="icon-spark navbar-icon-spark"></i>
                </div>
        </div>

            </div>
            </div>
</div>
<input id="routename" name="RouteName" type="hidden" value="bioinformatics"> 
    </div>

                    </div>
                </div>
            </div></div>
<div id="ContentColumn" class="page-column page-column--center">

    <div class="article-browse-top article-browse-mobile-nav js-mobile-nav" style="">
    <div class="article-browse-mobile-nav-inner js-mobile-nav-inner">
        <button class="toggle-left-col toggle-left-col__article btn-as-link">
            Article Navigation
        </button>
    </div>
</div>
<div class="article-browse-top article-browse-mobile-nav mobile-sticky-toolbar js-mobile-nav-sticky">
    <div class="article-browse-mobile-nav-inner">
        <button class="toggle-left-col toggle-left-col__article btn-as-link">
            Article Navigation
        </button>
    </div>
</div>

    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    <span class="journal-info__format-label">Journal Article</span>
                </div>


    <div class="widget-items">
                <div class="title-wrap">
                    <h1 class="wi-article-title article-title-main accessible-content-title at-articleTitle">
                        Leveraging large language models for literature-driven prioritization of protein binding pockets
<i class="icon-availability_open" title="Open Access"><span class="screenreader-text">Open Access</span></i>                    </h1>

                </div>
                <div class="wi-authors at-ArticleAuthors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name js-flyout-wrap">







                                            <a href="javascript:;" class="linked-name js-linked-name-trigger btn-as-link" aria-controls="author-flyout-44877958" aria-expanded="false" data-google-interstitial="false">Roman Stratiichuk</a><span class="delimiter">, </span>
                                        
                                        <span id="author-flyout-44877958" class="al-author-info-wrap arrow-up">
<div class="info-card-author authorInfo_OUP_ArticleTop_Info_Widget">
    <div class="name-role-wrap">
        <div class="info-card-name">
    Roman Stratiichuk
</div>

            <div class="info-card-role">
            Data curation, Formal analysis, Validation
            </div>
    </div>

    <div class="info-card-affilitation">
        <div class="aff"><div class="institution">Receptor.AI Inc.</div>, London N1 7GU, <div class="country">United Kingdom</div></div><div class="aff"><div class="institution">Department
 of Biophysics and Medical Informatics, Educational and Scientific 
Centre “Іnstitute of Biology and Medicine”, Taras Shevchenko Kyiv 
National University</div>, Kyiv 01601, <div class="country">Ukraine</div></div>
    </div>

    <div class="info-card-location">
        <a id="contrib-orcid-0009-0001-6632-7642" href="https://orcid.org/0009-0001-6632-7642" data-google-interstitial="false">
            <img class="orchid-icon" alt="ORCID logo" aria-hidden="true" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/mini-icon.png">&nbsp;&nbsp;https://orcid.org/0009-0001-6632-7642
        </a>
    </div>
    <div class="info-card-search-label">
        Search for other works by this author on:
    </div>

<div class="info-card-search info-card-search-internal">
    <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Roman+Stratiichuk" rel="nofollow" data-google-interstitial="false">Oxford Academic</a>
</div>

    <div class="info-card-search info-card-search-pubmed">
        <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Stratiichuk%20R" data-google-interstitial="false">PubMed</a>
    </div>
    <div class="info-card-search info-card-search-google">
        <a href="http://scholar.google.com/scholar?q=author:%22Stratiichuk%20Roman%22" data-google-interstitial="false">Google Scholar</a>
    </div>
</div>                                        </span>
                                    </span>
                                    <span class="al-author-name js-flyout-wrap">







                                            <a href="javascript:;" class="linked-name js-linked-name-trigger btn-as-link" aria-controls="author-flyout-44877959" aria-expanded="false" data-google-interstitial="false">Mykola Melnychenko<i class="icon-general-mail"></i></a><span class="delimiter">, </span>
                                        
                                        <span id="author-flyout-44877959" class="al-author-info-wrap arrow-up">
<div class="info-card-author authorInfo_OUP_ArticleTop_Info_Widget">
    <div class="name-role-wrap">
        <div class="info-card-name">
    Mykola Melnychenko
            <span class="info-card-footnote"><span class="xrefLink" id="jumplink-btaf449-cor1"></span><a href="javascript:;" reveal-id="btaf449-cor1" data-open="btaf449-cor1" class="link link-ref link-reveal xref-default" data-google-interstitial="false"><!----></a></span>
</div>

            <div class="info-card-role">
            Data curation, Formal analysis, Investigation, Software, Writing - original draft
            </div>
    </div>

    <div class="info-card-affilitation">
        <div class="aff"><div class="institution">Receptor.AI Inc.</div>, London N1 7GU, <div class="country">United Kingdom</div></div>
    </div>

    <div class="info-author-correspondence">
        <div content-id="btaf449-cor1">Corresponding author. Receptor.AI Inc., 20-22 Wenlock Road, London N1 7GU, United Kingdom. E-mail: <a href="mailto:mykola.melnychenko@receptor.ai" target="" data-google-interstitial="false">mykola.melnychenko@receptor.ai</a>.</div>
    </div>
    <div class="info-card-location">
        <a id="contrib-orcid-0009-0006-5671-9809" href="https://orcid.org/0009-0006-5671-9809" data-google-interstitial="false">
            <img class="orchid-icon" alt="ORCID logo" aria-hidden="true" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/mini-icon.png">&nbsp;&nbsp;https://orcid.org/0009-0006-5671-9809
        </a>
    </div>
    <div class="info-card-search-label">
        Search for other works by this author on:
    </div>

<div class="info-card-search info-card-search-internal">
    <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Mykola+Melnychenko" rel="nofollow" data-google-interstitial="false">Oxford Academic</a>
</div>

    <div class="info-card-search info-card-search-pubmed">
        <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Melnychenko%20M" data-google-interstitial="false">PubMed</a>
    </div>
    <div class="info-card-search info-card-search-google">
        <a href="http://scholar.google.com/scholar?q=author:%22Melnychenko%20Mykola%22" data-google-interstitial="false">Google Scholar</a>
    </div>
</div>                                        </span>
                                    </span>
                                    <span class="al-author-name js-flyout-wrap">







                                            <a href="javascript:;" class="linked-name js-linked-name-trigger btn-as-link" aria-controls="author-flyout-44877960" aria-expanded="false" data-google-interstitial="false">Ihor Koleiev</a><span class="delimiter">, </span>
                                        
                                        <span id="author-flyout-44877960" class="al-author-info-wrap arrow-up">
<div class="info-card-author authorInfo_OUP_ArticleTop_Info_Widget">
    <div class="name-role-wrap">
        <div class="info-card-name">
    Ihor Koleiev
</div>

            <div class="info-card-role">
            Data curation, Validation
            </div>
    </div>

    <div class="info-card-affilitation">
        <div class="aff"><div class="institution">Receptor.AI Inc.</div>, London N1 7GU, <div class="country">United Kingdom</div></div><div class="aff"><div class="institution">Department of Physics of Biological Systems, Institute of Physics of The National Academy of Sciences of Ukraine</div>, Kyiv 03038, <div class="country">Ukraine</div></div>
    </div>

    <div class="info-card-location">
        <a id="contrib-orcid-0009-0001-4620-3175" href="https://orcid.org/0009-0001-4620-3175" data-google-interstitial="false">
            <img class="orchid-icon" alt="ORCID logo" aria-hidden="true" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/mini-icon.png">&nbsp;&nbsp;https://orcid.org/0009-0001-4620-3175
        </a>
    </div>
    <div class="info-card-search-label">
        Search for other works by this author on:
    </div>

<div class="info-card-search info-card-search-internal">
    <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Ihor+Koleiev" rel="nofollow" data-google-interstitial="false">Oxford Academic</a>
</div>

    <div class="info-card-search info-card-search-pubmed">
        <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Koleiev%20I" data-google-interstitial="false">PubMed</a>
    </div>
    <div class="info-card-search info-card-search-google">
        <a href="http://scholar.google.com/scholar?q=author:%22Koleiev%20Ihor%22" data-google-interstitial="false">Google Scholar</a>
    </div>
</div>                                        </span>
                                    </span>
                                    <span class="al-author-name js-flyout-wrap">







                                            <a href="javascript:;" class="linked-name js-linked-name-trigger btn-as-link" aria-controls="author-flyout-44877961" aria-expanded="false" data-google-interstitial="false">Taras Voitsitskyi</a><span class="delimiter">, </span>
                                        
                                        <span id="author-flyout-44877961" class="al-author-info-wrap arrow-up">
<div class="info-card-author authorInfo_OUP_ArticleTop_Info_Widget">
    <div class="name-role-wrap">
        <div class="info-card-name">
    Taras Voitsitskyi
</div>

            <div class="info-card-role">
            Data curation, Validation
            </div>
    </div>

    <div class="info-card-affilitation">
        <div class="aff"><div class="institution">Receptor.AI Inc.</div>, London N1 7GU, <div class="country">United Kingdom</div></div><div class="aff"><div class="institution">Department of Physics of Biological Systems, Institute of Physics of The National Academy of Sciences of Ukraine</div>, Kyiv 03038, <div class="country">Ukraine</div></div>
    </div>

    <div class="info-card-location">
        <a id="contrib-orcid-0000-0003-3127-3688" href="https://orcid.org/0000-0003-3127-3688" data-google-interstitial="false">
            <img class="orchid-icon" alt="ORCID logo" aria-hidden="true" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/mini-icon.png">&nbsp;&nbsp;https://orcid.org/0000-0003-3127-3688
        </a>
    </div>
    <div class="info-card-search-label">
        Search for other works by this author on:
    </div>

<div class="info-card-search info-card-search-internal">
    <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Taras+Voitsitskyi" rel="nofollow" data-google-interstitial="false">Oxford Academic</a>
</div>

    <div class="info-card-search info-card-search-pubmed">
        <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Voitsitskyi%20T" data-google-interstitial="false">PubMed</a>
    </div>
    <div class="info-card-search info-card-search-google">
        <a href="http://scholar.google.com/scholar?q=author:%22Voitsitskyi%20Taras%22" data-google-interstitial="false">Google Scholar</a>
    </div>
</div>                                        </span>
                                    </span>
                                    <span class="al-author-name js-flyout-wrap">







                                            <a href="javascript:;" class="linked-name js-linked-name-trigger btn-as-link" aria-controls="author-flyout-44877962" aria-expanded="false" data-google-interstitial="false">Vladyslav Husak</a><span class="delimiter">, </span>
                                        
                                        <span id="author-flyout-44877962" class="al-author-info-wrap arrow-up">
<div class="info-card-author authorInfo_OUP_ArticleTop_Info_Widget">
    <div class="name-role-wrap">
        <div class="info-card-name">
    Vladyslav Husak
</div>

            <div class="info-card-role">
            Data curation, Validation
            </div>
    </div>

    <div class="info-card-affilitation">
        <div class="aff"><div class="institution">Receptor.AI Inc.</div>, London N1 7GU, <div class="country">United Kingdom</div></div><div class="aff"><div class="institution">Department of Cellular, Computational and Integrative Biology, The University of Trento</div>, Povo, Trento 38123, <div class="country">Italy</div></div>
    </div>

    <div class="info-card-location">
        <a id="contrib-orcid-0009-0000-7398-2330" href="https://orcid.org/0009-0000-7398-2330" data-google-interstitial="false">
            <img class="orchid-icon" alt="ORCID logo" aria-hidden="true" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/mini-icon.png">&nbsp;&nbsp;https://orcid.org/0009-0000-7398-2330
        </a>
    </div>
    <div class="info-card-search-label">
        Search for other works by this author on:
    </div>

<div class="info-card-search info-card-search-internal">
    <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Vladyslav+Husak" rel="nofollow" data-google-interstitial="false">Oxford Academic</a>
</div>

    <div class="info-card-search info-card-search-pubmed">
        <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Husak%20V" data-google-interstitial="false">PubMed</a>
    </div>
    <div class="info-card-search info-card-search-google">
        <a href="http://scholar.google.com/scholar?q=author:%22Husak%20Vladyslav%22" data-google-interstitial="false">Google Scholar</a>
    </div>
</div>                                        </span>
                                    </span>
                                    <span class="al-author-name js-flyout-wrap">







                                            <a href="javascript:;" class="linked-name js-linked-name-trigger btn-as-link" aria-controls="author-flyout-44877963" aria-expanded="false" data-google-interstitial="false">Nazar Shevchuk</a><span class="delimiter">, </span>
                                        
                                        <span id="author-flyout-44877963" class="al-author-info-wrap arrow-up">
<div class="info-card-author authorInfo_OUP_ArticleTop_Info_Widget">
    <div class="name-role-wrap">
        <div class="info-card-name">
    Nazar Shevchuk
</div>

            <div class="info-card-role">
            Data curation, Validation
            </div>
    </div>

    <div class="info-card-affilitation">
        <div class="aff"><div class="institution">Receptor.AI Inc.</div>, London N1 7GU, <div class="country">United Kingdom</div></div>
    </div>

    <div class="info-card-location">
        <a id="contrib-orcid-0009-0003-8474-9376" href="https://orcid.org/0009-0003-8474-9376" data-google-interstitial="false">
            <img class="orchid-icon" alt="ORCID logo" aria-hidden="true" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/mini-icon.png">&nbsp;&nbsp;https://orcid.org/0009-0003-8474-9376
        </a>
    </div>
    <div class="info-card-search-label">
        Search for other works by this author on:
    </div>

<div class="info-card-search info-card-search-internal">
    <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Nazar+Shevchuk" rel="nofollow" data-google-interstitial="false">Oxford Academic</a>
</div>

    <div class="info-card-search info-card-search-pubmed">
        <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shevchuk%20N" data-google-interstitial="false">PubMed</a>
    </div>
    <div class="info-card-search info-card-search-google">
        <a href="http://scholar.google.com/scholar?q=author:%22Shevchuk%20Nazar%22" data-google-interstitial="false">Google Scholar</a>
    </div>
</div>                                        </span>
                                    </span>
                                    <span class="al-author-name js-flyout-wrap">







                                            <a href="javascript:;" class="linked-name js-linked-name-trigger btn-as-link" aria-controls="author-flyout-44877964" aria-expanded="false" data-google-interstitial="false">Zakhar Ostrovsky</a><span class="delimiter">, </span>
                                        
                                        <span id="author-flyout-44877964" class="al-author-info-wrap arrow-up">
<div class="info-card-author authorInfo_OUP_ArticleTop_Info_Widget">
    <div class="name-role-wrap">
        <div class="info-card-name">
    Zakhar Ostrovsky
</div>

            <div class="info-card-role">
            Data curation, Software
            </div>
    </div>

    <div class="info-card-affilitation">
        <div class="aff"><div class="institution">Receptor.AI Inc.</div>, London N1 7GU, <div class="country">United Kingdom</div></div>
    </div>

    <div class="info-card-location">
        <a id="contrib-orcid-0009-0003-4644-3587" href="https://orcid.org/0009-0003-4644-3587" data-google-interstitial="false">
            <img class="orchid-icon" alt="ORCID logo" aria-hidden="true" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/mini-icon.png">&nbsp;&nbsp;https://orcid.org/0009-0003-4644-3587
        </a>
    </div>
    <div class="info-card-search-label">
        Search for other works by this author on:
    </div>

<div class="info-card-search info-card-search-internal">
    <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Zakhar+Ostrovsky" rel="nofollow" data-google-interstitial="false">Oxford Academic</a>
</div>

    <div class="info-card-search info-card-search-pubmed">
        <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ostrovsky%20Z" data-google-interstitial="false">PubMed</a>
    </div>
    <div class="info-card-search info-card-search-google">
        <a href="http://scholar.google.com/scholar?q=author:%22Ostrovsky%20Zakhar%22" data-google-interstitial="false">Google Scholar</a>
    </div>
</div>                                        </span>
                                    </span>
                                    <span class="al-author-name js-flyout-wrap">







                                            <a href="javascript:;" class="linked-name js-linked-name-trigger btn-as-link" aria-controls="author-flyout-44877965" aria-expanded="false" data-google-interstitial="false">Volodymyr Bdzhola</a><span class="delimiter">, </span>
                                        
                                        <span id="author-flyout-44877965" class="al-author-info-wrap arrow-up">
<div class="info-card-author authorInfo_OUP_ArticleTop_Info_Widget">
    <div class="name-role-wrap">
        <div class="info-card-name">
    Volodymyr Bdzhola
</div>

            <div class="info-card-role">
            Data curation, Validation
            </div>
    </div>

    <div class="info-card-affilitation">
        <div class="aff"><div class="institution">Institute of Molecular Biology and Genetics of The National Academy of Sciences of Ukraine</div>, Kyiv 03143, <div class="country">Ukraine</div></div>
    </div>

    <div class="info-card-location">
        <a id="contrib-orcid-0000-0003-0315-450X" href="https://orcid.org/0000-0003-0315-450X" data-google-interstitial="false">
            <img class="orchid-icon" alt="ORCID logo" aria-hidden="true" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/mini-icon.png">&nbsp;&nbsp;https://orcid.org/0000-0003-0315-450X
        </a>
    </div>
    <div class="info-card-search-label">
        Search for other works by this author on:
    </div>

<div class="info-card-search info-card-search-internal">
    <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Volodymyr+Bdzhola" rel="nofollow" data-google-interstitial="false">Oxford Academic</a>
</div>

    <div class="info-card-search info-card-search-pubmed">
        <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bdzhola%20V" data-google-interstitial="false">PubMed</a>
    </div>
    <div class="info-card-search info-card-search-google">
        <a href="http://scholar.google.com/scholar?q=author:%22Bdzhola%20Volodymyr%22" data-google-interstitial="false">Google Scholar</a>
    </div>
</div>                                        </span>
                                    </span>
                                    <span class="al-author-name js-flyout-wrap">







                                            <a href="javascript:;" class="linked-name js-linked-name-trigger btn-as-link" aria-controls="author-flyout-44877966" aria-expanded="false" data-google-interstitial="false">Semen Yesylevskyy</a><span class="delimiter">, </span>
                                        
                                        <span id="author-flyout-44877966" class="al-author-info-wrap arrow-up">
<div class="info-card-author authorInfo_OUP_ArticleTop_Info_Widget">
    <div class="name-role-wrap">
        <div class="info-card-name">
    Semen Yesylevskyy
</div>

            <div class="info-card-role">
            Formal analysis, Methodology, Project administration, Supervision, Writing - original draft
            </div>
    </div>

    <div class="info-card-affilitation">
        <div class="aff"><div class="institution">Receptor.AI Inc.</div>, London N1 7GU, <div class="country">United Kingdom</div></div><div class="aff"><div class="institution">Department of Physics of Biological Systems, Institute of Physics of The National Academy of Sciences of Ukraine</div>, Kyiv 03038, <div class="country">Ukraine</div></div><div class="aff"><div class="institution">Institute of Organic Chemistry and Biochemistry, Czech Academy of Sciences</div>, Prague 6 CZ-166 10, <div class="country">Czech Republic</div></div><div class="aff"><div class="institution">Department of Physical Chemistry, Faculty of Science, Palacký University Olomouc</div>, Olomouc 771 46, <div class="country">Czech Republic</div></div>
    </div>

    <div class="info-card-location">
        <a id="contrib-orcid-0000-0002-6748-8931" href="https://orcid.org/0000-0002-6748-8931" data-google-interstitial="false">
            <img class="orchid-icon" alt="ORCID logo" aria-hidden="true" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/mini-icon.png">&nbsp;&nbsp;https://orcid.org/0000-0002-6748-8931
        </a>
    </div>
    <div class="info-card-search-label">
        Search for other works by this author on:
    </div>

<div class="info-card-search info-card-search-internal">
    <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Semen+Yesylevskyy" rel="nofollow" data-google-interstitial="false">Oxford Academic</a>
</div>

    <div class="info-card-search info-card-search-pubmed">
        <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yesylevskyy%20S" data-google-interstitial="false">PubMed</a>
    </div>
    <div class="info-card-search info-card-search-google">
        <a href="http://scholar.google.com/scholar?q=author:%22Yesylevskyy%20Semen%22" data-google-interstitial="false">Google Scholar</a>
    </div>
</div>                                        </span>
                                    </span>
                                    <span class="al-author-name js-flyout-wrap">







                                            <a href="javascript:;" class="linked-name js-linked-name-trigger btn-as-link" aria-controls="author-flyout-44877967" aria-expanded="false" data-google-interstitial="false">Serhii Starosyla</a><span class="delimiter-hidden">, </span>
                                        
                                        <span id="author-flyout-44877967" class="al-author-info-wrap arrow-up">
<div class="info-card-author authorInfo_OUP_ArticleTop_Info_Widget">
    <div class="name-role-wrap">
        <div class="info-card-name">
    Serhii Starosyla
</div>

            <div class="info-card-role">
            Conceptualization, Project administration, Supervision
            </div>
    </div>

    <div class="info-card-affilitation">
        <div class="aff"><div class="institution">Receptor.AI Inc.</div>, London N1 7GU, <div class="country">United Kingdom</div></div>
    </div>

    <div class="info-card-location">
        <a id="contrib-orcid-0000-0002-5103-0635" href="https://orcid.org/0000-0002-5103-0635" data-google-interstitial="false">
            <img class="orchid-icon" alt="ORCID logo" aria-hidden="true" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/mini-icon.png">&nbsp;&nbsp;https://orcid.org/0000-0002-5103-0635
        </a>
    </div>
    <div class="info-card-search-label">
        Search for other works by this author on:
    </div>

<div class="info-card-search info-card-search-internal">
    <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Serhii+Starosyla" rel="nofollow" data-google-interstitial="false">Oxford Academic</a>
</div>

    <div class="info-card-search info-card-search-pubmed">
        <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Starosyla%20S" data-google-interstitial="false">PubMed</a>
    </div>
    <div class="info-card-search info-card-search-google">
        <a href="http://scholar.google.com/scholar?q=author:%22Starosyla%20Serhii%22" data-google-interstitial="false">Google Scholar</a>
    </div>
</div>                                        </span>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more
                                </a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more js-flyout-wrap">





                                                <a href="javascript:;" class="linked-name js-linked-name-trigger btn-as-link" aria-controls="author-flyout-44877968" aria-expanded="false" data-google-interstitial="false">Alan Nafiiev</a><span class="delimiter"></span>
                                            
                                            <span id="author-flyout-44877968" class="al-author-info-wrap arrow-up">
<div class="info-card-author authorInfo_OUP_ArticleTop_Info_Widget">
    <div class="name-role-wrap">
        <div class="info-card-name">
    Alan Nafiiev
</div>

            <div class="info-card-role">
            Conceptualization, Project administration, Supervision
            </div>
    </div>

    <div class="info-card-affilitation">
        <div class="aff"><div class="institution">Receptor.AI Inc.</div>, London N1 7GU, <div class="country">United Kingdom</div></div>
    </div>

    <div class="info-card-location">
        <a id="contrib-orcid-0009-0004-8604-377X" href="https://orcid.org/0009-0004-8604-377X" data-google-interstitial="false">
            <img class="orchid-icon" alt="ORCID logo" aria-hidden="true" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/mini-icon.png">&nbsp;&nbsp;https://orcid.org/0009-0004-8604-377X
        </a>
    </div>
    <div class="info-card-search-label">
        Search for other works by this author on:
    </div>

<div class="info-card-search info-card-search-internal">
    <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Alan+Nafiiev" rel="nofollow" data-google-interstitial="false">Oxford Academic</a>
</div>

    <div class="info-card-search info-card-search-pubmed">
        <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nafiiev%20A" data-google-interstitial="false">PubMed</a>
    </div>
    <div class="info-card-search info-card-search-google">
        <a href="http://scholar.google.com/scholar?q=author:%22Nafiiev%20Alan%22" data-google-interstitial="false">Google Scholar</a>
    </div>
</div>                                            </span>
                                        </span>
                                </div>
                            </div>

                    </div>
                </div>
<div class="pub-history-wrap clearfix js-history-dropdown-wrap">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary"><em>Bioinformatics</em>, Volume 41, Issue 8, August 2025, btaf449, <a href="https://doi.org/10.1093/bioinformatics/btaf449" data-google-interstitial="false">https://doi.org/10.1093/bioinformatics/btaf449</a></div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">07 August 2025</div>
            </div>
                <a href="javascript:;" class="history-label js-history-dropdown-trigger st-article-history at-ArticleHistory" data-google-interstitial="false">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
                    </div>
            <div class="ww-history js-history-entries-wrap at-history-entries-wrap">
                <div class="history-entry at-history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">21 May 2025</div>
                </div>
                <div class="history-entry at-history-entry">
                    <div class="wi-state">Revision received:</div>
                    <div class="wi-date">07 July 2025</div>
                </div>
                <div class="history-entry at-history-entry">
                    <div class="wi-state">Editorial decision:</div>
                    <div class="wi-date">24 July 2025</div>
                </div>
                <div class="history-entry at-history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">06 August 2025</div>
                </div>
                <div class="history-entry at-history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">07 August 2025</div>
                </div>
                <div class="history-entry at-history-entry">
                    <div class="wi-state">Corrected and typeset:</div>
                    <div class="wi-date">21 August 2025</div>
                </div>
        </div>
        </div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>

 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body js-content-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
    <li class="toolbar-item item-pdf js-item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="8225722" href="https://academic.oup.com/bioinformatics/article-pdf/41/8/btaf449/63978887/btaf449.pdf">
            <img src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view at-split-view" target="" data-google-interstitial="false">
                                    <i class="icon-menu-split"></i>
                                    Split View
                                </a>
                            </li>

                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="at-views-dropdown drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true" data-google-interstitial="false">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown js-dropdown-content" data-dropdown-content="" aria-label="submenu">
                                <div class="arrow-up" style="display: block;"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content" style="display: none;">
                                    <a href="javascript:;" data-google-interstitial="false"><span>Article contents</span></a>
                                </li>
                                <li class="at-figures-tables article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;" data-google-interstitial="false"><span>Figures &amp; tables</span></a>
                                </li>
                                
                                
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;" data-google-interstitial="false"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
                        </li>


                        <li class="toolbar-item item-cite js-item-cite">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        <a href="#" class="js-cite-button at-CiteButton" data-reveal-id="getCitation" data-reveal="">
    <i class="icon-read-more"></i>
    <span>Cite</span>
</a>

<div id="getCitation" class="reveal-modal js-citation-modal" data-reveal="">
    <h3 class="modal-title">Cite</h3>
        <div class="oxford-citation-text">
            <p>Roman Stratiichuk, Mykola Melnychenko, Ihor Koleiev, 
Taras Voitsitskyi, Vladyslav Husak, Nazar Shevchuk, Zakhar Ostrovsky, 
Volodymyr Bdzhola, Semen Yesylevskyy, Serhii Starosyla, Alan Nafiiev,  
Leveraging large language models for literature-driven prioritization of
 protein binding pockets, <em>Bioinformatics</em>, Volume 41, Issue 8, August 2025, btaf449, <a href="https://doi.org/10.1093/bioinformatics/btaf449" data-google-interstitial="false">https://doi.org/10.1093/bioinformatics/btaf449</a></p>
        </div>

    <div class="citation-download-wrap">
        <form action="/Citation/Download" method="get" id="citationModal">
            <input type="hidden" name="resourceId" value="8225722">
            <input type="hidden" name="resourceType" value="3">
            <label for="selectFormat" class="hide js-citation-format-label">Select Format</label>
            <select required="" name="citationFormat" class="citation-download-format js-citation-format" id="selectFormat">
                <option selected="selected" disabled="disabled">Select format</option>
                        <option value="0">.ris (Mendeley, Papers, Zotero)</option>
                        <option value="1">.enw (EndNote)</option>
                        <option value="2">.bibtex (BibTex)</option>
                        <option value="3">.txt (Medlars, RefWorks)</option>

            </select>
            <button class="btn citation-download-link disabled" type="submit">Download citation</button>
        </form>
    </div>

    <a class="close-reveal-modal" href="javascript:;" data-google-interstitial="false"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-OUP_Get_Permissions">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1367-4811&amp;title=Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets&amp;publicationDate=2025-08-07&amp;volumeNum=41&amp;issueNum=8&amp;author=Stratiichuk%2C%20Roman%3B%20Melnychenko%2C%20Mykola&amp;startPage=1&amp;endPage=11&amp;contentId=10.1093%2Fbioinformatics%2Fbtaf449&amp;oa=CC%20BY&amp;copyright=%C2%A9%20The%20Author%28s%29%202025.%20Published%20by%20Oxford%20University%20Press.&amp;orderBeanReset=True" id="PermissionsLink" class="" target="" rel="nofollow" data-google-interstitial="false">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>

    <li class="toolbar-item item-with-dropdown item-share">
        <a href="javascript:;" class="drop-trigger js-toolbar-dropdown at-ShareButton" data-dropdown="ShareDrop" data-google-interstitial="false">
            <i class="icon-menu_share"><span class="screenreader-text">Share Icon</span></i>
            <span class="toolbar-label">
                <span class="toolbar-text">Share</span>
                <i class="arrow-icon icon-general-arrow-filled-down js-toolbar-arrow-icon"></i>
            </span>
        </a>
        <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown js-dropdown-content" data-dropdown-content="">

    <li>
        <a class="st-custom-button addthis_button_bluesky js-share-link" data-network="bluesky" data-title="Leveraging large language models for literature-driven prioritization of protein binding pockets" data-url=" https://academic.oup.com/bioinformatics/article/41/8/btaf449/8225722" data-email-subject="Leveraging large language models for literature-driven prioritization of protein binding pockets" href="javascript:;" rel="nofollow" data-google-interstitial="false"><span>Bluesky</span></a>
    </li>
    <li>
        <a class="st-custom-button addthis_button_facebook js-share-link" data-network="facebook" data-title="Leveraging large language models for literature-driven prioritization of protein binding pockets" data-url="https://academic.oup.com/bioinformatics/article/41/8/btaf449/8225722" data-email-subject="Leveraging large language models for literature-driven prioritization of protein binding pockets" href="javascript:;" rel="nofollow" data-google-interstitial="false"><span>Facebook</span></a>
    </li>
    <li>
        <a class="st-custom-button addthis_button_twitter js-share-link" data-network="twitter" data-title="Leveraging large language models for literature-driven prioritization of protein binding pockets" data-url="https://academic.oup.com/bioinformatics/article/41/8/btaf449/8225722" data-email-subject="Leveraging large language models for literature-driven prioritization of protein binding pockets" href="javascript:;" rel="nofollow" data-google-interstitial="false"><span>X</span></a>
    </li>
    <li>
        <a class="st-custom-button addthis_button_linkedin js-share-link" data-network="linkedin" data-title="Leveraging large language models for literature-driven prioritization of protein binding pockets" data-url="https://academic.oup.com/bioinformatics/article/41/8/btaf449/8225722" data-email-subject="Leveraging large language models for literature-driven prioritization of protein binding pockets" href="javascript:;" rel="nofollow" data-google-interstitial="false"><span>LinkedIn</span></a>
    </li>
    <li>
        <a class="st-custom-button addthis_button_email js-share-link" data-network="email" data-title="Leveraging large language models for literature-driven prioritization of protein binding pockets" data-url="https://academic.oup.com/bioinformatics/article/41/8/btaf449/8225722" data-email-subject="Leveraging large language models for literature-driven prioritization of protein binding pockets" href="javascript:;" rel="nofollow" data-google-interstitial="false"><span>Email</span></a>
    </li>



        </ul>
    </li>


                    </ul>
                    <div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        



<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">


<div class="navbar-search">
    <search aria-label="Site search">
        <div class="mobile-microsite-search">
                <label for="OUP_ArticleToolbarSearchBox-mobile-navbar-search-filter" class="screenreader-text js-mobile-navbar-search-filter-label">
                    Navbar Search Filter
                </label>
                <select class="mobile-navbar-search-filter js-mobile-navbar-search-filter at-navbar-search-filter" id="OUP_ArticleToolbarSearchBox-mobile-navbar-search-filter">

<option class="navbar-search-filter-option at-navbar-search-filter-option" value="" selected="selected">Bioinformatics</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Issue">This issue</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Parent">Bioinformatics Journals</option>
                        <optgroup class="navbar-search-optgroup" label="Search across Oxford Academic">
<option class="navbar-search-filter-option at-navbar-search-filter-option" value="AcademicSubjects/SCI01060">Bioinformatics and Computational Biology</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Books">Books</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Journals">Journals</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Umbrella">Oxford Academic</option>                        </optgroup>
                </select>

            <label for="OUP_ArticleToolbarSearchBox-mobile-microsite-search-term" class="screenreader-text js-mobile-microsite-search-term-label">
                Mobile Enter search term
            </label>
            <input class="mobile-search-input mobile-microsite-search-term js-mobile-microsite-search-term at-microsite-search-term " type="search" maxlength="255" placeholder="Search" id="OUP_ArticleToolbarSearchBox-mobile-microsite-search-term">

            <a href="javascript:;" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search" data-google-interstitial="false">
                <span class="screenreader-text">Search</span>
            </a>

        </div>
    </search>
</div>


        </div>
    </div>
   
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>


<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">
        </nav>
            <div class="navbar-right-container">
                <div class="navbar-search-container js-navbar-search-container">
                    <a href="javascript:;" class="navbar-search-close js_close-navsearch" data-google-interstitial="false">Close</a>


<div class="navbar-search">
    <search aria-label="Site search">
        <div class="microsite-search">
                <label for="OUP_ArticleToolbarSearchBox-navbar-search-filter" class="screenreader-text js-navbar-search-filter-label">
                    Navbar Search Filter
                </label>
                <select class="navbar-search-filter js-navbar-search-filter at-navbar-search-filter" id="OUP_ArticleToolbarSearchBox-navbar-search-filter">

<option class="navbar-search-filter-option at-navbar-search-filter-option" value="" selected="selected">Bioinformatics</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Issue">This issue</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Parent">Bioinformatics Journals</option>
                        <optgroup class="navbar-search-optgroup" label="Search across Oxford Academic">
<option class="navbar-search-filter-option at-navbar-search-filter-option" value="AcademicSubjects/SCI01060">Bioinformatics and Computational Biology</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Books">Books</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Journals">Journals</option><option class="navbar-search-filter-option at-navbar-search-filter-option" value="Umbrella">Oxford Academic</option>                        </optgroup>
                </select>

            <label for="OUP_ArticleToolbarSearchBox-microsite-search-term" class="screenreader-text js-microsite-search-term-label">
                Enter search term
            </label>
            <input class="navbar-search-input microsite-search-term js-microsite-search-term at-microsite-search-term " type="search" maxlength="255" placeholder="Search" id="OUP_ArticleToolbarSearchBox-microsite-search-term">

            <a href="javascript:;" class="microsite-search-icon navbar-search-submit icon-menu_search" data-google-interstitial="false">
                <span class="screenreader-text">Search</span>
            </a>

        </div>
    </search>
</div>


                    <div class="navbar-search-advanced"><a href="https://academic.oup.com/bioinformatics/advanced-search?page=1&amp;fl_SiteID=5139&amp;SearchSourceType=1" class="advanced-search js-advanced-search" data-google-interstitial="false">Advanced Search</a></div>
                </div>
                
                <div class="navbar-search-collapsed"><a href="javascript:;" class="icon-menu_search js_expand-navsearch" data-google-interstitial="false"><span class="screenreader-text">Search Menu</span></a></div>

        <div class="navbar-menu navbar-ai-assistant-container site-menu-item">
            <a href="javascript:;" class="navbar-ai-assistant-link js-navbar-ai-assistant-link" data-google-interstitial="false">
                <span aria-label="AI Discovery Assistant" class="navbar-ai-assistant-span">
                    <i class="icon-spark navbar-icon-spark"></i>
                    AI Discovery Assistant
                </span>
            </a>
                <div class="ai-onboarding-discovery-tooltip hidden js-navbar-ai-assistant-tooltip" data-discoverymode="discovery">
                    <div class="tooltiptext">
                        Discover the most relevant content quickly with our AI Discovery Assistant
                    </div>
                    <i class="icon-spark navbar-icon-spark"></i>
                </div>
        </div>

            </div>
            </div>
</div>
<input id="routename" name="RouteName" type="hidden" value="bioinformatics"> 
    </div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">                                                
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="529947872" id="529947872" class="abstract-title js-splitscreen-abstract-title">Abstract</h2>
<section class="abstract"><div class=" sec"><div class="title">Motivation</div><p class="chapter-para">Accurately
 identifying and prioritizing protein binding pockets is a foundational 
element of small-molecule drug discovery. Defining these known pockets 
currently relies on a laborious manual process of extracting key residue
 data from selected publications, reconciling inconsistent terminology, 
and independently computing volumetric representations. This manual 
curation to ensure biological relevance is time-consuming, error-prone, 
and represents a major bottleneck for efficient, high-throughput drug 
discovery.</p></div><div class=" sec"><div class="title">Results</div><p class="chapter-para">We
 present a novel approach for the identification and prioritization of 
protein binding pockets for small molecules by combining geometric 
pocket detection with large language models (LLMs). Our method leverages
 Fpocket to generate candidate pockets, which are then validated against
 published experimental data extracted from research articles using LLM 
with a series of prompts fine-tuned to identify and extract 
residue-level information associated with experimentally confirmed 
binding sites. We developed a curated benchmark dataset of diverse 
proteins and associated literature to train and evaluate the LLM’s 
performance in paper relevance assessment and pocket extraction.</p></div><div class=" sec"><div class="title">Availability and implementation</div><p class="chapter-para">The developed benchmark dataset and methodology are freely available at the GitHub repository (<a class="link link-uri openInAnotherWindow" href="https://github.com/receptor-ai/LLM-benchmark-dataset" target="_blank" data-google-interstitial="false">https://github.com/receptor-ai/LLM-benchmark-dataset</a>) and Zenodo (DOI: 10.5281/zenodo.15798647).</p></div></section>                    
                    <div id="529947873"></div>
<section class="abstract abstractBorder graphicalAbstract"><div class="
            block-child-p
             js-p-fig-section"><span id="btaf449-f11"></span><div data-id="btaf449-f11" data-content-id="btaf449-f11" class="fig fig-section" swap-content-for-modal="true"><div class="graphic-wrap"><img class="content-image" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/m_btaf449f11.jpeg" alt="Graphical Abstract" data-path-from-xml="btaf449f11.tif"><div class="graphic-bottom"><div class="label fig-label" id="label-529947873">Graphical Abstract</div><div class="ajax-articleAbstract-exclude-regex fig-orig original-slide figure-button-wrap"><a class="fig-view-orig js-view-large at-figureViewLarge openInAnotherWindow" role="button" aria-describedby="label-529947873" href="https://academic.oup.com/view-large/figure/529947873/btaf449f11.tif?itm_medium=graphical+abstract+image&amp;itm_content=open+image&amp;itm_source=http://academic.oup.com/bioinformatics/article/41/8/btaf449/8225722&amp;itm_campaign=graphical+abstract" data-path-from-xml="btaf449f11.tif" target="_blank">Open in new tab</a><a class="download-slide" role="button" aria-describedby="label-529947873" data-section="529947873" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449f11.jpeg?Expires=1765920142&amp;Signature=JXgl2-yroZremNtlhFWjMGAq~3ZS~9cza36MsDlQR4IM7TEBZgdM8nSeEqdJtVf918nhuITKzbrjEz-0aG1k83nXKatjkwxqTgFh2w2yucXCuiRbJuM1qvRc0xbwxtmc0WFM0EV8GWzloVCRFkmUWSerlWyCyO2~0rM0QSrs4rTP7vC1vsScx67BQMTvcIj9k-5KQ3r~nl6McWFCLph29BZRJpfRfgYDDMRm~So3FXiYXY3LMtV1vN5DxUlfyBczCVGvVzhMZQBlpr1bR95DHRBbPAcmnp-dd3DgLnE5Hruzsc7NlyXQRKbfqBqu9yhUaVVy966PQcFTWZd2cInR~A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=529947873&amp;ar=8225722&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5139&amp;itm_medium=graphical+abstract+image&amp;itm_content=download+button&amp;itm_source=http://academic.oup.com/bioinformatics/article/41/8/btaf449/8225722&amp;itm_campaign=graphical+abstract" data-path-from-xml="btaf449f11.tif">Download slide</a></div></div></div></div></div></section>                    <div class="article-metadata-panel clearfix at-ArticleMetadata"><div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="https://academic.oup.com/bioinformatics/search-results?f_TocHeadingTitleList=Original+Paper" data-google-interstitial="false">Original Paper</a>
                            <span class="article-metadata-tocSections-delimiter">&gt;</span>
                        <a href="https://academic.oup.com/bioinformatics/search-results?f_TocHeadingTitleList=DATA+@@@+TEXT+MINING" data-google-interstitial="false">DATA AND TEXT MINING</a>
                </div><div class="wi-editors">
            <div class="al-authors-list">

                    <div class="editor-tag-label">
                            <span class="editor-role-name">Associate Editor:</span>


                        <span class="al-author-name js-flyout-wrap">

                            <a class="linked-name js-linked-name stats-author-info-trigger" href="javascript:;" aria-expanded="false" data-google-interstitial="false">Jonathan Wren</a><span class="delimiter"></span>
                            <span id="info-44877969" class="al-author-info-wrap arrow-up stats-author-info-panel">
<div class="info-card-author authorInfo_OUP_Article_EditorInformation_Widget">
    <div class="name-role-wrap">
        <div class="info-card-name">
    Jonathan Wren
</div>

            <div class="info-card-role">
            Associate Editor
            </div>
    </div>


    <div class="info-card-search-label">
        Search for other works by this author on:
    </div>

<div class="info-card-search info-card-search-internal">
    <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Jonathan+Wren" rel="nofollow" data-google-interstitial="false">Oxford Academic</a>
</div>

    <div class="info-card-search info-card-search-pubmed">
        <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wren%20J" data-google-interstitial="false">PubMed</a>
    </div>
    <div class="info-card-search info-card-search-google">
        <a href="http://scholar.google.com/scholar?q=author:%22Wren%20Jonathan%22" data-google-interstitial="false">Google Scholar</a>
    </div>
</div>
                            </span>

                        </span>
                        <br>
                    </div>
            </div>
        </div><div class="metadata-taggedCollection">
                    <span class="metadata-collection-tagging-title">
                        Collection: 
                    </span>
                        <a href="https://academic.oup.com/bioinformaticsjournals" data-google-interstitial="false">Bioinformatics Journals</a>                </div></div>
                    <h2 scrollto-destination="529947875" id="529947875" class="section-title js-splitscreen-section-title">1 Introduction</h2>
<p class="chapter-para">The protein binding pockets are the regions on 
the protein surface where the ligands—such as small molecules, peptides,
 nucleic acids, or other proteins—can bind. They play a critical role in
 facilitating biological processes and are essential for understanding 
protein function. Accurate identification of these sites is crucial for 
the development of new drugs and the rational design of novel proteins 
with specific functions.</p><p class="chapter-para">Over the years, 
various computational methods have been developed to predict protein 
binding pockets. Geometry-based methods rely on the structural 
identification of cavities or clefts on the protein surface that may 
accommodate ligands. The most widely used method of this family is 
Fpocket (<span class="xrefLink" id="jumplink-btaf449-B15"></span><a href="javascript:;" reveal-id="btaf449-B15" data-open="btaf449-B15" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Le Guilloux <em>et al.</em> 2009</a>), which uses alpha spheres (<span class="xrefLink" id="jumplink-btaf449-B16"></span><a href="javascript:;" reveal-id="btaf449-B16" data-open="btaf449-B16" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Liang <em>et al.</em> 1998</a>) to represent the shape of the pocket cavities.</p><p class="chapter-para">Energy-based methods, such as EASYMIFs and SITEHOUND (<span class="xrefLink" id="jumplink-btaf449-B7"></span><a href="javascript:;" reveal-id="btaf449-B7" data-open="btaf449-B7" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Ghersi and Sanchez 2009</a>),
 use clustering and filtering of the molecular interaction fields (MIFs)
 to locate the spatial regions on the protein surface that are 
energetically favorable for ligand binding. Despite their advantages in 
accounting for biologically relevant interactions, these techniques are 
computationally more expensive and require careful parameter tuning.</p><p class="chapter-para">Methods like ConCavity (<span class="xrefLink" id="jumplink-btaf449-B5"></span><a href="javascript:;" reveal-id="btaf449-B5" data-open="btaf449-B5" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Capra <em>et al.</em> 2009</a>)
 combine a geometry-based approach with a sequence conservation analysis
 to prioritize the tentative pockets based on their evolutionary 
significance. However, such techniques often overlook unique 
non-conserved binding pockets and perform poorly for proteins without a 
large number of homologs.</p><p class="chapter-para">In recent years, 
pocket detection techniques based on machine learning and deep learning 
have emerged. Such methods as DeepSite (<span class="xrefLink" id="jumplink-btaf449-B10"></span><a href="javascript:;" reveal-id="btaf449-B10" data-open="btaf449-B10" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Jiménez <em>et al.</em> 2017</a>), DeepPocket (<span class="xrefLink" id="jumplink-btaf449-B1"></span><a href="javascript:;" reveal-id="btaf449-B1" data-open="btaf449-B1" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Aggarwal <em>et al.</em> 2022</a>), PUResNet (<span class="xrefLink" id="jumplink-btaf449-B11"></span><a href="javascript:;" reveal-id="btaf449-B11" data-open="btaf449-B11" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Kandel <em>et al.</em> 2021</a>), and PUResNetV2.0 (<span class="xrefLink" id="jumplink-btaf449-B9"></span><a href="javascript:;" reveal-id="btaf449-B9" data-open="btaf449-B9" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Jeevan <em>et al.</em> 2024</a>)
 integrate diverse geometric, energetic, and evolutionary features, 
resulting in improved prediction accuracy. Methods like FRASE-bot (<span class="xrefLink" id="jumplink-btaf449-B4"></span><a href="javascript:;" reveal-id="btaf449-B4" data-open="btaf449-B4" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">An <em>et al.</em> 2024</a>)
 apply deep learning to identify and score small molecular fragments 
that bind to different regions of the protein surface, which is often 
beneficial for proteins without well-defined pockets.</p><p class="chapter-para">Finally, hybrid methods, such as DELIA (<span class="xrefLink" id="jumplink-btaf449-B22"></span><a href="javascript:;" reveal-id="btaf449-B22" data-open="btaf449-B22" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Xia <em>et al.</em> 2020</a>), PGpocket (<span class="xrefLink" id="jumplink-btaf449-B26"></span><a href="javascript:;" reveal-id="btaf449-B26" data-open="btaf449-B26" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Zhao <em>et al.</em> 2024</a>), and P2Rank (<span class="xrefLink" id="jumplink-btaf449-B13"></span><a href="javascript:;" reveal-id="btaf449-B13" data-open="btaf449-B13" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Krivák and Hoksza 2018</a>),
 combine multiple predictive approaches to overcome the limitations of 
individual techniques. However, this has the disadvantage of higher 
computational demands and a large number of parameters.</p><p class="chapter-para">Despite
 the large diversity of available techniques, most of them tend to 
identify more pocket-like regions than justified by experimental data. 
Nowadays, it is generally straightforward for a researcher to find a 
bunch of cavities, pits, and grooves on the surface of any protein that 
is capable of binding to different kinds of ligands, while only some of 
them are functionally significant in the biological context. This 
redundancy of pocket detection is desirable in certain scenarios, such 
as identification of the previously unknown allosteric, hidden or 
“cryptic” binding pockets, but becomes a significant obstacle otherwise.</p><p class="chapter-para">One
 of the most common tasks in computational drug discovery is docking the
 ligands into the known binding site of the target protein. In order to 
set up the docking procedure, the researcher has to know the exact 
geometric extents of the binding pocket—something that is rarely 
explicitly annotated either in public databases or in research papers. 
The most obvious solution is to run one of the pocket prediction 
techniques, which provides such information. However, the result often 
includes redundant predicted pockets that partially overlap with the 
desired spatial or sequence region, so it is unclear which of them 
should be used as is, merged or discarded. Correct post-processing of 
such predictions usually requires extensive analysis of the research 
papers, which is both time-consuming and error-prone. This issue becomes
 particularly pressing in the high-throughput or time-constrained 
scenarios in commercial drug discovery when human assessment of the 
binding pockets becomes the major bottleneck and a source of errors.</p><p class="chapter-para">In
 this article, we propose a hybrid approach that combines the 
geometry-based detection of the pockets with the large language models 
(LLMs), which assess them against the research papers and retain 
biologically relevant ones. We intend to test whether modern LLMs are 
powerful and accurate enough to substitute human researchers in binding 
pocket validation and prioritization tasks.</p><p class="chapter-para">Specifically,
 we use Fpocket to generate multiple pocket candidates and LLMs to 
identify those that are supported by literature evidence. The LLMs are 
used to extract information from the literature about the residues 
associated with experimentally confirmed binding sites, which is then 
used to select the relevant pockets (<span class="xrefLink" id="jumplink-btaf449-F1"></span><a href="javascript:;" data-modal-source-id="btaf449-F1" class="link xref-fig" data-google-interstitial="false">Fig.&nbsp;1</a>).</p>                    <a id="529947886" scrollto-destination="529947886" data-google-interstitial="false"></a>
<div data-id="btaf449-f1" data-content-id="btaf449-f1" class="fig fig-section js-fig-section" swap-content-for-modal="true"><div class="graphic-wrap"><img class="content-image" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/m_btaf449f1.jpeg" alt="The overview of the proposed pipeline for the hybrid binding pocket detection. Fpocket—the geometry-based method—predicts numerous pocket candidates. The LLM analyses scientific articles and selects biologically relevant pockets. The results are then mapped and merged so that LLM-extracted pockets are primarily used as filters for Fpocket predictions. Finally, the Fpocket pocket representation is converted into the spatial grid that undergoes postprocessing and refinement." data-path-from-xml="btaf449f1.tif"><div class="graphic-bottom"><div class="label fig-label" id="label-529947886">Figure 1.</div><div class="caption fig-caption"><p class="chapter-para">The
 overview of the proposed pipeline for the hybrid binding pocket 
detection. Fpocket—the geometry-based method—predicts numerous pocket 
candidates. The LLM analyses scientific articles and selects 
biologically relevant pockets. The results are then mapped and merged so
 that LLM-extracted pockets are primarily used as filters for Fpocket 
predictions. Finally, the Fpocket pocket representation is converted 
into the spatial grid that undergoes postprocessing and refinement.</p></div><div class="ajax-articleAbstract-exclude-regex fig-orig original-slide figure-button-wrap"><a class="fig-view-orig js-view-large at-figureViewLarge openInAnotherWindow" role="button" aria-describedby="label-529947886" href="https://academic.oup.com/view-large/figure/529947886/btaf449f1.tif" data-path-from-xml="btaf449f1.tif" target="_blank" data-google-interstitial="false">Open in new tab</a><a class="download-slide" role="button" aria-describedby="label-529947886" data-section="529947886" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449f1.jpeg?Expires=1765920142&amp;Signature=0FntW-BuoyXdr9FqSCkNXCmD2SvmkNjnzS8ltGmdjElv~bqVigAMQ-2bTG~vN9oJAyw-vStKt5fOEKFY9ZeC4rNLnizQ44NZoPNmRvQC0jXXHKPXey-wRuoKeWXWp~iIXaAVKppEjtIittm6hu8eSi~BXjKgNG4ki-RiE0cRv9SW4l49Lh3kPmFnR-swotGxkPFAxx5qisj0eCo42t~jMgit0CFjp1iT~vnf1Jc2bmkE-ZjIaoZ3lP7ZtQwpbLFYZCnW6Q0fz~ohZ95vfhfNDYsM0P-DENFhFrrh0MLAnyX9rIc8RkERPQJCIIkjNEqj~N7ZFUeJBtpWYsHnUT-xsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=529947886&amp;ar=8225722&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5139" data-path-from-xml="btaf449f1.tif" data-google-interstitial="false">Download slide</a></div></div></div></div><p class="chapter-para">We
 have created an open-source benchmark dataset to evaluate LLM 
performance specifically for protein binding site detection tasks.</p>                    <h2 scrollto-destination="529947888" id="529947888" class="section-title js-splitscreen-section-title">2 Materials and methods</h2>
                    <h3 scrollto-destination="529947889" id="529947889" class="section-title js-splitscreen-section-title">2.1 Dataset</h3>
<p class="chapter-para">At the time of writing, there are no datasets 
for evaluating the performance of the protein binding pocket triaging 
and post-processing. To address this problem, we created a curated and 
manually annotated dataset that can be used as a public benchmark for 
LLM-based extraction of pocket-related information from the literature.</p><p class="chapter-para">We
 selected 10 diverse proteins: DNA polymerase alpha catalytic subunit; 
Tyrosine-protein kinase ABL1; 5-hydroxytryptamine receptor 2A; 
Muscarinic acetylcholine receptor M3; Voltage-gated sodium channel type 
7; Voltage-gated sodium channel from American cockroach; 
Gamma-aminobutyric acid receptor; GTPase KRas; Dihydroorotate 
dehydrogenase; and Mixed lineage kinase domain-like protein.</p><p class="chapter-para">For
 each of these targets, we found 2–5 peer-reviewed research articles (31
 in total) and 3D structures from the Protein Data Bank (PDB) (see <span class="link link-data-supplement" data-supplement-target="sup1"></span><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449_supplementary_data.docx?Expires=1765920142&amp;Signature=rEHUSLvtUag7CZgpA9SGOxJ22UbvVQjJfPsA754a1iM6fSlYc~EIL6CPTQNJcIr4MKIRsSZJfrTeH9vBceKPeJd9cLR7sFxMwDWxcb4AoE9ycXowG8kGsPfL7MxGKbNMq3BhUqeZ5yAegDkDlpt~Sgz99BsTXEQNTTXfp92NnDVmsDzU0~ZZzUTljE0upu1WiC6mLpIjFz1wevcqcegeWPG~af35y1WuxadfNRfE8KCLwVbxuEPSwqSsg7FdavfgAGK~T8BOMb~tkiqweo4QGe3Rd0DzUavpRzkbmowY1EzF~82U8hnMdZq62UpcfWyxvcU76wnGiMTsdKv~WhYx-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" data-google-interstitial="false">Supplementary Information</a></span>, available as <span class="link link-data-supplement" data-supplement-target="sup1"></span><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449_supplementary_data.docx?Expires=1765920142&amp;Signature=rEHUSLvtUag7CZgpA9SGOxJ22UbvVQjJfPsA754a1iM6fSlYc~EIL6CPTQNJcIr4MKIRsSZJfrTeH9vBceKPeJd9cLR7sFxMwDWxcb4AoE9ycXowG8kGsPfL7MxGKbNMq3BhUqeZ5yAegDkDlpt~Sgz99BsTXEQNTTXfp92NnDVmsDzU0~ZZzUTljE0upu1WiC6mLpIjFz1wevcqcegeWPG~af35y1WuxadfNRfE8KCLwVbxuEPSwqSsg7FdavfgAGK~T8BOMb~tkiqweo4QGe3Rd0DzUavpRzkbmowY1EzF~82U8hnMdZq62UpcfWyxvcU76wnGiMTsdKv~WhYx-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" data-google-interstitial="false">supplementary data</a></span> at <em>Bioinformatics</em> online).</p><p class="chapter-para">Although
 the structure and functional features of popular proteins could be 
described in hundreds of papers, there is an empirical observation that 
only a handful of them usually contain well-defined information about 
the binding pockets. To simulate a real-world user workflow for our 
pipeline, we selected input articles for each target directly from top 
web search results, without imposing other selection criteria. Selected 
articles represent different complexity tiers for extracting 
pocket-related information:</p><ul class="bullet"><li><p class="chapter-para">Articles that do not describe binding sites are used as a negative control and allow false positives to be caught (11 articles).</p></li><li><p class="chapter-para">Articles
 that describe only one binding site for a given protein target. This is
 the easiest tier, which reveals LLM’s general under-standing of 
structural biology concepts (17 articles).</p></li><li><p class="chapter-para">Articles
 that describe multiple binding pockets within a given target protein to
 evaluate the ability of LLM to precisely attribute specific amino acids
 to distinct pockets (3 articles).</p></li></ul><p class="chapter-para">The
 resulting benchmark dataset comprises JSON files, one for each selected
 paper. These files detail the small-molecule binding sites explicitly 
described in the text, including the site’s name, description, and 
constituent amino acids.</p><p class="chapter-para">We utilize a GROBID (<a class="link link-uri openInAnotherWindow" href="https://github.com/kermitt2/grobid" target="_blank" data-google-interstitial="false">https://github.com/kermitt2/grobid</a>)
 PDF-to-text tool, which extracts text from PDFs into a markdown file 
with high accuracy for most papers, maintaining the document structure 
with a title, sections, and figure and table captions. Some manual 
editing was required to clean up the title and section formatting and to
 remove irrelevant sections, such as Acknowledgements and Author 
Contributions.</p>                    <h3 scrollto-destination="529947897" id="529947897" class="section-title js-splitscreen-section-title">2.2 LLM-based binding pocket extraction</h3>
<p class="chapter-para">In order to keep our technique reasonably fast 
and cost-effective, we omitted advanced reasoning architectures like 
Chain of Thought (CoT) (<span class="xrefLink" id="jumplink-btaf449-B21"></span><a href="javascript:;" reveal-id="btaf449-B21" data-open="btaf449-B21" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Wei <em>et al.</em> 2023</a>) or Tree of Thought (ToT) (<span class="xrefLink" id="jumplink-btaf449-B23"></span><a href="javascript:;" reveal-id="btaf449-B23" data-open="btaf449-B23" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Yao <em>et al.</em> 2023</a>)
—methods designed to guide complex, multi-step problem-solving. Instead,
 we relied on direct prompting, using examples and detailed instructions
 to steer the model. The final workflow incorporates three sequential 
steps: (i) paper relevance assessment, (ii) pocket extraction, and (iii)
 pocket refinement, conducted by the same LLM model (<span class="xrefLink" id="jumplink-btaf449-F2"></span><a href="javascript:;" data-modal-source-id="btaf449-F2" class="link xref-fig" data-google-interstitial="false">Fig.&nbsp;2</a>). The full text of each article was processed in full since all papers fit into the context window of modern LLMs.</p>                    <a id="529947899" scrollto-destination="529947899" data-google-interstitial="false"></a>
<div data-id="btaf449-f2" data-content-id="btaf449-f2" class="fig fig-section js-fig-section" swap-content-for-modal="true"><div class="graphic-wrap"><img class="content-image" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/m_btaf449f2.jpeg" alt="Overview of the LLM pocket extraction pipeline utilizing direct prompting. The pipeline processes complete articles through three LLM stages: (i) filtering removes irrelevant papers lacking target relevance or residue-level descriptions. (ii) Extraction, guided by extensive prompt optimization for format and accuracy, identifies small molecule binding site details (name, description, residues). (iii) Refinement further improves data quality by correcting errors and omissions." data-path-from-xml="btaf449f2.tif"><div class="graphic-bottom"><div class="label fig-label" id="label-529947899">Figure 2.</div><div class="caption fig-caption"><p class="chapter-para">Overview
 of the LLM pocket extraction pipeline utilizing direct prompting. The 
pipeline processes complete articles through three LLM stages: (i) 
filtering removes irrelevant papers lacking target relevance or 
residue-level descriptions. (ii) Extraction, guided by extensive prompt 
optimization for format and accuracy, identifies small molecule binding 
site details (name, description, residues). (iii) Refinement further 
improves data quality by correcting errors and omissions.</p></div><div class="ajax-articleAbstract-exclude-regex fig-orig original-slide figure-button-wrap"><a class="fig-view-orig js-view-large at-figureViewLarge openInAnotherWindow" role="button" aria-describedby="label-529947899" href="https://academic.oup.com/view-large/figure/529947899/btaf449f2.tif" data-path-from-xml="btaf449f2.tif" target="_blank" data-google-interstitial="false">Open in new tab</a><a class="download-slide" role="button" aria-describedby="label-529947899" data-section="529947899" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449f2.jpeg?Expires=1765920142&amp;Signature=N4ZNpGNhg-klU-sBYS2fXHQoCUzxHE-8vfCRAbmSgsClPcIRbOo-oO4Iqzc-9QVcunDbzywQn7M881NpV9Ae1FlY1-PgNmlViADQYy~J-az4Ry8PIo2ZDgUNWUhkAWQlaXVVLfByut~Mh19YrXvkbUEBIlZ-x8LpYMQKJh22GrTFFPDREUwBpkoQDcuEhGytbdNONv0gw5txXI~pljw-IDdaHvb7ZGGFlYr9FSqEroeKb0vKmfd8vk7Z~nGE1cU4BFDtVb4Y7spB3GGcNUw2OzJf-t~Z7Dm94ns4bBGGZGF99k4RJ3ie-cXczcrCn2E-h7BPGJbaFFJzv5bAj~QqEA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=529947899&amp;ar=8225722&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5139" data-path-from-xml="btaf449f2.tif" data-google-interstitial="false">Download slide</a></div></div></div></div><p class="chapter-para">In
 the first step, we assess whether the provided paper is relevant for 
searching for the pockets for a given target protein, i.e. whether it 
describes the correct protein and whether it contains a description of 
the binding pockets at the residue level.</p><p class="chapter-para">The
 pocket extraction step focuses on parsing the text and identifying 
amino acid residues comprising distinct binding pockets. Each pocket is 
assigned a name, description, and a list of residues in a predefined 
format. Significant prompt optimization efforts were focused on this 
step to ensure both strict output formatting and the correct 
identification of binding pockets.</p><p class="chapter-para">The 
subsequent pocket refinement step evaluates the extraction results to 
correct any remaining inaccuracies, such as identifying and including 
missing residues, rectifying improperly segmented binding sites, and 
filtering out binding sites irrelevant for small molecule binding (such 
as those involved in PPI or DNA/RNA binding). Validation checks were 
performed during extraction and refinement, specifically to guarantee 
the correct and consistent formatting of residue details (chain ID, 
residue name, and residue ID).</p>                    <h3 scrollto-destination="529947903" id="529947903" class="section-title js-splitscreen-section-title">2.3 Pocket extraction accuracy metrics</h3>
<p class="chapter-para">To assess the performance of our pipeline, we calculated the following pocket- and paper-based metrics:</p><p class="chapter-para"><strong><em>Pocket Number Accuracy—</em></strong>for
 each paper, if the number of annotated and extracted pockets match, 
then the prediction is correct (1), otherwise it is incorrect 0. Based 
on these values for each pocket, the final accuracy per paper is 
calculated.</p><p class="chapter-para"><strong><em>Pocket Recall—</em></strong>represents the percentage of annotated pockets that have been correctly extracted by the LLM:</p><div class="formula-wrap"><div class="disp-formula" id="jumplink-E1" content-id="E1"><span class="mathFormula"></span><div class="math"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" style="font-size: 125.5%; position: relative;" tabindex="0" ctxtmenu_counter="0"><mjx-math class="MJX-TEX" aria-hidden="true"><mjx-mtext class="mjx-n"><mjx-c class="mjx-c50"></mjx-c><mjx-c class="mjx-c6F"></mjx-c><mjx-c class="mjx-c63"></mjx-c><mjx-c class="mjx-c6B"></mjx-c><mjx-c class="mjx-c65"></mjx-c><mjx-c class="mjx-c74"></mjx-c></mjx-mtext><mjx-mi class="mjx-i"><mjx-c class="mjx-cA0"></mjx-c></mjx-mi><mjx-mtext class="mjx-n"><mjx-c class="mjx-c52"></mjx-c><mjx-c class="mjx-c65"></mjx-c><mjx-c class="mjx-c63"></mjx-c><mjx-c class="mjx-c61"></mjx-c><mjx-c class="mjx-c6C"></mjx-c><mjx-c class="mjx-c6C"></mjx-c></mjx-mtext><mjx-mo class="mjx-n" space="4"><mjx-c class="mjx-c3D"></mjx-c></mjx-mo><mjx-mrow space="4"><mjx-mfrac><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s"><mjx-mrow><mjx-msub><mjx-mrow><mjx-mi class="mjx-i"><mjx-c class="mjx-c1D45B TEX-I"></mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mtext class="mjx-n"><mjx-c class="mjx-c6D"></mjx-c><mjx-c class="mjx-c61"></mjx-c><mjx-c class="mjx-c74"></mjx-c><mjx-c class="mjx-c63"></mjx-c><mjx-c class="mjx-c68"></mjx-c><mjx-c class="mjx-c65"></mjx-c><mjx-c class="mjx-c64"></mjx-c></mjx-mtext><mjx-mi class="mjx-i"><mjx-c class="mjx-cA0"></mjx-c></mjx-mi><mjx-mtext class="mjx-n"><mjx-c class="mjx-c61"></mjx-c><mjx-c class="mjx-c6E"></mjx-c><mjx-c class="mjx-c6E"></mjx-c><mjx-c class="mjx-c6F"></mjx-c><mjx-c class="mjx-c74"></mjx-c><mjx-c class="mjx-c61"></mjx-c><mjx-c class="mjx-c74"></mjx-c><mjx-c class="mjx-c65"></mjx-c><mjx-c class="mjx-c64"></mjx-c></mjx-mtext><mjx-mi class="mjx-i"><mjx-c class="mjx-cA0"></mjx-c></mjx-mi><mjx-mtext class="mjx-n"><mjx-c class="mjx-c70"></mjx-c><mjx-c class="mjx-c6F"></mjx-c><mjx-c class="mjx-c63"></mjx-c><mjx-c class="mjx-c6B"></mjx-c><mjx-c class="mjx-c65"></mjx-c><mjx-c class="mjx-c74"></mjx-c><mjx-c class="mjx-c73"></mjx-c></mjx-mtext></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s"><mjx-mrow><mjx-msub><mjx-mrow><mjx-mi class="mjx-i"><mjx-c class="mjx-c1D45B TEX-I"></mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mtext class="mjx-n"><mjx-c class="mjx-c61"></mjx-c><mjx-c class="mjx-c6E"></mjx-c><mjx-c class="mjx-c6E"></mjx-c><mjx-c class="mjx-c6F"></mjx-c><mjx-c class="mjx-c74"></mjx-c><mjx-c class="mjx-c61"></mjx-c><mjx-c class="mjx-c74"></mjx-c><mjx-c class="mjx-c65"></mjx-c><mjx-c class="mjx-c64"></mjx-c></mjx-mtext><mjx-mi class="mjx-i"><mjx-c class="mjx-cA0"></mjx-c></mjx-mi><mjx-mtext class="mjx-n"><mjx-c class="mjx-c70"></mjx-c><mjx-c class="mjx-c6F"></mjx-c><mjx-c class="mjx-c63"></mjx-c><mjx-c class="mjx-c6B"></mjx-c><mjx-c class="mjx-c65"></mjx-c><mjx-c class="mjx-c74"></mjx-c><mjx-c class="mjx-c73"></mjx-c></mjx-mtext></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow></mjx-math><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtext>Pocket</mtext><mi>&nbsp;</mi><mtext>Recall</mtext><mo>=</mo><mrow><mfrac><mrow><mrow><msub><mrow><mi>n</mi></mrow><mrow><mtext>matched</mtext><mi>&nbsp;</mi><mtext>annotated</mtext><mi>&nbsp;</mi><mtext>pockets</mtext></mrow></msub></mrow></mrow><mrow><mrow><msub><mrow><mi>n</mi></mrow><mrow><mtext>annotated</mtext><mi>&nbsp;</mi><mtext>pockets</mtext></mrow></msub></mrow></mrow></mfrac></mrow></math></mjx-assistive-mml></mjx-container></div></div></div><p class="chapter-para"><strong><em>Pocket Specificity</em></strong> <em>(</em><strong><em>True Negative Rate</em></strong>, <strong><em>TNR</em></strong><em>):</em>
 Reflects how often the pipeline correctly avoids predicting a pocket 
when none is annotated in the ground truth. This is calculated as 1 - 
False Positive Rate (FPR). The FPR indicates the frequency at which the 
pipeline generates a fake pocket that does not correspond to any actual 
annotated pocket.</p><div class="formula-wrap"><div class="disp-formula" id="jumplink-E2" content-id="E2"><span class="mathFormula"></span><div class="math"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" style="font-size: 125.5%; position: relative;" tabindex="0" ctxtmenu_counter="1"><mjx-math class="MJX-TEX" aria-hidden="true"><mjx-mtext class="mjx-n"><mjx-c class="mjx-c50"></mjx-c><mjx-c class="mjx-c6F"></mjx-c><mjx-c class="mjx-c63"></mjx-c><mjx-c class="mjx-c6B"></mjx-c><mjx-c class="mjx-c65"></mjx-c><mjx-c class="mjx-c74"></mjx-c></mjx-mtext><mjx-mi class="mjx-i"><mjx-c class="mjx-cA0"></mjx-c></mjx-mi><mjx-mtext class="mjx-n"><mjx-c class="mjx-c53"></mjx-c><mjx-c class="mjx-c70"></mjx-c><mjx-c class="mjx-c65"></mjx-c><mjx-c class="mjx-c63"></mjx-c><mjx-c class="mjx-c69"></mjx-c><mjx-c class="mjx-c66"></mjx-c><mjx-c class="mjx-c69"></mjx-c><mjx-c class="mjx-c63"></mjx-c><mjx-c class="mjx-c69"></mjx-c><mjx-c class="mjx-c74"></mjx-c><mjx-c class="mjx-c79"></mjx-c></mjx-mtext><mjx-mo class="mjx-n" space="4"><mjx-c class="mjx-c3D"></mjx-c></mjx-mo><mjx-mn class="mjx-n" space="4"><mjx-c class="mjx-c31"></mjx-c></mjx-mn><mjx-mo class="mjx-n" space="3"><mjx-c class="mjx-c2212"></mjx-c></mjx-mo><mjx-mrow space="3"><mjx-mfrac><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s"><mjx-mrow><mjx-msub><mjx-mrow><mjx-mi class="mjx-i"><mjx-c class="mjx-c1D45B TEX-I"></mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mtext class="mjx-n"><mjx-c class="mjx-c6D"></mjx-c><mjx-c class="mjx-c69"></mjx-c><mjx-c class="mjx-c73"></mjx-c><mjx-c class="mjx-c73"></mjx-c><mjx-c class="mjx-c65"></mjx-c><mjx-c class="mjx-c73"></mjx-c></mjx-mtext></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s"><mjx-mrow><mjx-msub><mjx-mrow><mjx-mi class="mjx-i"><mjx-c class="mjx-c1D45B TEX-I"></mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mtext class="mjx-n"><mjx-c class="mjx-c65"></mjx-c><mjx-c class="mjx-c78"></mjx-c><mjx-c class="mjx-c74"></mjx-c><mjx-c class="mjx-c72"></mjx-c><mjx-c class="mjx-c61"></mjx-c><mjx-c class="mjx-c63"></mjx-c><mjx-c class="mjx-c74"></mjx-c><mjx-c class="mjx-c65"></mjx-c><mjx-c class="mjx-c64"></mjx-c></mjx-mtext><mjx-mi class="mjx-i"><mjx-c class="mjx-cA0"></mjx-c></mjx-mi><mjx-mtext class="mjx-n"><mjx-c class="mjx-c70"></mjx-c><mjx-c class="mjx-c6F"></mjx-c><mjx-c class="mjx-c63"></mjx-c><mjx-c class="mjx-c6B"></mjx-c><mjx-c class="mjx-c65"></mjx-c><mjx-c class="mjx-c74"></mjx-c><mjx-c class="mjx-c73"></mjx-c></mjx-mtext></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow></mjx-math><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtext>Pocket</mtext><mi>&nbsp;</mi><mtext>Specificity</mtext><mo>=</mo><mn>1</mn><mo>-</mo><mrow><mfrac><mrow><mrow><msub><mrow><mi>n</mi></mrow><mrow><mtext>misses</mtext></mrow></msub></mrow></mrow><mrow><mrow><msub><mrow><mi>n</mi></mrow><mrow><mtext>extracted</mtext><mi>&nbsp;</mi><mtext>pockets</mtext></mrow></msub></mrow></mrow></mfrac></mrow></math></mjx-assistive-mml></mjx-container></div></div></div><p class="chapter-para">where <span class="inline-formula"><span class="mathFormula"></span><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" style="font-size: 125.5%; position: relative;" tabindex="0" ctxtmenu_counter="2"><mjx-math class="MJX-TEX" aria-hidden="true"><mjx-mrow><mjx-msub><mjx-mrow><mjx-mi class="mjx-i"><mjx-c class="mjx-c1D45B TEX-I"></mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mtext class="mjx-n"><mjx-c class="mjx-c6D"></mjx-c><mjx-c class="mjx-c69"></mjx-c><mjx-c class="mjx-c73"></mjx-c><mjx-c class="mjx-c73"></mjx-c><mjx-c class="mjx-c65"></mjx-c><mjx-c class="mjx-c73"></mjx-c></mjx-mtext></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mi class="mjx-i"><mjx-c class="mjx-cA0"></mjx-c></mjx-mi></mjx-math><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow><msub><mrow><mi>n</mi></mrow><mrow><mtext>misses</mtext></mrow></msub></mrow><mi>&nbsp;</mi></math></mjx-assistive-mml></mjx-container></span>—the number of parsed pockets with no correspondence to any annotated pocket.</p><p class="chapter-para">To
 evaluate performance at the amino acid level within the pockets, we 
track the F1 score, recall, and precision metrics for the lists of amino
 acids in annotated and extracted pockets. The primary optimization 
objectives for our pipeline are maximizing the F1 score at the amino 
acid level, minimizing the Pocket False Positive Rate (FPR), and 
maximizing Pocket Accuracy and Pocket Recall. While other metrics are 
monitored to evaluate overall pocket extraction quality, they do not 
guide the optimization process.</p>                    <h3 scrollto-destination="529947912" id="529947912" class="section-title js-splitscreen-section-title">2.4 Pocket clustering and post-processing</h3>
<p class="chapter-para">The list of amino acids extracted by the LLM 
from the papers may not map directly into a specific PDB structure due 
to the sequence differences (e.g. insertions/deletions and engineered 
mutations) or errors in residue numbers, names, or chain IDs of the LLM 
extracted residues. Therefore, an additional algorithmic mapping step is
 required to mitigate these issues (<span class="xrefLink" id="jumplink-btaf449-F3"></span><a href="javascript:;" data-modal-source-id="btaf449-F3" class="link xref-fig" data-google-interstitial="false">Fig.&nbsp;3</a>).
 It allows handling such cases as assigning residues consistently across
 identical chains in homomers and correctly identifying binding sites 
situated at the interfaces between different chains, which are otherwise
 very hard to address on the level of LLM extraction.</p>                    <a id="529947914" scrollto-destination="529947914" data-google-interstitial="false"></a>
<div data-id="btaf449-f3" data-content-id="btaf449-f3" class="fig fig-section js-fig-section" swap-content-for-modal="true"><div class="graphic-wrap"><img class="content-image" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/m_btaf449f3.jpeg" alt="Workflow of the residue mapping algorithm. (i) LLM-extracted residues are assigned to potential PDB chains. (ii) All mapped residues in the PDB structure undergo spatial clustering. (iii) Resulting residue clusters are matched against LLM-defined pockets using an overlap threshold. (iv) Matched clusters containing residues from multiple chains are identified as interface pockets, and their residues are output. (v) Otherwise, original mapped residues are used." data-path-from-xml="btaf449f3.tif"><div class="graphic-bottom"><div class="label fig-label" id="label-529947914">Figure 3.</div><div class="caption fig-caption"><p class="chapter-para">Workflow
 of the residue mapping algorithm. (i) LLM-extracted residues are 
assigned to potential PDB chains. (ii) All mapped residues in the PDB 
structure undergo spatial clustering. (iii) Resulting residue clusters 
are matched against LLM-defined pockets using an overlap threshold. (iv)
 Matched clusters containing residues from multiple chains are 
identified as interface pockets, and their residues are output. (v) 
Otherwise, original mapped residues are used.</p></div><div class="ajax-articleAbstract-exclude-regex fig-orig original-slide figure-button-wrap"><a class="fig-view-orig js-view-large at-figureViewLarge openInAnotherWindow" role="button" aria-describedby="label-529947914" href="https://academic.oup.com/view-large/figure/529947914/btaf449f3.tif" data-path-from-xml="btaf449f3.tif" target="_blank" data-google-interstitial="false">Open in new tab</a><a class="download-slide" role="button" aria-describedby="label-529947914" data-section="529947914" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449f3.jpeg?Expires=1765920142&amp;Signature=aueZugOJocnNA7uUgB~1MqX8pZonY4TapxJKQLDHj8C3URkb0Aa~MflkfPC9OQM9cuJn3b88Ai8iLfRPlHNyhimXOvSueOZyMPdXleOMvsoYIXg8DAU4voUDlZOlZv5E8bCm8uXSLGgzYL-5ZpEW1XeS6uXSoPAYSvEJzhZdcOldxHQUIsuGaFajHWdb6qsQyWAOnD9zCzSiM-KtBfMQMZmDvVBqvJUDySW5wHD-YEzBwZpuB~3YCkrBuxwwuI4Q8RqB9hB1EbqHSyE3~e1odbK-K0iTfbiax462sD~y7HsjeKQBOJNewp8QcC1bfroQOwnSwy9ebkjlBQ-3XzFTyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=529947914&amp;ar=8225722&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5139" data-path-from-xml="btaf449f3.tif" data-google-interstitial="false">Download slide</a></div></div></div></div><p class="chapter-para">First,
 chain correspondence between the paper’s residue descriptions and the 
specific PDB structure is established using a sequence similarity 
threshold. Following this, spatial clustering is performed on all 
identified pocket residues within the PDB to detect potential 
inter-chain interfaces.</p><p class="chapter-para">A key step involves 
determining if these spatially clustered inter-chain residue groups 
match any specific pockets extracted by the LLM using a different 
sequence similarity threshold. If a match is found, the amino acids from
 that corresponding cluster are used, defining a unique inter-chain 
pocket. If no such match is identified, the LLM-extracted residues for 
the pocket are instead mapped onto all potentially relevant PDB chain 
identifiers, indicating the possibility of multiple, non-unique pockets 
located on different protein subunits.</p><p class="chapter-para">Although
 clustering can be applied to identify all binding sites, we found this 
approach yields suboptimal results when distinct binding sites are 
located in close spatial proximity. However, clustering is particularly 
effective at capturing identical binding sites formed at the interfaces 
between subunits in homo- or hetero-multimers.</p><p class="chapter-para">For
 this task, we used the MeanShift algorithm, which is particularly 
advantageous as it does not require the number of clusters to be 
specified beforehand. While our experiments showed that other methods 
(e.g. DBSCAN) could also be effective, they typically require more 
extensive, dataset-specific parameter tuning to achieve comparable 
results.</p><p class="chapter-para">Manipulations with protein structures were performed with custom scripts based on MolAR molecular modeling library (<span class="xrefLink" id="jumplink-btaf449-B24"></span><a href="javascript:;" reveal-id="btaf449-B24" data-open="btaf449-B24" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Yesylevskyy 2025</a>).</p>                    <h3 scrollto-destination="529947920" id="529947920" class="section-title js-splitscreen-section-title">2.5 Construction of pocket volumetric representation</h3>
<p class="chapter-para">Since residue information alone lacks the 
spatial detail to define a volumetric binding pocket, a geometry-based 
method provides this necessary geometric representation, while the LLM 
output subsequently serves as a filter to select the most plausible 
candidates among these geometric predictions. For geometry-based 
prediction, we use Fpocket, a method that identifies potential binding 
sites using Voronoi tessellation and alpha spheres. These predictions 
are subsequently filtered based on the LLM output, retaining only those 
pockets whose alpha spheres are located near the LLM-extracted residues.</p><p class="chapter-para">The
 resulting alpha spheres may contact zero, one, or multiple atoms from 
the same residue resulting in multiple ways of performing the Fpocket 
binding site selection. We introduced a threshold for residue match that
 accepts only those residues that have a certain percentage of atoms in 
contact with alpha spheres from the same pocket. Additionally, we use a 
threshold pocket match parameter, which defines the minimum Jaccard 
index for residue overlap between the LLM extracted and Fpocket binding 
pockets.</p><p class="chapter-para">If multiple LLM-extracted pockets 
overlap with the same geometric pocket, they are merged into a single 
pocket if the residue overlap between them exceeds a predefined 
threshold. This also allows blending the data from different 
publications describing the same binding pocket on the target protein 
but reporting different or incomplete sets of associated residues.</p><p class="chapter-para">The
 space bounded by the binding pocket surface can be filled with either 
alpha-spheres or grid points, depending on the technique used. Multiple 
geometry-based approaches, such as CB-Dock2 (<span class="xrefLink" id="jumplink-btaf449-B17"></span><a href="javascript:;" reveal-id="btaf449-B17" data-open="btaf449-B17" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Liu <em>et al.</em> 2022</a>), CASTp 3.0 (<span class="xrefLink" id="jumplink-btaf449-B19"></span><a href="javascript:;" reveal-id="btaf449-B19" data-open="btaf449-B19" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Tian <em>et al.</em> 2018</a>), PyVOL (<a class="link link-uri openInAnotherWindow" href="https://schlessinger-lab.github.io/pyvol/" target="_blank" data-google-interstitial="false">https://schlessinger-lab.github.io/pyvol/</a>), and Fpocket itself use spheres to represent protein cavities. Grid representation is used by DoGSite3 (<span class="xrefLink" id="jumplink-btaf449-B8"></span><a href="javascript:;" reveal-id="btaf449-B8" data-open="btaf449-B8" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Graef <em>et al.</em> 2023</a>) and POVME 2.0 (<span class="xrefLink" id="jumplink-btaf449-B6"></span><a href="javascript:;" reveal-id="btaf449-B6" data-open="btaf449-B6" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Durrant <em>et al.</em> 2014</a>).</p><p class="chapter-para">The
 representation of alpha spheres used by Fpocket is not very practical 
for our purposes because merging or splitting the pockets represented by
 spheres is non-trivial and involves unnecessarily complex algorithms. 
Thus, we convert the pockets to a grid representation, as shown in <span class="xrefLink" id="jumplink-btaf449-F4"></span><a href="javascript:;" data-modal-source-id="btaf449-F4" class="link xref-fig" data-google-interstitial="false">Fig.&nbsp;4</a>.</p>                    <a id="529947926" scrollto-destination="529947926" data-google-interstitial="false"></a>
<div data-id="btaf449-f4" data-content-id="btaf449-f4" class="fig fig-section js-fig-section" swap-content-for-modal="true"><div class="graphic-wrap"><img class="content-image" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/m_btaf449f4.jpeg" alt="Mapped and filtered pockets for the inhibitor site of Dihydroorotate dehydrogenase (PDB: 6OC0) visualized using (a) Fpocket alpha spheres representation; (b) grid representation." data-path-from-xml="btaf449f4.tif"><div class="graphic-bottom"><div class="label fig-label" id="label-529947926">Figure 4.</div><div class="caption fig-caption"><p class="chapter-para">Mapped
 and filtered pockets for the inhibitor site of Dihydroorotate 
dehydrogenase (PDB: 6OC0) visualized using (a) Fpocket alpha spheres 
representation; (b) grid representation.</p></div><div class="ajax-articleAbstract-exclude-regex fig-orig original-slide figure-button-wrap"><a class="fig-view-orig js-view-large at-figureViewLarge openInAnotherWindow" role="button" aria-describedby="label-529947926" href="https://academic.oup.com/view-large/figure/529947926/btaf449f4.tif" data-path-from-xml="btaf449f4.tif" target="_blank" data-google-interstitial="false">Open in new tab</a><a class="download-slide" role="button" aria-describedby="label-529947926" data-section="529947926" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449f4.jpeg?Expires=1765920142&amp;Signature=ebYB5AqT4VKpyr2t4sOGxvO4pBsUZyOfW3Mt4tFUr3KgcSvao-Pz9ULBfW5sVM~v9GC1gcAnjI36z3PashcitmOwkiCFiURtbkoPbZC7NjpfcWy~3NuOeTIS3feW0mxnWig4pANB5SES2fTqx0pTgkwvtKFgAbjBfsK4etjh2c1U~BvdZ5NKOVsfu6w0xrSOcJyzxw9xhirj0iuc3I-6wJ22eMTq7zPdMyctLg5uDlOvlURBkva0ckv2MFPbmMNu0N-MPLjJDND6T2hLSlRl3OKoV~YEpgjZ3j8cXvbnD~KEPwlxX2MzAurpW1OUecDvjjyRGjiSBIRSb6dwd109Jg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=529947926&amp;ar=8225722&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5139" data-path-from-xml="btaf449f4.tif" data-google-interstitial="false">Download slide</a></div></div></div></div><p class="chapter-para">We
 define a rectangular grid with 1.5 Å steps whose boundaries correspond 
to the minimum and maximum coordinates of the binding site heavy atoms 
along each axis. Only grid points located inside the alpha spheres are 
retained.</p><p class="chapter-para">The alpha spheres may protrude 
significantly from the protein cavities into the bulk solvent. The grid 
representations of such spheres overestimate the pocket volume and may 
lead to artifacts if this volume is used for molecular docking by 
allowing much bulkier compounds that can actually fit into the pocket. 
To address this, we construct a convex hull on the pocket-heavy atoms 
and remove all grid points that fall outside it (<span class="xrefLink" id="jumplink-btaf449-F5"></span><a href="javascript:;" data-modal-source-id="btaf449-F5" class="link xref-fig" data-google-interstitial="false">Fig.&nbsp;5</a>).</p>                    <a id="529947929" scrollto-destination="529947929" data-google-interstitial="false"></a>
<div data-id="btaf449-f5" data-content-id="btaf449-f5" class="fig fig-section js-fig-section" swap-content-for-modal="true"><div class="graphic-wrap"><img class="content-image" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/m_btaf449f5.jpeg" alt="Filtering pocket grid points using a convex hull defined by pocket atoms. Grid points falling outside the hull boundary are discarded, while those inside are retained, resulting in a corrected pocket representation without protruding parts." data-path-from-xml="btaf449f5.tif"><div class="graphic-bottom"><div class="label fig-label" id="label-529947929">Figure 5.</div><div class="caption fig-caption"><p class="chapter-para">Filtering
 pocket grid points using a convex hull defined by pocket atoms. Grid 
points falling outside the hull boundary are discarded, while those 
inside are retained, resulting in a corrected pocket representation 
without protruding parts.</p></div><div class="ajax-articleAbstract-exclude-regex fig-orig original-slide figure-button-wrap"><a class="fig-view-orig js-view-large at-figureViewLarge openInAnotherWindow" role="button" aria-describedby="label-529947929" href="https://academic.oup.com/view-large/figure/529947929/btaf449f5.tif" data-path-from-xml="btaf449f5.tif" target="_blank" data-google-interstitial="false">Open in new tab</a><a class="download-slide" role="button" aria-describedby="label-529947929" data-section="529947929" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449f5.jpeg?Expires=1765920142&amp;Signature=xuCP0eOOL3nK756YRQ9FGu6yNozVTk4r57mfqJkklZDPeSEDMQoI3DYXQPF~VCiDAYymxvIbH8-9uK8kmEyKkUgRVt3REskK3TTyWKNgFcZBJeOZLzLiC3u~SmBVv40nqMu16wxUfRvzg7WQBjCiFkqfFTjFUiw2I4mi-7BBHeHVU~uOzetnZvFB-XDUm3ME5NDVGET2iKivvf-pYmDFU8xUeAD3zw0xuxaemZdRjH5y5gxZ14OkYbIPe43m8uhPbJG5-5cEeOs34-c5dcJ7ALcApZ1H9Bn5JLFPr8todUBgj~liRznT~WtHtlJlbINxk1e3pXaZzYga6DpvHyUPTA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=529947929&amp;ar=8225722&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5139" data-path-from-xml="btaf449f5.tif" data-google-interstitial="false">Download slide</a></div></div></div></div><p class="chapter-para">The
 resulting grid is further refined by deleting the points located closer
 than the Van der Waals radius to any protein atom. This step is not 
strictly necessary since the docking algorithms prevent steric clashes 
anyway, but it improves the visual quality of the pocket representation.</p><p class="chapter-para">All parameter values are available in the <span class="link link-data-supplement" data-supplement-target="sup1"></span><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449_supplementary_data.docx?Expires=1765920142&amp;Signature=rEHUSLvtUag7CZgpA9SGOxJ22UbvVQjJfPsA754a1iM6fSlYc~EIL6CPTQNJcIr4MKIRsSZJfrTeH9vBceKPeJd9cLR7sFxMwDWxcb4AoE9ycXowG8kGsPfL7MxGKbNMq3BhUqeZ5yAegDkDlpt~Sgz99BsTXEQNTTXfp92NnDVmsDzU0~ZZzUTljE0upu1WiC6mLpIjFz1wevcqcegeWPG~af35y1WuxadfNRfE8KCLwVbxuEPSwqSsg7FdavfgAGK~T8BOMb~tkiqweo4QGe3Rd0DzUavpRzkbmowY1EzF~82U8hnMdZq62UpcfWyxvcU76wnGiMTsdKv~WhYx-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" data-google-interstitial="false">Supplementary Information</a></span>, available as <span class="link link-data-supplement" data-supplement-target="sup1"></span><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449_supplementary_data.docx?Expires=1765920142&amp;Signature=rEHUSLvtUag7CZgpA9SGOxJ22UbvVQjJfPsA754a1iM6fSlYc~EIL6CPTQNJcIr4MKIRsSZJfrTeH9vBceKPeJd9cLR7sFxMwDWxcb4AoE9ycXowG8kGsPfL7MxGKbNMq3BhUqeZ5yAegDkDlpt~Sgz99BsTXEQNTTXfp92NnDVmsDzU0~ZZzUTljE0upu1WiC6mLpIjFz1wevcqcegeWPG~af35y1WuxadfNRfE8KCLwVbxuEPSwqSsg7FdavfgAGK~T8BOMb~tkiqweo4QGe3Rd0DzUavpRzkbmowY1EzF~82U8hnMdZq62UpcfWyxvcU76wnGiMTsdKv~WhYx-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" data-google-interstitial="false">supplementary data</a></span> at <em>Bioinformatics</em> online.</p>                    <h2 scrollto-destination="529947932" id="529947932" class="section-title js-splitscreen-section-title">3 Results</h2>
                    <h3 scrollto-destination="529947933" id="529947933" class="section-title js-splitscreen-section-title">3.1 LLM pipeline performance</h3>
<p class="chapter-para">Due to the rapid advancement of LLMs, our 
development and testing focused exclusively on the model versions at the
 time of writing, and results are reported accordingly. Our goal is to 
show that a reliable pocket prioritization pipeline can be built with 
the LLM technologies at the time of writing. All steps of the LLM 
pipeline are performed by the GPT-4o-mini model.</p><p class="chapter-para">The dataset was initially evaluated by the baseline set of prompts (see <span class="link link-data-supplement" data-supplement-target="sup1"></span><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449_supplementary_data.docx?Expires=1765920142&amp;Signature=rEHUSLvtUag7CZgpA9SGOxJ22UbvVQjJfPsA754a1iM6fSlYc~EIL6CPTQNJcIr4MKIRsSZJfrTeH9vBceKPeJd9cLR7sFxMwDWxcb4AoE9ycXowG8kGsPfL7MxGKbNMq3BhUqeZ5yAegDkDlpt~Sgz99BsTXEQNTTXfp92NnDVmsDzU0~ZZzUTljE0upu1WiC6mLpIjFz1wevcqcegeWPG~af35y1WuxadfNRfE8KCLwVbxuEPSwqSsg7FdavfgAGK~T8BOMb~tkiqweo4QGe3Rd0DzUavpRzkbmowY1EzF~82U8hnMdZq62UpcfWyxvcU76wnGiMTsdKv~WhYx-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" data-google-interstitial="false">Supplementary Information</a></span>, available as <span class="link link-data-supplement" data-supplement-target="sup1"></span><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449_supplementary_data.docx?Expires=1765920142&amp;Signature=rEHUSLvtUag7CZgpA9SGOxJ22UbvVQjJfPsA754a1iM6fSlYc~EIL6CPTQNJcIr4MKIRsSZJfrTeH9vBceKPeJd9cLR7sFxMwDWxcb4AoE9ycXowG8kGsPfL7MxGKbNMq3BhUqeZ5yAegDkDlpt~Sgz99BsTXEQNTTXfp92NnDVmsDzU0~ZZzUTljE0upu1WiC6mLpIjFz1wevcqcegeWPG~af35y1WuxadfNRfE8KCLwVbxuEPSwqSsg7FdavfgAGK~T8BOMb~tkiqweo4QGe3Rd0DzUavpRzkbmowY1EzF~82U8hnMdZq62UpcfWyxvcU76wnGiMTsdKv~WhYx-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" data-google-interstitial="false">supplementary data</a></span> at <em>Bioinformatics</em> online). Then, the prompts for filtering and extraction stages were optimized using the following strategies:</p><ul class="bullet"><li><p class="chapter-para">Adding
 explicit instructions to ensure that LLM identifies only binding sites 
relevant to small molecules, while actively excluding other functional 
regions such as protein-protein interaction (PPI) sites and areas 
involved in DNA/RNA binding.</p></li><li><p class="chapter-para">Adding directives to only include amino acids explicitly involved in pocket formation or ligand interaction.</p></li><li><p class="chapter-para">Detailed
 specifications of the response format and quality stand-ards, including
 guidelines that strongly reward correct residue reporting while 
penalizing hallucinated or inaccurate entries.</p></li><li><p class="chapter-para">Enforcement
 of strict formatting rules during residue extraction, ensuring the 
correct parsing and consistent representation of chain identifiers, 
residue names, and residue IDs.</p></li><li><p class="chapter-para">Enforcement
 of strict formatting rules during residue extraction, ensuring the 
correct parsing and consistent representation of chain identifiers, 
residue names, and residue IDs.</p></li></ul><p class="chapter-para">These
 changes were implemented sequentially, with monitoring performance 
following each prompt adjustment. Adjustments were only retained if they
 improved the metrics and/or correctness of output formatting. The final
 prompts are available in the <span class="link link-data-supplement" data-supplement-target="sup1"></span><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449_supplementary_data.docx?Expires=1765920142&amp;Signature=rEHUSLvtUag7CZgpA9SGOxJ22UbvVQjJfPsA754a1iM6fSlYc~EIL6CPTQNJcIr4MKIRsSZJfrTeH9vBceKPeJd9cLR7sFxMwDWxcb4AoE9ycXowG8kGsPfL7MxGKbNMq3BhUqeZ5yAegDkDlpt~Sgz99BsTXEQNTTXfp92NnDVmsDzU0~ZZzUTljE0upu1WiC6mLpIjFz1wevcqcegeWPG~af35y1WuxadfNRfE8KCLwVbxuEPSwqSsg7FdavfgAGK~T8BOMb~tkiqweo4QGe3Rd0DzUavpRzkbmowY1EzF~82U8hnMdZq62UpcfWyxvcU76wnGiMTsdKv~WhYx-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" data-google-interstitial="false">Supplementary Information</a></span>, available as <span class="link link-data-supplement" data-supplement-target="sup1"></span><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449_supplementary_data.docx?Expires=1765920142&amp;Signature=rEHUSLvtUag7CZgpA9SGOxJ22UbvVQjJfPsA754a1iM6fSlYc~EIL6CPTQNJcIr4MKIRsSZJfrTeH9vBceKPeJd9cLR7sFxMwDWxcb4AoE9ycXowG8kGsPfL7MxGKbNMq3BhUqeZ5yAegDkDlpt~Sgz99BsTXEQNTTXfp92NnDVmsDzU0~ZZzUTljE0upu1WiC6mLpIjFz1wevcqcegeWPG~af35y1WuxadfNRfE8KCLwVbxuEPSwqSsg7FdavfgAGK~T8BOMb~tkiqweo4QGe3Rd0DzUavpRzkbmowY1EzF~82U8hnMdZq62UpcfWyxvcU76wnGiMTsdKv~WhYx-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" data-google-interstitial="false">supplementary data</a></span> at <em>Bioinformatics</em> online.</p><p class="chapter-para">The
 initial Baseline configuration utilized non-optimized prompts for 
extraction and filtering. This yielded a Pocket Number Accuracy of 
0.484, Pocket Recall of 0.958, and Pocket Specificity of 0.46 (<span class="xrefLink" id="jumplink-btaf449-F6"></span><a href="javascript:;" data-modal-source-id="btaf449-F6" class="link xref-fig" data-google-interstitial="false">Fig.&nbsp;6</a>).</p>                    <a id="529947939" scrollto-destination="529947939" data-google-interstitial="false"></a>
<div data-id="btaf449-f6" data-content-id="btaf449-f6" class="fig fig-section js-fig-section" swap-content-for-modal="true"><div class="graphic-wrap"><img class="content-image" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/m_btaf449f6.jpeg" alt="GPT-4o-mini performance metrics for three pipeline configurations: Baseline (filter and extraction steps with initial prompts), Optimized (filter and extraction steps with optimized prompts), and Full pipeline (adding a refinement step to the Optimized configuration)." data-path-from-xml="btaf449f6.tif"><div class="graphic-bottom"><div class="label fig-label" id="label-529947939">Figure 6.</div><div class="caption fig-caption"><p class="chapter-para">GPT-4o-
mini performance metrics for three pipeline configurations: Baseline 
(filter and extraction steps with initial prompts), Optimized (filter 
and extraction steps with optimized prompts), and Full pipeline (adding a
 refinement step to the Optimized configuration).</p></div><div class="ajax-articleAbstract-exclude-regex fig-orig original-slide figure-button-wrap"><a class="fig-view-orig js-view-large at-figureViewLarge openInAnotherWindow" role="button" aria-describedby="label-529947939" href="https://academic.oup.com/view-large/figure/529947939/btaf449f6.tif" data-path-from-xml="btaf449f6.tif" target="_blank" data-google-interstitial="false">Open in new tab</a><a class="download-slide" role="button" aria-describedby="label-529947939" data-section="529947939" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449f6.jpeg?Expires=1765920142&amp;Signature=aSsIP89buCsOd72tMdAXtdPbm49QrYHteD14pC8R1Ph2b2hvac43d~Xxc6app5aLWkItw6S48lKTbCapSKL1iU5aySfY-ssy32z7QHO4NQDR3ZBADK4duCcRbi-ieYsyn~WKcMcQcw04yGZOv-Iidh15b5y-AYRAm4tOMKv3osLvQVqG47Sc~lHvYIQe-gmpBF2RubJ18wB6OGmxRmYiVRMNaY~vt6Igyoa~o4VDrluaLo8yMMcRRKm46XK-TrUoJFL1XMqlIQepw~77jxQKzafoQh4bwjCmb2QcyGYCSzTRfE5M29rF9qUuB7jlTugVBIOH14IxHFU9vHjah-XJxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=529947939&amp;ar=8225722&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5139" data-path-from-xml="btaf449f6.tif" data-google-interstitial="false">Download slide</a></div></div></div></div><p class="chapter-para">Optimization
 of the extraction step (while keeping the filtering unchanged) boosted 
performance significantly. It resulted in a Pocket Number Accuracy of 
0.581, perfect Pocket Recall (1.0), and Pocket Specificity of 0.558. 
However, manual inspection revealed that this configuration sometimes 
incorrectly split single pockets into multiple entries.</p><p class="chapter-para">The
 addition of the Refinement step addressed this issue, while further 
improving Pocket Number Accuracy to 0.71 and Pocket Specificity to 0.657
 and maintaining perfect Pocket Recall. This final step also enhanced 
residue-level metrics, as the refinement process often incorporates 
missing amino acids.</p><p class="chapter-para">The filtering step was 
also evaluated separately as a binary classification task to determine 
paper relevance based on the presence of binding site descriptions. The 
model achieved an accuracy of 0.87, a recall of 1.0, a precision of 
0.833, and a False Positive Rate (FPR) of 0.363 while retaining FNR at 
0. The perfect recall indicates the model effectively identifies all of 
the relevant papers. However, the precision and FPR suggest that 
approximately 36% of papers flagged as relevant are false positives that
 do not get filtered out at this stage. We considered this acceptable 
because the inclusion of potentially irrelevant papers is unlikely to 
predict fake pockets that are hard to spot on post-processing.</p>                    <h3 scrollto-destination="529947943" id="529947943" class="section-title js-splitscreen-section-title">3.2 Volumetric pocket representations</h3>
<p class="chapter-para">We visually assessed the residue mapping 
algorithm’s performance for each target structure, with the quality of 
the final mapping assessed manually by a human expert. This assessment 
focused primarily on ensuring correct residue assignments across all 
relevant PDB chains, as these would influence the pocket’s volumetric 
representation. To achieve optimal performance across the whole dataset,
 we empirically set the threshold to 0.6 for chain matching and 0.7 for 
cluster matching. <span class="xrefLink" id="jumplink-btaf449-F7"></span><a href="javascript:;" data-modal-source-id="btaf449-F7" class="link xref-fig" data-google-interstitial="false">Figure&nbsp;7</a>
 shows mapped residues for the GABAA receptor PDB structure, where the 
residue mapping algorithm identified two equivalent binding pockets at 
distinct subunit interfaces. The output includes atom coordinates for 
the pocket’s constituent residues along with the corresponding pocket 
name and description extracted by the LLM (<span class="xrefLink" id="jumplink-btaf449-B12"></span><a href="javascript:;" reveal-id="btaf449-B12" data-open="btaf449-B12" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Kim <em>et al.</em> 2020</a>).</p>                    <a id="529947945" scrollto-destination="529947945" data-google-interstitial="false"></a>
<div data-id="btaf449-f7" data-content-id="btaf449-f7" class="fig fig-section js-fig-section" swap-content-for-modal="true"><div class="graphic-wrap"><img class="content-image" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/m_btaf449f7.jpeg" alt="The proposed algorithm’s residue mapping shows equivalent etomidate binding pockets in the GABAA receptor (PDB: 6X3T) at (a) the A:B interface and (b) the C:D interface." data-path-from-xml="btaf449f7.tif"><div class="graphic-bottom"><div class="label fig-label" id="label-529947945">Figure 7.</div><div class="caption fig-caption"><p class="chapter-para">The
 proposed algorithm’s residue mapping shows equivalent etomidate binding
 pockets in the GABAA receptor (PDB: 6X3T) at (a) the A:B interface and 
(b) the C:D interface.</p></div><div class="ajax-articleAbstract-exclude-regex fig-orig original-slide figure-button-wrap"><a class="fig-view-orig js-view-large at-figureViewLarge openInAnotherWindow" role="button" aria-describedby="label-529947945" href="https://academic.oup.com/view-large/figure/529947945/btaf449f7.tif" data-path-from-xml="btaf449f7.tif" target="_blank" data-google-interstitial="false">Open in new tab</a><a class="download-slide" role="button" aria-describedby="label-529947945" data-section="529947945" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449f7.jpeg?Expires=1765920142&amp;Signature=NCEIMeFjOinju-~-3ofVeJYEG1AsNCh6x4bwahPqoyaTQ7uoZydR~5lk~uhujFfk9TeSrW4WzIl3lG5Y0fVma4oOMBLiHNtdgyCt3MHpZ~aymIPT1kgSb~V8qjad5Ha2O8Jrlhp~zr9LH7ys7CK7GwX2H~czJ2Cv44dwuTydsXVgl5SMinS3p-ZXsiU~VJ7NSFVO7RSXbxqkjL38jKmuUs737ixsvivaNj5WL~Ert~mvJMYptzo0-bh4mdNXggnsqY-Bf1JmIZpizr8qiVZQZpHbaE3O6UTq7Js8na8wGHGgIt57BHetsnw32Xt9MGknaNwVu3nMSAER9dIkqgZpjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=529947945&amp;ar=8225722&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5139" data-path-from-xml="btaf449f7.tif" data-google-interstitial="false">Download slide</a></div></div></div></div><p class="chapter-para">We
 generated initial geometric pocket representations using Fpocket with 
default parameters. However, because Fpocket sometimes fragments larger 
sites, a post-processing step is implemented to merge spatially adjacent
 pockets meeting a predefined empirical residue overlap threshold. 
Kinase structures frequently necessitate such merging, as Fpocket often 
splits its ATP-binding sites into two or more smaller subpockets that 
must be combined to accurately represent the complete catalytic site (<span class="xrefLink" id="jumplink-btaf449-F8"></span><a href="javascript:;" data-modal-source-id="btaf449-F8" class="link xref-fig" data-google-interstitial="false">Fig.&nbsp;8</a>) (<span class="xrefLink" id="jumplink-btaf449-B18"></span><a href="javascript:;" reveal-id="btaf449-B18" data-open="btaf449-B18" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Murphy <em>et al.</em> 2014</a>).</p>                    <a id="529947947" scrollto-destination="529947947" data-google-interstitial="false"></a>
<div data-id="btaf449-f8" data-content-id="btaf449-f8" class="fig fig-section js-fig-section" swap-content-for-modal="true"><div class="graphic-wrap"><img class="content-image" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/m_btaf449f8.jpeg" alt="Mixed lineage kinase domain-like pseudokinase (PDB: 7MON) with (a) selected pockets as alpha spheres and (b) the final pocket after merging and post-processing." data-path-from-xml="btaf449f8.tif"><div class="graphic-bottom"><div class="label fig-label" id="label-529947947">Figure 8.</div><div class="caption fig-caption"><p class="chapter-para">Mixed
 lineage kinase domain-like pseudokinase (PDB: 7MON) with (a) selected 
pockets as alpha spheres and (b) the final pocket after merging and 
post-processing.</p></div><div class="ajax-articleAbstract-exclude-regex fig-orig original-slide figure-button-wrap"><a class="fig-view-orig js-view-large at-figureViewLarge openInAnotherWindow" role="button" aria-describedby="label-529947947" href="https://academic.oup.com/view-large/figure/529947947/btaf449f8.tif" data-path-from-xml="btaf449f8.tif" target="_blank" data-google-interstitial="false">Open in new tab</a><a class="download-slide" role="button" aria-describedby="label-529947947" data-section="529947947" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449f8.jpeg?Expires=1765920142&amp;Signature=41613WBchQSSBuOMEGtKk5P3DVro-zVOt1YMDBGHwihjEe6SFvYUEbsFr9h45pYTcV-3wcEFvEs7l2BF0bssqEMXd1mvuTbYuNMXyijoim4cHdUMxSivg7AXWYCdR5KnkzSYpS0C4HWTVBR5135A0OA25CfuK4HTXNb5CId7trrbCaloTmloixT914PC8xxYiliFdRyRx8cG2BQBZ-J6eNsz3deADPlc532Pmwo8O~KXuGatHUCrY5DnQWvf8Wn3zVJcYNq5BMQ4ToWcgcKJXBEo8nCH8yps6jYuV989QVKq8ei5rawYgapx8R9JDa7C7uhtWriRWutOrF5qyYt9Gw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=529947947&amp;ar=8225722&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5139" data-path-from-xml="btaf449f8.tif" data-google-interstitial="false">Download slide</a></div></div></div></div><p class="chapter-para">The
 same merging strategy addresses another challenge: consolidating 
information from multiple papers. Pockets identified from different 
sources might describe the same physical site, but often list different 
constituent residues. The sequence-level comparisons could be attempted;
 however, merging based on the spatial proximity of the pockets is a 
more robust approach for identifying and unifying equivalent sites 
across various publications.</p><p class="chapter-para">This merging 
process, however, cannot address the cases where Fpocket initially fails
 to separate distinct binding sites and outputs them incorrectly as a 
large single pocket. This limitation was observed with the M2 receptor (<span class="xrefLink" id="jumplink-btaf449-B14"></span><a href="javascript:;" reveal-id="btaf449-B14" data-open="btaf449-B14" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Kruse <em>et al.</em> 2014</a>), where Fpocket did not distinguish between known separate binding sites (<span class="xrefLink" id="jumplink-btaf449-F9"></span><a href="javascript:;" data-modal-source-id="btaf449-F9" class="link xref-fig" data-google-interstitial="false">Fig.&nbsp;9</a>).
 Despite these challenges, the residue mapping algorithm performed 
reliably for binding sites contained within a single chain or distinct 
protein domain.</p>                    <a id="529947950" scrollto-destination="529947950" data-google-interstitial="false"></a>
<div data-id="btaf449-f9" data-content-id="btaf449-f9" class="fig fig-section js-fig-section" swap-content-for-modal="true"><div class="graphic-wrap"><img class="content-image" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/m_btaf449f9.jpeg" alt="Muscarinic acetylcholine receptor M2 (PDB: 4MQT) with detected and mapped pocket that combines the orthosteric and extracellular vestibule binding sites." data-path-from-xml="btaf449f9.tif"><div class="graphic-bottom"><div class="label fig-label" id="label-529947950">Figure 9.</div><div class="caption fig-caption"><p class="chapter-para">Muscarinic
 acetylcholine receptor M2 (PDB: 4MQT) with detected and mapped pocket 
that combines the orthosteric and extracellular vestibule binding sites.</p></div><div class="ajax-articleAbstract-exclude-regex fig-orig original-slide figure-button-wrap"><a class="fig-view-orig js-view-large at-figureViewLarge openInAnotherWindow" role="button" aria-describedby="label-529947950" href="https://academic.oup.com/view-large/figure/529947950/btaf449f9.tif" data-path-from-xml="btaf449f9.tif" target="_blank" data-google-interstitial="false">Open in new tab</a><a class="download-slide" role="button" aria-describedby="label-529947950" data-section="529947950" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449f9.jpeg?Expires=1765920142&amp;Signature=m~LqDv4wtNfh-9jSbwHPi5bD7Bo7fa7dUSI8yFi4FBoF8InBVxw5LKgImvAZqu6tRyqXtldpIPGXuFYQfEz80VqQbSm6BSPgKl8nCC7MyfyoOFn5c3J~7az4DQxw6A~y4WWYfKCVth~FP-FvduTf~0~PcTzOnied4uR2o41cBOOXzsbhfjg8GaaTUIhglN5yhP3aY~5DQ2Q2OfNyyR6xl4pRzbTQTLW-nZZH9fbOKXBIodr~WlFNdI3FQep340Sdh7z6761yDGeSKVLG85JCjnuC~bfWyYKFjQVoG88oZJ05vbFyG0FgO-3KPsl~pfnBsKoLlByDNi3S-apkjHv7AQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=529947950&amp;ar=8225722&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5139" data-path-from-xml="btaf449f9.tif" data-google-interstitial="false">Download slide</a></div></div></div></div><p class="chapter-para">For
 multimeric assemblies, the merging algorithm only performs reliably 
assuming correct initial residue identification by the LLM. For 
instance, the human Nav1.7-VSD4-NavAb chimera (PDB: 5EK0) is a tetramer 
complex where each of the four chains incorporates a human NaV1.7 VSD4 
domain introduced via protein engineering. In this case, our algorithm 
correctly located four identical binding pockets—one within each of the 
VSD4 domains (<span class="xrefLink" id="jumplink-btaf449-F10"></span><a href="javascript:;" data-modal-source-id="btaf449-F10" class="link xref-fig" data-google-interstitial="false">Fig.&nbsp;10</a>) that correspond to the known deep binding site of the GX-936 warhead in the VSD4 extracellular cleft (<span class="xrefLink" id="jumplink-btaf449-B2"></span><a href="javascript:;" reveal-id="btaf449-B2" data-open="btaf449-B2" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Ahuja <em>et al.</em> 2015</a>).</p>                    <a id="529947952" scrollto-destination="529947952" data-google-interstitial="false"></a>
<div data-id="btaf449-f10" data-content-id="btaf449-f10" class="fig fig-section js-fig-section" swap-content-for-modal="true"><div class="graphic-wrap"><img class="content-image" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/m_btaf449f10.jpeg" alt="GX-936 binding pocket located at the VSD4 domain of the human NaV1.7 (PDB: 5EK0)." data-path-from-xml="btaf449f10.tif"><div class="graphic-bottom"><div class="label fig-label" id="label-529947952">Figure 10.</div><div class="caption fig-caption"><p class="chapter-para">GX-936 binding pocket located at the VSD4 domain of the human Na<sub>V</sub>1.7 (PDB: 5EK0).</p></div><div class="ajax-articleAbstract-exclude-regex fig-orig original-slide figure-button-wrap"><a class="fig-view-orig js-view-large at-figureViewLarge openInAnotherWindow" role="button" aria-describedby="label-529947952" href="https://academic.oup.com/view-large/figure/529947952/btaf449f10.tif" data-path-from-xml="btaf449f10.tif" target="_blank" data-google-interstitial="false">Open in new tab</a><a class="download-slide" role="button" aria-describedby="label-529947952" data-section="529947952" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449f10.jpeg?Expires=1765920142&amp;Signature=Zbr7RgxhcC6AaWKZUMgG-3-NfBUEqTbvmcd~48HysXUg4J6zIIKdl7D0NpareQ5UDLiX~YhvM1fyi5XT2YjZigZ1ZX2a6KEYzGL0x9X1FqXUCp3KIgwZB9fUcaBtrERbTxfVgn~OJfiTmp0k-gUBysMo2yt~UckoxJJ1maJ4THZIzCxViD4zrHQuxc7evXWdCky0ZNSan5KoraEQCq5o1fulB3gAQ4My~-dPMsW87WFfgmzuOhkkcZD1pcSwnq~Q8Ry~xhOlbDETb2eRE0gCFDfwJWMb6AK7SFdR1KfCFdOzl3IC53gOXiLOHUYCeJzwTR46FxXGl1MoNHEuhp6ntw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=529947952&amp;ar=8225722&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5139" data-path-from-xml="btaf449f10.tif" data-google-interstitial="false">Download slide</a></div></div></div></div>                    <h2 scrollto-destination="529947953" id="529947953" class="section-title js-splitscreen-section-title">4 Discussion</h2>
<p class="chapter-para">The identification and prioritization of protein
 binding pockets is a critical step in small-molecule drug discovery. 
While numerous computational methods exist, practical workflows often 
face challenges, particularly when focusing on biologically validated 
binding sites relevant for virtual screening or druggability assessment,
 rather than discovering entirely novel pockets. Selecting and defining 
these known sites typically involves a laborious process: manually 
analyzing multiple research papers to identify key residues, dealing 
with inconsistent terminologies and data presentation across 
publications, computing volumetric representations (which are rarely 
provided directly), and mapping the literature-derived residue 
information onto these geometric pockets to filter for biological 
relevance. This manual workflow is time-consuming and prone to errors, 
especially in high-throughput industrial settings.</p><p class="chapter-para">Although databases such as BioLip2 (<span class="xrefLink" id="jumplink-btaf449-B25"></span><a href="javascript:;" reveal-id="btaf449-B25" data-open="btaf449-B25" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Zhang <em>et al.</em> 2024</a>) and Q-BioLip (<span class="xrefLink" id="jumplink-btaf449-B20"></span><a href="javascript:;" reveal-id="btaf449-B20" data-open="btaf449-B20" class="link link-ref link-reveal xref-bibr" data-google-interstitial="false">Wei <em>et al.</em> 2024</a>)
 provide a convenient interface for exploring the binding pockets of 
protein structures in PDB, they mark the residues as belonging to the 
pocket by using a distance to the crystallized ligand. This rough 
approximation does not take into account transient dynamic contacts and 
misses the residues of larger pockets crystallized with small ligands 
inside. It is also not applicable if no co-crystallized ligands are 
available. Furthermore, BioLip only uses literature information from the
 PubMed abstracts to confirm the biological relevance of the ligand 
itself without parsing any information about the pocket residues. In 
contrast, we aim to identify the binding pockets by collecting 
information about their residues from the literature, regardless of the 
availability of co-crystallized ligands. This includes information 
obtained by such techniques as mutational scanning or fluorescent 
probing and aggregates information for multiple ligands of different 
sizes and chemical classes, which is not possible when relying on PDB 
structures only. That is why our approach is potentially more robust and
 covers a large set of target proteins without crystallized ligands, 
which occur routinely in practical drug discovery scenarios.</p><p class="chapter-para">The
 integration of our technique with BioLip is possible and may be useful 
for validating residues extracted by LLM and retrieving activity 
constants or other cross-referenced data present in the database. The 
general approach, in this case, would be to use the BioLip binding 
pocket as a starting point and to extend it with information about the 
residues extracted from the papers. Although such a combined approach 
might be of interest in certain cases, it is beyond the scope of the 
current work.</p><p class="chapter-para">This article investigates the 
potential of automating this workflow using a hybrid approach combining 
geometry-based pocket detection with LLMs for literature analysis and 
pocket validation. We aimed to determine if current LLM technology can 
effectively substitute human researchers in the binding pocket 
validation and prioritization tasks outlined above. Our proposed 
pipeline integrates Fpocket for initial geometric pocket prediction with
 an LLM (specifically, GPT-4o-mini) for extracting and validating 
binding site information from scientific literature, followed by 
algorithmic mapping, merging, and refinement steps to produce accurate 
volumetric pocket representations.</p><p class="chapter-para">A direct 
comparison with human researchers is not feasible because we 
concentrated on the time and accuracy of LLM-based pocket extraction and
 mapping. While a competent researcher can achieve near-perfect results 
with sufficient time (typically on the scale of hours), our automated 
pipeline produces its results in seconds or minutes (depending on the 
number and complexity of articles provided). However, its accuracy is 
not perfect (<span class="xrefLink" id="jumplink-btaf449-F6"></span><a href="javascript:;" data-modal-source-id="btaf449-F6" class="link xref-fig" data-google-interstitial="false">Fig.&nbsp;6</a>).
 Human post-processing of the results is likely to eliminate most of the
 mistakes while taking much less time than the complete human-based 
pipeline. Reliably quantifying this would require a large cohort of 
human experts and a much larger dataset, which is beyond the scope of 
this work.</p><p class="chapter-para">While the automated parsing of 
complex scientific PDFs remains a non-trivial task, we used a robust 
semi-automated workflow to mitigate this issue. We utilized the GROBID 
tool, which reliably extracts structured text, including figure and 
table captions, from research articles. This was followed by a light 
manual curation step focused on standardizing formatting and removing 
non-essential sections. Although this process minimizes most parsing 
artifacts, the model’s performance is inherently dependent on the 
quality of the text extraction. A key remaining challenge, common to 
text-based pipelines, is the inability to directly interpret data from 
within tables or graphical elements themselves, a task that is beyond 
the scope of this work.</p><p class="chapter-para">Our results indicate 
that the LLM pipeline, particularly after prompt optimization and the 
inclusion of a refinement step, can effectively identify relevant papers
 and extract pocket information with high accuracy. The refinement stage
 proved valuable in correcting initial extraction errors, such as 
improper pocket splitting or missing residues. However, some challenges 
remain: the LLM filtering step still passes some irrelevant papers 
through, and accurately interpreting the nuances of binding site 
descriptions spread across varied writing styles and sections within 
papers requires careful handling. Explicit instructions were needed to 
ensure that LLM focused only on small-molecule binding sites.</p><p class="chapter-para">Despite
 its limited number of articles describing multiple binding pockets, our
 dataset allows us to capture the major hurdles that arise when 
processing complex biological texts, such as distinguishing between 
multiple binding sites, correctly associating specific residue lists 
with the correct pocket, and resolving inconsistent terminology used to 
describe the same site. Although increasing the number of papers is 
likely to reveal some other corner cases, we did not aim to find and 
address all of them in this paper.</p><p class="chapter-para">The 
subsequent algorithmic mapping and clustering successfully bridged the 
gap between literature descriptions and specific PDB structures, 
handling inconsistencies like sequence differences, numbering errors, 
and multimeric interfaces. This step demonstrated its utility in 
identifying equivalent pockets across different subunits, as seen in the
 GABAA receptor and Nav1.7 chimera examples. Nevertheless, its accuracy 
depends on the initial LLM extraction quality and the structural 
similarity between the PDB file and the experimental construct 
described. Multimeric proteins remain challenging in this regard due to 
possible confusion of residue attribution between the chains and the 
positioning of the pockets between multiple subunits.</p><p class="chapter-para">Generating
 the final volumetric representation involves filtering Fpocket’s 
geometric predictions using LLM-validated residues. Converting to a grid
 representation and subsequent post-processing, including merging 
fragmented pockets and applying convex hull truncation, significantly 
improved the final pocket definition. This merging also effectively 
consolidated information from multiple publications. Still, the pipeline
 inherits Fpocket’s limitations, such as its occasional inability to 
separate closely adjacent sites, as observed with the M2 receptor.</p><p class="chapter-para">Despite
 existing limitations, including the reliance on the specific LLM used 
(GPT-4o-mini) and the limited benchmark dataset, our hybrid approach 
shows considerable promise. Future work could involve exploring more 
advanced LLM techniques, integrating alternative geometric predictors, 
expanding the benchmark dataset, and developing methods to automatically
 resolve structural discrepancies. Overall, leveraging LLMs to automate 
the analysis of research literature for binding pocket identification 
appears to be a valuable direction, potentially streamlining the drug 
discovery workflows.</p>                    <h2 scrollto-destination="529947965" id="529947965" class="section-title js-splitscreen-section-title">5 Conclusion</h2>
<p class="chapter-para">This study introduces a practical hybrid 
approach for identifying and prioritizing protein binding pockets. By 
integrating geometric detection with LLM-based evidence extraction, our 
pipeline is designed to assist researchers by automating the most 
laborious steps of literature analysis and pocket validation. This 
method effectively reduces the manual burden in high-throughput drug 
discovery scenarios, enabling experts to focus on the most promising, 
literature-supported pockets. While human oversight remains essential 
for complex cases, this work represents a significant step toward more 
efficient, semi-automated pocket identification and prioritization 
workflows. The benchmark dataset is made publicly available to further 
support future advancements in this field.</p>                    <h2 scrollto-destination="529947967" id="529947967" class="backsection-title js-splitscreen-backsection-title">Author contributions</h2>
<p class="chapter-para">Roman Stratiichuk (Data curation [equal], Formal
 analysis [equal], Validation [equal]), Mykola Melnychenko (Data 
curation [equal], Formal analysis [equal], Investigation [lead], 
Software [lead], Writing—original draft [equal]), Ihor Koleiev (Data 
curation [equal], Validation [equal]), Taras Voitsitskyi (Data curation 
[equal], Validation [equal]), Vladyslav Husak (Data curation [equal], 
Validation [equal]), Nazar Shevchuk (Data curation [equal], Validation 
[equal]), Zakhar Ostrovsky (Data curation [equal], Software [equal]), 
Volodymyr Bdzhola (Data curation [supporting], Validation [supporting]),
 Semen Yesylevskyy (Formal analysis [equal], Methodology [equal], 
Project administration [equal], Supervision [equal], Writing—original 
draft [equal]), Serhii Starosyla (Conceptualization [equal], Project 
administration [equal], Supervision [equal]), and Alan Nafiiev 
(Conceptualization [lead], Project administration [lead], Supervision 
[equal])</p>                    <h2 scrollto-destination="529947969" id="529947969" class="backsection-title js-splitscreen-backsection-title">Supplementary data</h2>
<p class="chapter-para"><span class="link link-data-supplement" data-supplement-target="sup1"></span><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449_supplementary_data.docx?Expires=1765920142&amp;Signature=rEHUSLvtUag7CZgpA9SGOxJ22UbvVQjJfPsA754a1iM6fSlYc~EIL6CPTQNJcIr4MKIRsSZJfrTeH9vBceKPeJd9cLR7sFxMwDWxcb4AoE9ycXowG8kGsPfL7MxGKbNMq3BhUqeZ5yAegDkDlpt~Sgz99BsTXEQNTTXfp92NnDVmsDzU0~ZZzUTljE0upu1WiC6mLpIjFz1wevcqcegeWPG~af35y1WuxadfNRfE8KCLwVbxuEPSwqSsg7FdavfgAGK~T8BOMb~tkiqweo4QGe3Rd0DzUavpRzkbmowY1EzF~82U8hnMdZq62UpcfWyxvcU76wnGiMTsdKv~WhYx-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" data-google-interstitial="false">Supplementary data</a></span> are available at <em>Bioinformatics</em> online.</p><p class="chapter-para">Conflict of interest: All authors are employees of Receptor. AI INC. S.S., A.N., and S.Y. have shares in Receptor.AI INC.</p>                    <h2 scrollto-destination="529947972" id="529947972" class="backsection-title js-splitscreen-backsection-title">Funding</h2>
<p class="chapter-para">S.Y. has received funding through the grant 
MSMT-355/2025-16 from the Ministry of Education, Youth and Sport of the 
Czech Republic and through the MSCA4Ukraine project 101101923, which is 
funded by the European Union. Views and opinions expressed are, however,
 those of the author(s) only and do not necessarily reflect those of the
 European Union. Neither the European Union nor the MSCA4Ukraine 
Consortium as a whole nor any individual member institutions of the 
MSCA4Ukraine Consortium can be held responsible for them.</p>                    <h2 scrollto-destination="529947974" id="529947974" class="dataavailabilitystatement-title js-splitscreen-dataavailabilitystatement-title">Data availability</h2>
<p class="chapter-para">The data underlying this article are available at the GitHub repository (<a class="link link-uri openInAnotherWindow" href="https://github.com/receptor-ai/LLM-benchmark-dataset" target="_blank" data-google-interstitial="false">https://github.com/receptor-ai/LLM-benchmark-dataset</a>) and Zenodo (DOI: <a class="link link-uri openInAnotherWindow" href="https://doi.org/10.5281/zenodo.15798647" target="_blank" data-google-interstitial="false">https://doi.org/10.5281/zenodo.15798647</a>).</p>                    <h2 scrollto-destination="529947976" id="529947976" class="backreferences-title js-splitscreen-backreferences-title" data-legacy-id="ref1">References</h2>
<div class="ref-list js-splitview-ref-list"><div content-id="btaf449-B1" class="js-splitview-ref-item" data-legacy-id="btaf449-B1"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B1" href="javascript:;" aria-label="jumplink-btaf449-B1" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B1" class="ref-content " data-id="btaf449-B1"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Aggarwal</div> <div class="given-names">R</div>, <span class="name string-name"><div class="surname">Gupta</div> <div class="given-names">A</div></span>, <span class="name string-name"><div class="surname">Chelur</div> <div class="given-names">V</div></span> &nbsp;et al.  &nbsp;<div class="article-title">DeepPocket: ligand binding site detection and segmentation using 3D convolutional neural networks</div>. <div class="source ">J Chem Inf Model</div> &nbsp;<div class="year">2022</div>;<div class="volume">62</div>:<div class="fpage">5069</div>–<div class="lpage">79</div>. <div class="pub-id"><a href="https://doi.org/10.1021/acs.jcim.1c00799" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1021/acs.jcim.1c00799</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=DeepPocket%3A%20ligand%20binding%20site%20detection%20and%20segmentation%20using%203D%20convolutional%20neural%20networks&amp;author=R%20Aggarwal&amp;author=A%20Gupta&amp;author=V%20Chelur&amp;publication_year=2022&amp;journal=J%20Chem%20Inf%20Model&amp;volume=62&amp;pages=5069-79" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1021/acs.jcim.1c00799" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Facs.jcim.1c00799" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1021%2Facs.jcim.1c00799"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/34374539" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:DeepPocket%3A%20ligand%20binding%20site%20detection%20and%20segmentation%20using%203D%20convolutional%20neural%20networks&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B2" class="js-splitview-ref-item" data-legacy-id="btaf449-B2"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B2" href="javascript:;" aria-label="jumplink-btaf449-B2" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B2" class="ref-content " data-id="btaf449-B2"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Ahuja</div> <div class="given-names">S</div>, <span class="name string-name"><div class="surname">Mukund</div> <div class="given-names">S</div></span>, <span class="name string-name"><div class="surname">Deng</div> <div class="given-names">L</div></span> &nbsp;et al.  &nbsp;<div class="article-title">Structural basis of Nav1.7 inhibition by an isoform-selective small-molecule antagonist</div>. <div class="source ">Science</div> &nbsp;<div class="year">2015</div>;<div class="volume">350</div>:<div class="fpage">aac5464</div>. <div class="pub-id"><a href="https://doi.org/10.1126/science.aac5464" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1126/science.aac5464</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Structural%20basis%20of%20Nav1.7%20inhibition%20by%20an%20isoform-selective%20small-molecule%20antagonist&amp;author=S%20Ahuja&amp;author=S%20Mukund&amp;author=L%20Deng&amp;publication_year=2015&amp;journal=Science&amp;volume=350&amp;pages=aac5464" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1126/science.aac5464" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.aac5464" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.aac5464"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26680203" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Structural%20basis%20of%20Nav1.7%20inhibition%20by%20an%20isoform-selective%20small-molecule%20antagonist&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B4" class="js-splitview-ref-item" data-legacy-id="btaf449-B4"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B4" href="javascript:;" aria-label="jumplink-btaf449-B4" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B4" class="ref-content " data-id="btaf449-B4"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">An</div> <div class="given-names">Y</div>, <span class="name string-name"><div class="surname">Lim</div> <div class="given-names">J</div></span>, <span class="name string-name"><div class="surname">Glavatskikh</div> <div class="given-names">M</div></span> &nbsp;et al.  &nbsp;<div class="article-title">In silico fragment-based discovery of CIB1-directed anti-tumor agents by FRASE-Bot</div>. <div class="source ">Nat Commun</div> &nbsp;<div class="year">2024</div>;<div class="volume">15</div>:<div class="fpage">5564</div>. <div class="pub-id"><a href="https://doi.org/10.1038/s41467-024-49892-9" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1038/s41467-024-49892-9</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=In%20silico%20fragment-based%20discovery%20of%20CIB1-directed%20anti-tumor%20agents%20by%20FRASE-Bot&amp;author=Y%20An&amp;author=J%20Lim&amp;author=M%20Glavatskikh&amp;publication_year=2024&amp;journal=Nat%20Commun&amp;volume=15&amp;pages=5564" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/s41467-024-49892-9" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fs41467-024-49892-9" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1038%2Fs41467-024-49892-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/38956119" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:In%20silico%20fragment-based%20discovery%20of%20CIB1-directed%20anti-tumor%20agents%20by%20FRASE-Bot&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B5" class="js-splitview-ref-item" data-legacy-id="btaf449-B5"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B5" href="javascript:;" aria-label="jumplink-btaf449-B5" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B5" class="ref-content " data-id="btaf449-B5"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Capra</div> <div class="given-names">JA</div>, <span class="name string-name"><div class="surname">Laskowski</div> <div class="given-names">RA</div></span>, <span class="name string-name"><div class="surname">Thornton</div> <div class="given-names">JM</div></span> &nbsp;et al.  &nbsp;<div class="article-title">Predicting protein ligand binding sites by combining evolutionary sequence conservation and 3D structure</div>. <div class="source ">PLoS Comput Biol</div> &nbsp;<div class="year">2009</div>;<div class="volume">5</div>:<div class="fpage">e1000585</div>. <div class="pub-id"><a href="https://doi.org/10.1371/journal.pcbi.1000585" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1371/journal.pcbi.1000585</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Predicting%20protein%20ligand%20binding%20sites%20by%20combining%20evolutionary%20sequence%20conservation%20and%203D%20structure&amp;author=JA%20Capra&amp;author=RA%20Laskowski&amp;author=JM%20Thornton&amp;publication_year=2009&amp;journal=PLoS%20Comput%20Biol&amp;volume=5&amp;pages=e1000585" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pcbi.1000585" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000585" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pcbi.1000585"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19997483" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Predicting%20protein%20ligand%20binding%20sites%20by%20combining%20evolutionary%20sequence%20conservation%20and%203D%20structure&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B6" class="js-splitview-ref-item" data-legacy-id="btaf449-B6"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B6" href="javascript:;" aria-label="jumplink-btaf449-B6" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B6" class="ref-content " data-id="btaf449-B6"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Durrant</div> <div class="given-names">JD</div>, <span class="name string-name"><div class="surname">Votapka</div> <div class="given-names">L</div></span>, <span class="name string-name"><div class="surname">Sørensen</div> <div class="given-names">J</div></span> &nbsp;et al.  &nbsp;<div class="article-title">POVME 2.0: an enhanced tool for determining pocket shape and volume characteristics</div>. <div class="source ">J Chem Theory Comput</div> &nbsp;<div class="year">2014</div>;<div class="volume">10</div>:<div class="fpage">5047</div>–<div class="lpage">56</div>. <div class="pub-id"><a href="https://doi.org/10.1021/ct500381c" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1021/ct500381c</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=POVME%202.0%3A%20an%20enhanced%20tool%20for%20determining%20pocket%20shape%20and%20volume%20characteristics&amp;author=JD%20Durrant&amp;author=L%20Votapka&amp;author=J%20S%C3%B8rensen&amp;publication_year=2014&amp;journal=J%20Chem%20Theory%20Comput&amp;volume=10&amp;pages=5047-56" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1021/ct500381c" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fct500381c" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1021%2Fct500381c"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25400521" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:POVME%202.0%3A%20an%20enhanced%20tool%20for%20determining%20pocket%20shape%20and%20volume%20characteristics&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B7" class="js-splitview-ref-item" data-legacy-id="btaf449-B7"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B7" href="javascript:;" aria-label="jumplink-btaf449-B7" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B7" class="ref-content " data-id="btaf449-B7"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Ghersi</div> <div class="given-names">D</div>, <span class="name string-name"><div class="surname">Sanchez</div> <div class="given-names">R.</div></span> &nbsp;<div class="article-title">Improving accuracy and efficiency of blind protein‐ligand docking by focusing on predicted binding sites</div>. <div class="source ">Proteins</div> &nbsp;<div class="year">2009</div>;<div class="volume">74</div>:<div class="fpage">417</div>–<div class="lpage">24</div>. <div class="pub-id"><a href="https://doi.org/10.1002/prot.22154" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1002/prot.22154</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Improving%20accuracy%20and%20efficiency%20of%20blind%20protein%E2%80%90ligand%20docking%20by%20focusing%20on%20predicted%20binding%20sites&amp;author=D%20Ghersi&amp;author=R.%20Sanchez&amp;publication_year=2009&amp;journal=Proteins&amp;volume=74&amp;pages=417-24" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1002/prot.v74:2" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fprot.v74:2" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1002%2Fprot.v74:2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18636505" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Improving%20accuracy%20and%20efficiency%20of%20blind%20protein%E2%80%90ligand%20docking%20by%20focusing%20on%20predicted%20binding%20sites&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B8" class="js-splitview-ref-item" data-legacy-id="btaf449-B8"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B8" href="javascript:;" aria-label="jumplink-btaf449-B8" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B8" class="ref-content " data-id="btaf449-B8"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Graef</div> <div class="given-names">J</div>, <span class="name string-name"><div class="surname">Ehrt</div> <div class="given-names">C</div></span>, <span class="name string-name"><div class="surname">Rarey</div> <div class="given-names">M.</div></span> &nbsp;<div class="article-title">Binding
 site detection remastered: enabling fast, robust, and reliable binding 
site detection and descriptor calculation with DoGSite3</div>. <div class="source ">J Chem Inf Model</div> &nbsp;<div class="year">2023</div>;<div class="volume">63</div>:<div class="fpage">3128</div>–<div class="lpage">37</div>. <div class="pub-id"><a href="https://doi.org/10.1021/acs.jcim.3c00336" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1021/acs.jcim.3c00336</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Binding%20site%20detection%20remastered%3A%20enabling%20fast%2C%20robust%2C%20and%20reliable%20binding%20site%20detection%20and%20descriptor%20calculation%20with%20DoGSite3&amp;author=J%20Graef&amp;author=C%20Ehrt&amp;author=M.%20Rarey&amp;publication_year=2023&amp;journal=J%20Chem%20Inf%20Model&amp;volume=63&amp;pages=3128-37" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1021/acs.jcim.3c00336" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Facs.jcim.3c00336" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1021%2Facs.jcim.3c00336"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/37130052" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Binding%20site%20detection%20remastered%3A%20enabling%20fast%2C%20robust%2C%20and%20reliable%20binding%20site%20detection%20and%20descriptor%20calculation%20with%20DoGSite3&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B9" class="js-splitview-ref-item" data-legacy-id="btaf449-B9"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B9" href="javascript:;" aria-label="jumplink-btaf449-B9" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B9" class="ref-content " data-id="btaf449-B9"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Jeevan</div> <div class="given-names">K</div>, <span class="name string-name"><div class="surname">Palistha</div> <div class="given-names">S</div></span>, <span class="name string-name"><div class="surname">Tayara</div> <div class="given-names">H</div></span> &nbsp;et al.  &nbsp;<div class="article-title">PUResNetV2.0: a deep learning model leveraging sparse representation for improved ligand binding site prediction</div>. <div class="source ">J Cheminform</div> &nbsp;<div class="year">2024</div>;<div class="volume">16</div>:<div class="fpage">66</div>. <div class="pub-id"><a href="https://doi.org/10.1186/s13321-024-00865-6" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1186/s13321-024-00865-6</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=PUResNetV2.0%3A%20a%20deep%20learning%20model%20leveraging%20sparse%20representation%20for%20improved%20ligand%20binding%20site%20prediction&amp;author=K%20Jeevan&amp;author=S%20Palistha&amp;author=H%20Tayara&amp;publication_year=2024&amp;journal=J%20Cheminform&amp;volume=16&amp;pages=66" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s13321-024-00865-6" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13321-024-00865-6" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs13321-024-00865-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/38849917" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:PUResNetV2.0%3A%20a%20deep%20learning%20model%20leveraging%20sparse%20representation%20for%20improved%20ligand%20binding%20site%20prediction&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B10" class="js-splitview-ref-item" data-legacy-id="btaf449-B10"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B10" href="javascript:;" aria-label="jumplink-btaf449-B10" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B10" class="ref-content " data-id="btaf449-B10"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Jiménez</div> <div class="given-names">J</div>, <span class="name string-name"><div class="surname">Doerr</div> <div class="given-names">S</div></span>, <span class="name string-name"><div class="surname">Martínez-Rosell</div> <div class="given-names">G</div></span> &nbsp;et al.  &nbsp;<div class="article-title">DeepSite: protein-Binding site predictor using 3D-convolutional neural networks</div>. <div class="source ">Bioinformatics</div> &nbsp;<div class="year">2017</div>;<div class="volume">33</div>:<div class="fpage">3036</div>–<div class="lpage">42</div>. <div class="pub-id"><a href="https://doi.org/10.1093/bioinformatics/btx350" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1093/bioinformatics/btx350</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=DeepSite%3A%20protein-Binding%20site%20predictor%20using%203D-convolutional%20neural%20networks&amp;author=J%20Jim%C3%A9nez&amp;author=S%20Doerr&amp;author=G%20Mart%C3%ADnez-Rosell&amp;publication_year=2017&amp;journal=Bioinformatics&amp;volume=33&amp;pages=3036-42" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btx350" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtx350" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtx350"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28575181" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:DeepSite%3A%20protein-Binding%20site%20predictor%20using%203D-convolutional%20neural%20networks&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B11" class="js-splitview-ref-item" data-legacy-id="btaf449-B11"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B11" href="javascript:;" aria-label="jumplink-btaf449-B11" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B11" class="ref-content " data-id="btaf449-B11"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Kandel</div> <div class="given-names">J</div>, <span class="name string-name"><div class="surname">Tayara</div> <div class="given-names">H</div></span>, <span class="name string-name"><div class="surname">Chong</div> <div class="given-names">KT.</div></span> &nbsp;<div class="article-title">PUResNet: prediction of protein–ligand binding sites using deep residual neural network</div>. <div class="source ">J Cheminform</div> &nbsp;<div class="year">2021</div>;<div class="volume">13</div>:<div class="fpage">65</div>. <div class="pub-id"><a href="https://doi.org/10.1186/s13321-021-00547-7" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1186/s13321-021-00547-7</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=PUResNet%3A%20prediction%20of%20protein%E2%80%93ligand%20binding%20sites%20using%20deep%20residual%20neural%20network&amp;author=J%20Kandel&amp;author=H%20Tayara&amp;author=KT.%20Chong&amp;publication_year=2021&amp;journal=J%20Cheminform&amp;volume=13&amp;pages=65" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s13321-021-00547-7" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13321-021-00547-7" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs13321-021-00547-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/34496970" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:PUResNet%3A%20prediction%20of%20protein%E2%80%93ligand%20binding%20sites%20using%20deep%20residual%20neural%20network&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B12" class="js-splitview-ref-item" data-legacy-id="btaf449-B12"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B12" href="javascript:;" aria-label="jumplink-btaf449-B12" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B12" class="ref-content " data-id="btaf449-B12"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Kim</div> <div class="given-names">JJ</div>, <span class="name string-name"><div class="surname">Gharpure</div> <div class="given-names">A</div></span>, <span class="name string-name"><div class="surname">Teng</div> <div class="given-names">J</div></span> &nbsp;et al.  &nbsp;<div class="article-title">Shared structural mechanisms of general anaesthetics and benzodiazepines</div>. <div class="source ">Nature</div> &nbsp;<div class="year">2020</div>;<div class="volume">585</div>:<div class="fpage">303</div>–<div class="lpage">8</div>. <div class="pub-id"><a href="https://doi.org/10.1038/s41586-020-2654-5" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1038/s41586-020-2654-5</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Shared%20structural%20mechanisms%20of%20general%20anaesthetics%20and%20benzodiazepines&amp;author=JJ%20Kim&amp;author=A%20Gharpure&amp;author=J%20Teng&amp;publication_year=2020&amp;journal=Nature&amp;volume=585&amp;pages=303-8" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/s41586-020-2654-5" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fs41586-020-2654-5" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1038%2Fs41586-020-2654-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32879488" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Shared%20structural%20mechanisms%20of%20general%20anaesthetics%20and%20benzodiazepines&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B13" class="js-splitview-ref-item" data-legacy-id="btaf449-B13"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B13" href="javascript:;" aria-label="jumplink-btaf449-B13" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B13" class="ref-content " data-id="btaf449-B13"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Krivák</div> <div class="given-names">R</div>, <span class="name string-name"><div class="surname">Hoksza</div> <div class="given-names">D.</div></span> &nbsp;<div class="article-title">P2Rank: machine learning based tool for rapid and accurate prediction of ligand binding sites from protein structure</div>. <div class="source ">J Cheminform</div> &nbsp;<div class="year">2018</div>;<div class="volume">10</div>:<div class="fpage">39</div>. <div class="pub-id"><a href="https://doi.org/10.1186/s13321-018-0285-8" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1186/s13321-018-0285-8</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=P2Rank%3A%20machine%20learning%20based%20tool%20for%20rapid%20and%20accurate%20prediction%20of%20ligand%20binding%20sites%20from%20protein%20structure&amp;author=R%20Kriv%C3%A1k&amp;author=D.%20Hoksza&amp;publication_year=2018&amp;journal=J%20Cheminform&amp;volume=10&amp;pages=39" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s13321-018-0285-8" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13321-018-0285-8" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs13321-018-0285-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30109435" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:P2Rank%3A%20machine%20learning%20based%20tool%20for%20rapid%20and%20accurate%20prediction%20of%20ligand%20binding%20sites%20from%20protein%20structure&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B14" class="js-splitview-ref-item" data-legacy-id="btaf449-B14"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B14" href="javascript:;" aria-label="jumplink-btaf449-B14" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B14" class="ref-content " data-id="btaf449-B14"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Kruse</div> <div class="given-names">AC</div>, <span class="name string-name"><div class="surname">Kobilka</div> <div class="given-names">BK</div></span>, <span class="name string-name"><div class="surname">Gautam</div> <div class="given-names">D</div></span> &nbsp;et al.  &nbsp;<div class="article-title">Muscarinic acetylcholine receptors: novel opportunities for drug development</div>. <div class="source ">Nat Rev Drug Discov</div> &nbsp;<div class="year">2014</div>;<div class="volume">13</div>:<div class="fpage">549</div>–<div class="lpage">60</div>. <div class="pub-id"><a href="https://doi.org/10.1038/nrd4295" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1038/nrd4295</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Muscarinic%20acetylcholine%20receptors%3A%20novel%20opportunities%20for%20drug%20development&amp;author=AC%20Kruse&amp;author=BK%20Kobilka&amp;author=D%20Gautam&amp;publication_year=2014&amp;journal=Nat%20Rev%20Drug%20Discov&amp;volume=13&amp;pages=549-60" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nrd4295" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrd4295" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnrd4295"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24903776" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Muscarinic%20acetylcholine%20receptors%3A%20novel%20opportunities%20for%20drug%20development&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B15" class="js-splitview-ref-item" data-legacy-id="btaf449-B15"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B15" href="javascript:;" aria-label="jumplink-btaf449-B15" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B15" class="ref-content " data-id="btaf449-B15"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Le Guilloux</div> <div class="given-names">V</div>, <span class="name string-name"><div class="surname">Schmidtke</div> <div class="given-names">P</div></span>, <span class="name string-name"><div class="surname">Tuffery</div> <div class="given-names">P.</div></span> &nbsp;<div class="article-title">Fpocket: an open source platform for ligand pocket detection</div>. <div class="source ">BMC Bioinformatics</div> &nbsp;<div class="year">2009</div>;<div class="volume">10</div>:<div class="fpage">168</div>. <div class="pub-id"><a href="https://doi.org/10.1186/1471-2105-10-168" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1186/1471-2105-10-168</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Fpocket%3A%20an%20open%20source%20platform%20for%20ligand%20pocket%20detection&amp;author=V%20Le%20Guilloux&amp;author=P%20Schmidtke&amp;author=P.%20Tuffery&amp;publication_year=2009&amp;journal=BMC%20Bioinformatics&amp;volume=10&amp;pages=168" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2105-10-168" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-10-168" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-10-168"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19486540" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Fpocket%3A%20an%20open%20source%20platform%20for%20ligand%20pocket%20detection&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B16" class="js-splitview-ref-item" data-legacy-id="btaf449-B16"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B16" href="javascript:;" aria-label="jumplink-btaf449-B16" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B16" class="ref-content " data-id="btaf449-B16"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Liang</div> <div class="given-names">J</div>, <span class="name string-name"><div class="surname">Edelsbrunner</div> <div class="given-names">H</div></span>, <span class="name string-name"><div class="surname">Woodward</div> <div class="given-names">C.</div></span> &nbsp;<div class="article-title">Anatomy of protein pockets and cavities: measurement of binding site geometry and implications for ligand design</div>. <div class="source ">Protein Sci</div> &nbsp;<div class="year">1998</div>;<div class="volume">7</div>:<div class="fpage">1884</div>–<div class="lpage">97</div>. <div class="pub-id"><a href="https://doi.org/10.1002/pro.5560070905" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1002/pro.5560070905</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Anatomy%20of%20protein%20pockets%20and%20cavities%3A%20measurement%20of%20binding%20site%20geometry%20and%20implications%20for%20ligand%20design&amp;author=J%20Liang&amp;author=H%20Edelsbrunner&amp;author=C.%20Woodward&amp;publication_year=1998&amp;journal=Protein%20Sci&amp;volume=7&amp;pages=1884-97" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1002/pro.v7:9" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fpro.v7:9" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1002%2Fpro.v7:9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9761470" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Anatomy%20of%20protein%20pockets%20and%20cavities%3A%20measurement%20of%20binding%20site%20geometry%20and%20implications%20for%20ligand%20design&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B17" class="js-splitview-ref-item" data-legacy-id="btaf449-B17"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B17" href="javascript:;" aria-label="jumplink-btaf449-B17" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B17" class="ref-content " data-id="btaf449-B17"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Liu</div> <div class="given-names">Y</div>, <span class="name string-name"><div class="surname">Yang</div> <div class="given-names">X</div></span>, <span class="name string-name"><div class="surname">Gan</div> <div class="given-names">J</div></span> &nbsp;et al.  &nbsp;<div class="article-title">CB-Dock2: improved protein–ligand blind docking by integrating cavity detection, docking and homologous template fitting</div>. <div class="source ">Nucleic Acids Res</div> &nbsp;<div class="year">2022</div>;<div class="volume">50</div>:<div class="fpage">W159</div>–<div class="lpage">64</div>. <div class="pub-id"><a href="https://doi.org/10.1093/nar/gkac394" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1093/nar/gkac394</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=CB-Dock2%3A%20improved%20protein%E2%80%93ligand%20blind%20docking%20by%20integrating%20cavity%20detection%2C%20docking%20and%20homologous%20template%20fitting&amp;author=Y%20Liu&amp;author=X%20Yang&amp;author=J%20Gan&amp;publication_year=2022&amp;journal=Nucleic%20Acids%20Res&amp;volume=50&amp;pages=W159-64" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkac394" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkac394" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkac394"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/35609983" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:CB-Dock2%3A%20improved%20protein%E2%80%93ligand%20blind%20docking%20by%20integrating%20cavity%20detection%2C%20docking%20and%20homologous%20template%20fitting&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B18" class="js-splitview-ref-item" data-legacy-id="btaf449-B18"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B18" href="javascript:;" aria-label="jumplink-btaf449-B18" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B18" class="ref-content " data-id="btaf449-B18"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Murphy</div> <div class="given-names">JM</div>, <span class="name string-name"><div class="surname">Lucet</div> <div class="given-names">IS</div></span>, <span class="name string-name"><div class="surname">Hildebrand</div> <div class="given-names">JM</div></span> &nbsp;et al.  &nbsp;<div class="article-title">Insights
 into the evolution of divergent nucleotide-binding mechanisms among 
pseudokinases revealed by crystal structures of human and mouse MLKL</div>. <div class="source ">Biochem J</div> &nbsp;<div class="year">2014</div>;<div class="volume">457</div>:<div class="fpage">369</div>–<div class="lpage">77</div>. <div class="pub-id"><a href="https://doi.org/10.1042/BJ20131270" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1042/BJ20131270</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20the%20evolution%20of%20divergent%20nucleotide-binding%20mechanisms%20among%20pseudokinases%20revealed%20by%20crystal%20structures%20of%20human%20and%20mouse%20MLKL&amp;author=JM%20Murphy&amp;author=IS%20Lucet&amp;author=JM%20Hildebrand&amp;publication_year=2014&amp;journal=Biochem%20J&amp;volume=457&amp;pages=369-77" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1042/BJ20131270" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBJ20131270" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1042%2FBJ20131270"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24219132" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Insights%20into%20the%20evolution%20of%20divergent%20nucleotide-binding%20mechanisms%20among%20pseudokinases%20revealed%20by%20crystal%20structures%20of%20human%20and%20mouse%20MLKL&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B19" class="js-splitview-ref-item" data-legacy-id="btaf449-B19"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B19" href="javascript:;" aria-label="jumplink-btaf449-B19" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B19" class="ref-content " data-id="btaf449-B19"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Tian</div> <div class="given-names">W</div>, <span class="name string-name"><div class="surname">Chen</div> <div class="given-names">C</div></span>, <span class="name string-name"><div class="surname">Lei</div> <div class="given-names">X</div></span> &nbsp;et al.  &nbsp;<div class="article-title">CASTp 3.0: computed atlas of surface topography of proteins</div>. <div class="source ">Nucleic Acids Res</div> &nbsp;<div class="year">2018</div>;<div class="volume">46</div>:<div class="fpage">W363</div>–<div class="lpage">7</div>. <div class="pub-id"><a href="https://doi.org/10.1093/nar/gky473" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1093/nar/gky473</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=CASTp%203.0%3A%20computed%20atlas%20of%20surface%20topography%20of%20proteins&amp;author=W%20Tian&amp;author=C%20Chen&amp;author=X%20Lei&amp;publication_year=2018&amp;journal=Nucleic%20Acids%20Res&amp;volume=46&amp;pages=W363-7" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gky473" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgky473" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgky473"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29860391" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:CASTp%203.0%3A%20computed%20atlas%20of%20surface%20topography%20of%20proteins&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B20" class="js-splitview-ref-item" data-legacy-id="btaf449-B20"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B20" href="javascript:;" aria-label="jumplink-btaf449-B20" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B20" class="ref-content " data-id="btaf449-B20"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Wei</div> <div class="given-names">H</div>, <span class="name string-name"><div class="surname">Wang</div> <div class="given-names">W</div></span>, <span class="name string-name"><div class="surname">Peng</div> <div class="given-names">Z</div></span> &nbsp;et al.  &nbsp;<div class="article-title">Q-BioLiP: a comprehensive resource for quaternary structure-based protein–ligand interactions</div>. <div class="source ">Genomics Proteomics Bioinform</div> &nbsp;<div class="year">2024</div>;<div class="volume">22</div>:<div class="fpage">qzae001</div>. <div class="pub-id"><a href="https://doi.org/10.1093/gpbjnl/qzae001" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1093/gpbjnl/qzae001</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Q-BioLiP%3A%20a%20comprehensive%20resource%20for%20quaternary%20structure-based%20protein%E2%80%93ligand%20interactions&amp;author=H%20Wei&amp;author=W%20Wang&amp;author=Z%20Peng&amp;publication_year=2024&amp;journal=Genomics%20Proteomics%20Bioinform&amp;volume=22&amp;pages=qzae001" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/gpbjnl/qzae001" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgpbjnl%2Fqzae001" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1093%2Fgpbjnl%2Fqzae001"> </span></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Q-BioLiP%3A%20a%20comprehensive%20resource%20for%20quaternary%20structure-based%20protein%E2%80%93ligand%20interactions&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B21" class="js-splitview-ref-item" data-legacy-id="btaf449-B21"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B21" href="javascript:;" aria-label="jumplink-btaf449-B21" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B21" class="ref-content " data-id="btaf449-B21"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Wei</div> <div class="given-names">J</div>, <span class="name string-name"><div class="surname">Wang</div> <div class="given-names">X</div></span>, <span class="name string-name"><div class="surname">Schuurmans</div> <div class="given-names">D</div></span> &nbsp;et al.  Chain-of-thought prompting elicits reasoning in large language models. arXiv, arXiv:2201.11903, <div class="year">2023</div>, preprint: not peer reviewed.<p></p></div></div></div></div><div content-id="btaf449-B22" class="js-splitview-ref-item" data-legacy-id="btaf449-B22"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B22" href="javascript:;" aria-label="jumplink-btaf449-B22" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B22" class="ref-content " data-id="btaf449-B22"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Xia</div> <div class="given-names">C-Q</div>, <span class="name string-name"><div class="surname">Pan</div> <div class="given-names">X</div></span>, <span class="name string-name"><div class="surname">Shen</div> <div class="given-names">H-B.</div></span> &nbsp;<div class="article-title">Protein-Ligand binding residue prediction enhancement through hybrid deep heterogeneous learning of sequence and structure data</div>. <div class="source ">Bioinformatics</div> &nbsp;<div class="year">2020</div>;<div class="volume">36</div>:<div class="fpage">3018</div>–<div class="lpage">27</div>. <div class="pub-id"><a href="https://doi.org/10.1093/bioinformatics/btaa110" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1093/bioinformatics/btaa110</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Protein-Ligand%20binding%20residue%20prediction%20enhancement%20through%20hybrid%20deep%20heterogeneous%20learning%20of%20sequence%20and%20structure%20data&amp;author=C-Q%20Xia&amp;author=X%20Pan&amp;author=H-B.%20Shen&amp;publication_year=2020&amp;journal=Bioinformatics&amp;volume=36&amp;pages=3018-27" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btaa110" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtaa110" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtaa110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32091580" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Protein-Ligand%20binding%20residue%20prediction%20enhancement%20through%20hybrid%20deep%20heterogeneous%20learning%20of%20sequence%20and%20structure%20data&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B23" class="js-splitview-ref-item" data-legacy-id="btaf449-B23"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B23" href="javascript:;" aria-label="jumplink-btaf449-B23" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B23" class="ref-content " data-id="btaf449-B23"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Yao</div> <div class="given-names">S</div>, <span class="name string-name"><div class="surname">Yu</div> <div class="given-names">D</div></span>, <span class="name string-name"><div class="surname">Zhao</div> <div class="given-names">J</div></span> &nbsp;et al.  Tree of thoughts: deliberate problem solving with large language models. arXiv, arXiv:2305.10601, <div class="year">2023</div>, preprint: not peer reviewed.<p></p></div></div></div></div><div content-id="btaf449-B24" class="js-splitview-ref-item" data-legacy-id="btaf449-B24"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B24" href="javascript:;" aria-label="jumplink-btaf449-B24" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B24" class="ref-content " data-id="btaf449-B24"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Yesylevskyy</div> <div class="given-names">S.</div> MolAR: memory‐safe library for analysis of MD simulations written in rust. <div class="source ">J Comput Chem</div> 2025;<div class="volume">46</div>:e27536. <div class="pub-id"><a href="https://doi.org/10.1002/jcc.27536" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1002/jcc.27536</a></div><p></p></div></div></div></div><div content-id="btaf449-B25" class="js-splitview-ref-item" data-legacy-id="btaf449-B25"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B25" href="javascript:;" aria-label="jumplink-btaf449-B25" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B25" class="ref-content " data-id="btaf449-B25"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Zhang</div> <div class="given-names">C</div>, <span class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X</div></span>, <span class="name string-name"><div class="surname">Freddolino</div> <div class="given-names">L</div></span> &nbsp;et al.  &nbsp;<div class="article-title">BioLiP2: an updated structure database for biologically relevant Ligand-Protein interactions</div>. <div class="source ">Nucleic Acids Res</div> &nbsp;<div class="year">2024</div>;<div class="volume">52</div>:<div class="fpage">D404</div>–<div class="lpage">12</div>. <div class="pub-id"><a href="https://doi.org/10.1093/nar/gkad630" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.1093/nar/gkad630</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=BioLiP2%3A%20an%20updated%20structure%20database%20for%20biologically%20relevant%20Ligand-Protein%20interactions&amp;author=C%20Zhang&amp;author=X%20Zhang&amp;author=L%20Freddolino&amp;publication_year=2024&amp;journal=Nucleic%20Acids%20Res&amp;volume=52&amp;pages=D404-12" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkad630" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkad630" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkad630"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/37522378" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BioLiP2%3A%20an%20updated%20structure%20database%20for%20biologically%20relevant%20Ligand-Protein%20interactions&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div><div content-id="btaf449-B26" class="js-splitview-ref-item" data-legacy-id="btaf449-B26"><div class="refLink-parent"><span class="refLink"><a name="jumplink-btaf449-B26" href="javascript:;" aria-label="jumplink-btaf449-B26" data-id="" data-google-interstitial="false"></a></span></div><div class="ref false"><div id="ref-auto-btaf449-B26" class="ref-content " data-id="btaf449-B26"><div class="mixed-citation citation"><p class="mixed-citation-compatibility"><span class="person-group"><span class="name string-name"></span></span></p><div class="surname">Zhao</div> <div class="given-names">Y</div>, <span class="name string-name"><div class="surname">He</div> <div class="given-names">S</div></span>, <span class="name string-name"><div class="surname">Xing</div> <div class="given-names">Y</div></span> &nbsp;et al.  &nbsp;<div class="article-title">A point cloud graph neural network for protein–ligand binding site prediction</div>. <div class="source ">Int J Mol Sci</div> &nbsp;<div class="year">2024</div>;<div class="volume">25</div>:<div class="fpage">9280</div>. <div class="pub-id"><a href="https://doi.org/10.3390/ijms25179280" class="link link-doi openInAnotherWindow" target="_blank" data-google-interstitial="false">10.3390/ijms25179280</a></div><p></p><!--citationLinks: case 1--><div class="citation-links"></div><div class="citation-links"><p class="citation-links-compatibility"><span class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20point%20cloud%20graph%20neural%20network%20for%20protein%E2%80%93ligand%20binding%20site%20prediction&amp;author=Y%20Zhao&amp;author=S%20He&amp;author=Y%20Xing&amp;publication_year=2024&amp;journal=Int%20J%20Mol%20Sci&amp;volume=25&amp;pages=9280" target="_blank" data-google-interstitial="false">Google Scholar</a></span></p><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.3390/ijms25179280" target="_blank" data-google-interstitial="false">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Fijms25179280" target="_blank" data-google-interstitial="false">Search ADS</a></div><div class="xslopenurl empty-target"><span class="inst-open-url-holders" data-targetid="10.3390%2Fijms25179280"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/39273227" class="link link-pub-id openInAnotherWindow" target="_blank" data-google-interstitial="false">PubMed</a></div><p class="citation-links-compatibility"><span class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style=""><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20point%20cloud%20graph%20neural%20network%20for%20protein%E2%80%93ligand%20binding%20site%20prediction&amp;qt=advanced&amp;dblist=638" target="_blank" data-google-interstitial="false">WorldCat</a></span></p>&nbsp;</div></div></div></div></div></div>    <!-- /foreach in Model.Sections -->
    <div class="widget widget-ArticlePubStateInfo widget-instance-OUP_ArticlePubStateInfo">
         
    </div>



        

        
<div class="copyright copyright-statement">© The Author(s) 2025. Published by Oxford University Press.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri openInAnotherWindow" href="https://creativecommons.org/licenses/by/4.0/" target="_blank" data-google-interstitial="false">https://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->





 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    


 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt" style="display: none;">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link btn" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=8225722&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5139" data-google-interstitial="false">Download all slides</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab" data-google-interstitial="false"></a>
        
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/41/8/10.1093_bioinformatics_btaf449/1/btaf449_supplementary_data.docx?Expires=1766142747&amp;Signature=ZXmAeUV9kpiLAQq4D5bN7xKb0AnUQC8ElGtXLSorCn4K6~E8YMX4ybYM7HY5IfYwecKG~zDjUYTGwvtd34i79W2G3PtB7x6THMkduU2vOW-H4jWDW9Fwl8Dn2jfJat7fqzri6qoLwS~X2ly6YK4AacbtwtfQr4yKpEEhpXNl4YtFpi4wajENAYPMWxOi3LQSBuFs3qufqXU5hNyRLeAa2~OwaRrDH8ABEWm9bSPGdCPYmCOnlN~w2-wxmHkSidEjwUN9uJCG1iTwtB~f7AGv4ZVIzch3RBuGx1m9oP~ke0HwVfxJXUUY7bFuPFvL3wR84LMbyi8AHT5wLSfzDlvU1w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" rel="nofollow" data-google-interstitial="false"><span class="label title-label">btaf449_Supplementary_Data</span></a> - docx file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div class="js-adBlock-parent-wrap adblock-parent-wrap">
    <div class="adBlockMainBodyTop-wrap js-adBlockMainBodyTop hide">
        <div id="adBlockMainBodyTop" class="js-adblock at-adblock" data-lazy-load-margin="150">
            <script>
                googletag.cmd.push(function () {
                    googletag.display('adBlockMainBodyTop');
                });
            </script>
        </div>
            <div class="advertisement-text at-adblock js-adblock-advertisement-text hide">Advertisement</div>
            </div>
</div> 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        



    <div class="artmet-wrapper horizontal-artmet">
<div class="contentmet-border">
    <div class="contentmet-wrapper horizontal-contentmet">
            <div class="contentmet-citations contentmet-badges-wrap js-contentmet-citations">
                <h3 class="contentmet-text">Citations</h3>
                    <div class="contentmet-item contentmet-dimensions">
    <div class="widget widget-DimensionsBadge widget-instance-ArticleLevelMetrics_DimensionsBadge">
        <span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/btaf449" data-legend="never" data-style="small_circle" data-hide-zero-citations="false" data-dimensions-badge-installed="true"><a class="__dimensions_Link" href="https://badge.dimensions.ai/details/doi/10.1093/bioinformatics/btaf449?domain=https://academic.oup.com" target="" style="display: none;" referrerpolicy="no-referrer-when-downgrade" aria-label="0 total citations on Dimensions." data-google-interstitial="false"><div class="__dimensions_Badge __dimensions_Badge_style_small_circle"><div class="__dimensions_Badge_Image">
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 600 600">
    <defs>
      <filter id="6l03j-blur" color-interpolation-filters="sRGB">
        <feGaussianBlur in="SourceGraphic" stdDeviation="40"></feGaussianBlur>
      </filter>
      <mask id="6l03j-shape-mask">
        <path fill="white" d="M546.5,287.3l-111.8-194c-1.8-3-4.1-5.6-6.7-7.7c-2.2-2.7-5.4-4.3-8.9-4.5c-2.3-0.7-4.7-1-7-1
          L188.2,80c-9.3,0-18,5-22.6,13.1l-112,193.8c-4.7,8.1-4.7,18,0,26.1l111.9,193.9c4.7,8.1,13.3,13,22.6,13.1l223.9,0.1
          c9.3,0,17.9-4.8,22.6-12.8l112-193.7l0,0C551.1,305.1,551.2,295.4,546.5,287.3z M516.6,287.4l-41.5,0.1l-26.7-118.2L516.6,287.4z
           M337.2,106.2l-114.5,36.3l-20.3-36.4L337.2,106.2z M180.7,119l20.1,36.2L112,237.7L180.7,119z M181,481.8l-69.3-120l89.2,84
          L181,481.8z M203.3,493.8l19.6-35.6l114.6,35.7L203.3,493.8z M408.1,489.3l-152.7-47.5l-32.3-10.6l-25.3-23.2L82.6,299.6
          l114.6-106.9l25-23.1v-0.2l0.7-0.2l32.4-10.7l153.6-48.7l35.5,156.6l7.5,33.4l-7.6,33.5L408.1,489.3z M447.1,432.9l27.9-120.3
          l41.7-0.1L447.1,432.9z"></path>
      </mask>
      <g id="6l03j-gradient" filter="url(#6l03j-blur)">
        <g>
          <path fill="#006EB6" d="M0,0l300,300v-424.4C182.9-124.4,76.8-76.9,0,0z"></path>
          <path fill="#28348A" d="M300-124.4V300L600,0C523.2-76.9,417.1-124.4,300-124.4z"></path>
          <path fill="#E20613" d="M600,600c76.8-76.8,124.3-182.9,124.3-300H300L600,600z"></path>
          <path fill="#DA0078" d="M300,300h424.4c0-117.1-47.5-223.2-124.3-300L300,300z"></path>
          <path fill="#5BC4F1" d="M-124.4,300H300L0,0C-76.9,76.8-124.4,182.9-124.4,300z"></path>
          <path fill="#A1C517" d="M0,600l300-300h-424.4C-124.4,417.1-76.9,523.2,0,600z"></path>
          <path fill="#F08800" d="M300,724.4c117.1,0,223.2-47.5,300-124.3L300,300V724.4z"></path>
          <path fill="#FFCB00" d="M300,724.4V300L0,600C76.8,676.9,182.9,724.4,300,724.4z"></path>
        </g>
      </g>
    </defs>
  </svg>

  <div class="__db_background"></div>

  <div class="__db_shape">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 600 600" xlink="http://www.w3.org/1999/xlink">
      <use mask="url(#6l03j-shape-mask)" xlink:href="#6l03j-gradient"></use>
    </svg>
  </div>

  <div class="__db_score __db_score_normal">0</div>
  <div class="__db_label">CITATIONS</div>

  <img src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/badge.png" class="__dimensions_png" alt="0 total citations on Dimensions.">
</div>
</div></a></span>
<script async="" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/badge.js" charset="utf-8"></script>
 
    </div>

                    </div>
                            </div>
                    <div class="contentmet-views contentmet-badges-wrap js-contentmet-views">
                <h3 class="contentmet-text">Views</h3>
                <div class="contentmet-item circle-text circle-text-views">
                    <div class="contentmet-number">1,068</div>
                </div>
            </div>
                    <div class="contentmet-item contentmet-badges-wrap">
                    <h3 class="contentmet-text">Altmetric</h3>
                    <div class="contentmet-item contentmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/bioinformatics/btaf449" data-uuid="ce223829-eadf-5724-51da-741fa7da106b"><a style="cursor: pointer; display: inline-block; background-image: url(&quot;https://badges.altmetric.com/?size=64&amp;score=?&amp;types=????????&amp;style=donut&quot;); background-repeat: no-repeat; background-size: contain; width: 64px; height: 64px;" href="https://www.altmetric.com/details.php?domain=academic.oup.com&amp;doi=10.1093%2Fbioinformatics%2Fbtaf449" target="" data-google-interstitial="false"> </a></div>
         <script type="text/javascript" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/embed.js"></script>
 
    </div>

                    </div>


            </div>
                    <div class="contentmet-modal-trigger-wrap clearfix">
                <button type="button" class="artmet-modal-trigger js-artmet-modal-trigger at-alm-metrics-modal-trigger" data-resource-id="8225722" data-resource-type="3" aria-haspopup="dialog" aria-expanded="false" aria-controls="MetricsModal">
                    <img class="contentmet-info-icon" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/i.svg" alt="Information">
                    <span class="contentmet-more-info">More metrics information</span>
                </button>
            </div>
    </div>
</div>
            <div class="artmet-modal js-artmet-modal" id="MetricsModal" role="dialog" aria-hidden="true" aria-live="assertive" aria-labelledby="widget-title-1-id">
                <div class="artmet-modal-contents js-metric-modal-contents at-alm-modal-contents" id="MetricsModalContents">




    <div id="artmet-full-details" class="artmet-full-wrap clearfix js-metric-full-wrap hide" data-autoload-chart="False">
            <h3 id="widget-title-1-id" class="widget-title-1 artmet-widget-title-1">Metrics</h3>

            <div class="artmet-item artmet-views-wrap">
                <div class="artmet-views clearfix">
                    <div class="artmet-total-views">
                        <span class="artmet-text">Total Views</span>
                        <span class="artmet-number">1,068</span>
                    </div>
                    <div class="artmet-breakdown-views-wrap">
                        <div class="artmet-breakdown-view breakdown-border">
                            <span class="artmet-number">796</span>
                            <span class="artmet-text">Pageviews</span>
                        </div>
                        <div class="artmet-breakdown-view">
                            <span class="artmet-number">272</span>
                            <span class="artmet-text">PDF Downloads</span>
                        </div>
                                            </div>
                </div>
                <div class="artmet-views-since">Since 8/1/2025</div>
            </div>

            <script>
                var ChartistData = ChartistData || {};

                ChartistData.data = {
                    labels: ['Aug 2025', 'Sep 2025', 'Oct 2025', 'Nov 2025'],
                    series: [[205, 576, 219, 68]]
                };
            </script>
            <div class="artmet-item artmet-chart">
                <div class="ct-chart ct-octave js-ct-chart"></div>

                <div class="artmet-table">
                    <table>
                        <thead>
                            <tr>
                                <th>Month:</th>
                                <th>Total Views:</th>
                            </tr>
                        </thead>
                        <tbody>
                                <tr>
                                    <td>August 2025</td>
                                    <td>205</td>
                                </tr>
                                <tr>
                                    <td>September 2025</td>
                                    <td>576</td>
                                </tr>
                                <tr>
                                    <td>October 2025</td>
                                    <td>219</td>
                                </tr>
                                <tr>
                                    <td>November 2025</td>
                                    <td>68</td>
                                </tr>
                        </tbody>
                    </table>
                </div>
            </div>

<div class="artmet-stats-wrap clearfix">
        <div class="artmet-item artmet-citations artmet-citation-border-bottom">
            <div class="widget-title-2 artmet-widget-title-2">Citations</div>
                <div class="artmet-badges-wrap artmet-dimensions">
    <div class="widget widget-DimensionsBadge widget-instance-ArticleLevelMetrics_DimensionsBadgeDetails">
        <span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/btaf449" data-legend="always" data-style="" data-hide-zero-citations="false" data-dimensions-badge-installed="true"><a class="__dimensions_Link" href="https://badge.dimensions.ai/details/doi/10.1093/bioinformatics/btaf449?domain=https://academic.oup.com" target="" style="display: none;" referrerpolicy="no-referrer-when-downgrade" aria-label="0 total citations on Dimensions." data-google-interstitial="false"><div class="__dimensions_Badge __dimensions_Badge_style_medium_circle"><div class="__dimensions_Badge_Image">
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 600 600">
    <defs>
      <filter id="clqkad-blur" color-interpolation-filters="sRGB">
        <feGaussianBlur in="SourceGraphic" stdDeviation="40"></feGaussianBlur>
      </filter>
      <mask id="clqkad-shape-mask">
        <path fill="white" d="M546.5,287.3l-111.8-194c-1.8-3-4.1-5.6-6.7-7.7c-2.2-2.7-5.4-4.3-8.9-4.5c-2.3-0.7-4.7-1-7-1
          L188.2,80c-9.3,0-18,5-22.6,13.1l-112,193.8c-4.7,8.1-4.7,18,0,26.1l111.9,193.9c4.7,8.1,13.3,13,22.6,13.1l223.9,0.1
          c9.3,0,17.9-4.8,22.6-12.8l112-193.7l0,0C551.1,305.1,551.2,295.4,546.5,287.3z M516.6,287.4l-41.5,0.1l-26.7-118.2L516.6,287.4z
           M337.2,106.2l-114.5,36.3l-20.3-36.4L337.2,106.2z M180.7,119l20.1,36.2L112,237.7L180.7,119z M181,481.8l-69.3-120l89.2,84
          L181,481.8z M203.3,493.8l19.6-35.6l114.6,35.7L203.3,493.8z M408.1,489.3l-152.7-47.5l-32.3-10.6l-25.3-23.2L82.6,299.6
          l114.6-106.9l25-23.1v-0.2l0.7-0.2l32.4-10.7l153.6-48.7l35.5,156.6l7.5,33.4l-7.6,33.5L408.1,489.3z M447.1,432.9l27.9-120.3
          l41.7-0.1L447.1,432.9z"></path>
      </mask>
      <g id="clqkad-gradient" filter="url(#clqkad-blur)">
        <g>
          <path fill="#006EB6" d="M0,0l300,300v-424.4C182.9-124.4,76.8-76.9,0,0z"></path>
          <path fill="#28348A" d="M300-124.4V300L600,0C523.2-76.9,417.1-124.4,300-124.4z"></path>
          <path fill="#E20613" d="M600,600c76.8-76.8,124.3-182.9,124.3-300H300L600,600z"></path>
          <path fill="#DA0078" d="M300,300h424.4c0-117.1-47.5-223.2-124.3-300L300,300z"></path>
          <path fill="#5BC4F1" d="M-124.4,300H300L0,0C-76.9,76.8-124.4,182.9-124.4,300z"></path>
          <path fill="#A1C517" d="M0,600l300-300h-424.4C-124.4,417.1-76.9,523.2,0,600z"></path>
          <path fill="#F08800" d="M300,724.4c117.1,0,223.2-47.5,300-124.3L300,300V724.4z"></path>
          <path fill="#FFCB00" d="M300,724.4V300L0,600C76.8,676.9,182.9,724.4,300,724.4z"></path>
        </g>
      </g>
    </defs>
  </svg>

  <div class="__db_background"></div>

  <div class="__db_shape">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 600 600" xlink="http://www.w3.org/1999/xlink">
      <use mask="url(#clqkad-shape-mask)" xlink:href="#clqkad-gradient"></use>
    </svg>
  </div>

  <div class="__db_score __db_score_normal">0</div>
  <div class="__db_label">CITATIONS</div>

  <img src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/badge.png" class="__dimensions_png" alt="0 total citations on Dimensions.">
</div>
</div><div class="__dimensions_Badge_Legend_padding __dimensions_Badge_Legend_hover-right  __dimensions_Badge_Legend_style_medium_circle __dimensions_Badge_Legend_always"><div class="__dimensions_Badge_Legend">
      <div class="__dimensions_Badge_stat_group __dimensions_Badge_stat_group_citations">
        <div class="__dimensions_Badge_stat __dimensions_Badge_stat_total_citations">
          <span class="__dimensions_Badge_stat_icon"></span>
          <span class="__dimensions_Badge_stat_count">0</span>
          <span class="__dimensions_Badge_stat_text">Total citations</span>
        </div>
        <div class="__dimensions_Badge_stat __dimensions_Badge_stat_recent_citations">
          <span class="__dimensions_Badge_stat_icon"></span>
          <span class="__dimensions_Badge_stat_count">0</span>
          <span class="__dimensions_Badge_stat_text">Recent citations</span>
        </div>
      </div>
      <div class="__dimensions_Badge_stat_group __dimensions_Badge_stat_group_cr">
        <div class="__dimensions_Badge_stat __dimensions_Badge_stat_fcr">
          <span class="__dimensions_Badge_stat_icon"></span>
          <span class="__dimensions_Badge_stat_count">n/a</span>
          <span class="__dimensions_Badge_stat_text">Field Citation Ratio</span>
        </div>
        <div class="__dimensions_Badge_stat __dimensions_Badge_stat_rcr">
          <span class="__dimensions_Badge_stat_icon"></span>
          <span class="__dimensions_Badge_stat_count">n/a</span>
          <span class="__dimensions_Badge_stat_text">Relative Citation Ratio</span>
        </div>
      </div></div></div></a></span>
<script async="" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/badge.js" charset="utf-8"></script>
 
    </div>

                    <span class="artmet-dimensions-text">Powered by Dimensions</span>
                </div>
                    </div>


        <div class="artmet-item artmet-altmetric artmet-position-right js-show-if-unknown">
            <div class="widget-title-2 artmet-widget-title-2">Altmetrics</div>
            <div class="artmet-badges-wrap js-artmet-badges">

    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkDetails">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/bioinformatics/btaf449" data-badge-details="right" data-uuid="1a987abb-4a16-6d55-6afd-ceedb7d3404d"><a style="cursor: pointer; display: inline-block; background-image: url(&quot;https://badges.altmetric.com/?size=64&amp;score=?&amp;types=????????&amp;style=donut&quot;); background-repeat: no-repeat; background-size: contain; width: 64px; height: 64px;" href="https://www.altmetric.com/details.php?domain=academic.oup.com&amp;doi=10.1093%2Fbioinformatics%2Fbtaf449" target="" data-google-interstitial="false"> </a></div>
         <script type="text/javascript" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/embed.js"></script>
 
    </div>
            </div>
        </div>

</div>

    </div>
                    <a href="javascript:;" role="button" class="artmet-close-modal js-artmet-close-modal" aria-label="Close metrics modal" data-google-interstitial="false">
                        <span aria-hidden="true">×</span>
                        <span class="screenreader-text">Close Modal</span>
                    </a>
                </div>
            </div>
    </div>

 
    </div>
    <div class="widget widget-OUPEmailAlerts widget-instance-Article_RightRailB0Article_RightRail_OUPEmailAlerts">
        
    <div class="alertsWidget js-alerts-widget">
        <h3>Email alerts</h3>
            <div class="userAlert alertType-LatestIssue">
                        <a href="https://academic.oup.com/bioinformatics/new-journals-alert" class="js-my-email-alert email-alert-link" data-google-interstitial="false">
                            New journal issues
                        </a>
            </div>
            <div class="userAlert alertType-NewJournalArticles">
                        <a href="https://academic.oup.com/bioinformatics/new-journal-articles-alert" class="js-my-email-alert email-alert-link" data-google-interstitial="false">
                            New journal articles
                        </a>
            </div>
            <div class="userAlert alertType-ContentActivity">
                        <button type="button" class="js-activity-alert-button email-alert-link" data-user-logged-in="False" data-alert-type="ContentActivity" data-is-content-page="True" data-alert-signup-link="" data-resourceid="8225722" data-resourcetypeid="Article" data-reveal-id="activityAlertModal" aria-expanded="false">
                            Activity related to this article
                        </button>
            </div>

        <div class="userAlert alertType-MarketingLink">
            <a href="https://academic.oup.com/my-account/communication-preferences" data-google-interstitial="false">Sign up for marketing</a>
        </div>
    </div>

    <div id="journalIssuesAlertModal" class="alert-resources-modal fix-width reveal-modal js-content-alert-modal" role="dialog" aria-modal="true" aria-labelledby="journalsIssuesAlertModalTitle" data-reveal="">
    <div class="widget widget-JournalsIssuesAlert widget-instance-JournalIssuesAlert">
        <div class="alerts-sign-up-wrapper js-journals-alerts-sign-up-wrapper page-column">
    <div class="modal-head border-bottom">
        <h2 id="journalsIssuesAlertModalTitle">New journal issues alert</h2>
    </div>

        <div class="modal-body">
            <div class="alert-loginbox-wrapper">
    <div class="widget widget-LoginBoxOnlyUser widget-instance-OUP_IssuesAlert">
            <div class="error-message"><p>To set up an email alert, please <a class="js-sign-in-link" data-linktype="user-pass" href="javascript:;" data-google-interstitial="false">sign in</a> to your personal account, or <a class="js-register-link" href="https://academic.oup.com/my-account/register?siteId=5139&amp;returnUrl=https://academic.oup.com%2fbioinformatics%2farticle%2f41%2f8%2fbtaf449%2f8225722" data-google-interstitial="false">register</a></p></div>

<div class="login-box" data-google-interstitial="false">
    <div class="login-box__main-section-wrap">
        <div class="login-box__section-wrap login-box_heading">
            <div class="login-box__main-heading-wrap">
               <h2>Sign in</h2>
               
            </div>
        </div>

        <div class="login-box__section-wrap cols login-box_sign-in-accounts">
    <div class="login-box__section column-border login-box_personal-account">
            <h3 class="login-box__heading">Personal account</h3>

    <ul class="bullets">
        <li>Sign in with email/username &amp; password</li>
        <li>Get email alerts</li>
        <li>Save searches</li>
        <li>Purchase content</li>
        <li>Activate your purchase/trial code</li>
            <li>Add your ORCID iD</li>
    </ul>

            <div class="login-box__button-row">
                    <a rel="nofollow" class="button-call-to-action at-personal-sign-in js-sign-in-link" data-linktype="user-pass" href="javascript:;" data-google-interstitial="false">
                    <span>Sign in</span>
                    <img class="society-link-icon" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/chevron-right-white.svg" alt="Arrow" aria-hidden="true">

                </a>
                                    <a rel="nofollow" class="at-personal-register-link js-register-link" href="https://academic.oup.com/my-account/register?siteId=5139&amp;returnUrl=http%3a%2f%2facademic.oup.com%2fbioinformatics%2farticle%2f41%2f8%2fbtaf449%2f8225722" data-google-interstitial="false">Register</a>
            </div>
    </div>

        </div>
    </div>
</div>
 
    </div>

            </div>
        </div>
</div> 
    </div>


        <button type="button" class="close-reveal-modal js-close-reveal-modal"><i class="icon-general-close" alt="Close modal"></i></button>
    </div>

    <div id="activityAlertModal" role="dialog" aria-modal="true" class="alert-resources-modal fix-width reveal-modal js-content-alert-modal js-activity-modal" data-reveal="">
        <div class="js-activity-alert-widget">
    <div class="widget widget-ActivityAlert widget-instance-Article_RightRailB0Article_RightRail_OUPEmailAlerts">
        <div class="alerts-sign-up-wrapper js-activity-alert-wrapper">
    <div class="modal-head border-bottom">
        <h2 aria-labelledby="activityAlertModal">Journal article activity alert</h2>
    </div>

        <div class="modal-body alert-loginbox-modal-wrapper">
            <div class="alert-loginbox-wrapper">
    <div class="widget widget-LoginBoxOnlyUser widget-instance-OUP_IssuesAlert">
            <div class="error-message"><p>To set up an email alert, please <a class="js-sign-in-link" data-linktype="user-pass" href="javascript:;" data-google-interstitial="false">sign in</a> to your personal account, or <a class="js-register-link" href="https://academic.oup.com/my-account/register?siteId=5139&amp;returnUrl=https://academic.oup.com%2fbioinformatics%2farticle%2f41%2f8%2fbtaf449%2f8225722" data-google-interstitial="false">register</a></p></div>

<div class="login-box" data-google-interstitial="false">
    <div class="login-box__main-section-wrap">
        <div class="login-box__section-wrap login-box_heading">
            <div class="login-box__main-heading-wrap">
               <h2>Sign in</h2>
               
            </div>
        </div>

        <div class="login-box__section-wrap cols login-box_sign-in-accounts">
    <div class="login-box__section column-border login-box_personal-account">
            <h3 class="login-box__heading">Personal account</h3>

    <ul class="bullets">
        <li>Sign in with email/username &amp; password</li>
        <li>Get email alerts</li>
        <li>Save searches</li>
        <li>Purchase content</li>
        <li>Activate your purchase/trial code</li>
            <li>Add your ORCID iD</li>
    </ul>

            <div class="login-box__button-row">
                    <a rel="nofollow" class="button-call-to-action at-personal-sign-in js-sign-in-link" data-linktype="user-pass" href="javascript:;" data-google-interstitial="false">
                    <span>Sign in</span>
                    <img class="society-link-icon" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/chevron-right-white.svg" alt="Arrow" aria-hidden="true">

                </a>
                                    <a rel="nofollow" class="at-personal-register-link js-register-link" href="https://academic.oup.com/my-account/register?siteId=5139&amp;returnUrl=http%3a%2f%2facademic.oup.com%2fbioinformatics%2farticle%2f41%2f8%2fbtaf449%2f8225722" data-google-interstitial="false">Register</a>
            </div>
    </div>

        </div>
    </div>
</div>
 
    </div>

            </div>
        </div>
</div>
 
    </div>

        </div>
        <button type="button" class="close-reveal-modal js-close-reveal-modal"><i class="icon-general-close" alt="Close modal"></i><span class="screenreader-text">Close Modal</span></button>
    </div>
 
    </div>
    <div class="widget widget-TrendMD widget-instance-Article_RightRailB0trendmd">
        
<div id="trendmd-suggestions"></div>

<div class="options" data-suppress-if-hum-is-present="True"></div> 
    </div>
    <div class="widget widget-Hum widget-instance-Article_RightRailB0hum hide">
        <div id="hum" data-hum-retry="10" data-hum-wait="500"></div> 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fbioinformatics%2Farticle%2F41%2F8%2Fbtaf449%2F8225722" target="" rel="nofollow" data-google-interstitial="false">Google Scholar</a>
            </div>
            </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <h3>
                    <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent" data-google-interstitial="false">Latest</a>
                </h3>
            </li>
            <li>
                <h3>
                    <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead" data-google-interstitial="false">Most Read</a>
                </h3>
            </li>
            <li>
                <h3>
                    <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited" data-google-interstitial="false">Most Cited</a>
                </h3>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    



<div class="widget-layout widget-layout--vert ">
            <div class="widget-columns widget-col-1">
                    <div class="col">

<div class="widget-dynamic-entry journalArticleItem at-widget-entry-SCL">
    <span class="hfDoi" data-attribute="10.1093/bioinformatics/btaf627" aria-hidden="true"></span>


    


<a class="journal-link" href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaf627/8322747?searchresult=1">
        <div class="widget-dynamic-journal-title">
Target and Biomarker Exploration Portal for Drug Discovery    </div>
</a>








<div class="widget-dynamic-journal-image-synopsis">
        <div class="widget-dynamic-journal-synopsis">
            
        </div>
</div>

</div>
<div class="widget-dynamic-entry journalArticleItem at-widget-entry-SCL">
    <span class="hfDoi" data-attribute="10.1093/bioinformatics/btaf611" aria-hidden="true"></span>


    


<a class="journal-link" href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaf611/8322746?searchresult=1">
        <div class="widget-dynamic-journal-title">
Unbiased and Error-Detecting Combinatorial Pooling Experiments with 
Balanced Constant-Weight Gray Codes for Consecutive Positives Detection 
   </div>
</a>








<div class="widget-dynamic-journal-image-synopsis">
        <div class="widget-dynamic-journal-synopsis">
            
        </div>
</div>

</div>
<div class="widget-dynamic-entry journalArticleItem at-widget-entry-SCL">
    <span class="hfDoi" data-attribute="10.1093/bioinformatics/btaf591" aria-hidden="true"></span>


    


<a class="journal-link" href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaf591/8322745?searchresult=1">
        <div class="widget-dynamic-journal-title">
Integrative Metabolic-Flux Platform for Analysis, Contextualization, and
 Targeting (IMPACT): Bridging Untargeted and Targeted Flux Analysis    </div>
</a>








<div class="widget-dynamic-journal-image-synopsis">
        <div class="widget-dynamic-journal-synopsis">
            
        </div>
</div>

</div>
<div class="widget-dynamic-entry journalArticleItem at-widget-entry-SCL">
    <span class="hfDoi" data-attribute="10.1093/bioinformatics/btaf626" aria-hidden="true"></span>


    


<a class="journal-link" href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaf626/8322724?searchresult=1">
        <div class="widget-dynamic-journal-title">
Synteny plot quality control with SyntenyQC    </div>
</a>








<div class="widget-dynamic-journal-image-synopsis">
        <div class="widget-dynamic-journal-synopsis">
            
        </div>
</div>

</div>
<div class="widget-dynamic-entry journalArticleItem at-widget-entry-SCL">
    <span class="hfDoi" data-attribute="10.1093/bioinformatics/btaf621" aria-hidden="true"></span>


    


<a class="journal-link" href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaf621/8321923?searchresult=1">
        <div class="widget-dynamic-journal-title">
PLAID: ultrafast single-sample gene set enrichment scoring    </div>
</a>








<div class="widget-dynamic-journal-image-synopsis">
        <div class="widget-dynamic-journal-synopsis">
            
        </div>
</div>

</div>
                    </div>
            </div>

</div></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>
 
    </div>
    <div class="widget widget-RelatedTaxonomies widget-instance-Article_RightRailB0Article_RightRail_RelatedTaxonomies">
            <div class="widget-related-taxonomies-wrap vt-related-taxonomies">
        <div class="widget-related-taxonomies_title">More from Oxford Academic</div>
            <div class="widget-related-taxonomies">
                <a id="more-from-oa-AcademicSubjects_SCI01060" class="related-taxonomies-link" href="https://academic.oup.com/search-results?tax=AcademicSubjects/SCI01060" data-google-interstitial="false">Bioinformatics and Computational Biology</a>
            </div>
            <div class="widget-related-taxonomies">
                <a id="more-from-oa-AcademicSubjects_SCI00960" class="related-taxonomies-link" href="https://academic.oup.com/search-results?tax=AcademicSubjects/SCI00960" data-google-interstitial="false">Biological Sciences</a>
            </div>
            <div class="widget-related-taxonomies">
                <a id="more-from-oa-AcademicSubjects_SCI00010" class="related-taxonomies-link" href="https://academic.oup.com/search-results?tax=AcademicSubjects/SCI00010" data-google-interstitial="false">Science and Mathematics</a>
            </div>

            <div class="widget-related-taxonomies">
                <a id="more-from-oa-format-Books" class="related-taxonomies-link" href="https://academic.oup.com/books" data-google-interstitial="false">Books</a>
            </div>
            <div class="widget-related-taxonomies">
                <a id="more-from-oa-format-Journals" class="related-taxonomies-link" href="https://academic.oup.com/journals" data-google-interstitial="false">Journals</a>
            </div>
    </div>
 
    </div>
    <div class="widget widget-JobTarget widget-instance-Article_RightRailB0Article_RightRail_JobTarget">
        <div class="wrapper">
    <div class="header">
        <h3>
            <a href="http://medicine-and-health-careernetwork.oxfordjournals.org/jobseeker/search/results/" data-google-interstitial="false">Looking for your next opportunity?</a>
        </h3>
    </div>

        <div class="network-or-parsing-error">
            Having trouble contacting the network. Please try again in a moment or two.
        </div>

    <div class="sponsor-and-jobnetwork-links">
        <div class="job-sponsor"><a href="http://global.oup.com/" data-google-interstitial="false"><img src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/oup-logo.svg" alt="Oxford University Press"></a></div>
        <div class="job-network"><a href="http://globaljobs-careernetwork.oxfordjournals.org/home/index.cfm?site_id=20101" data-google-interstitial="false"><img src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/journals-careers-network.svg" alt="Journals Career Network"></a></div>
    </div>
</div>
 
    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div class="js-adBlock-parent-wrap adblock-parent-wrap">
    <div class="adBlockMainBodyBottom-wrap js-adBlockMainBodyBottom hide">
        <div id="adBlockMainBodyBottom" class="js-adblock at-adblock" data-lazy-load-margin="150">
            <script>
                googletag.cmd.push(function () {
                    googletag.display('adBlockMainBodyBottom');
                });
            </script>
        </div>
            <div class="advertisement-text at-adblock js-adblock-advertisement-text hide">Advertisement</div>
            </div>
</div> 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" type="hidden" value="Leveraging large language models for literature-driven prioritization of protein binding pockets | Bioinformatics | Oxford Academic">
<input id="hfLeftNavStickyOffset" name="hfLeftNavStickyOffset" type="hidden" value="29">


                </div><!-- /.center-inner-row no-overflow -->
            </section>
    </div>

        <div class="mobile-mask">
        </div>

        <section class="footer_wrap vt-site-footer">
            


    <div class="ad-banner-footer sticky-footer-ad js-sticky-footer-ad">
    <div class="widget widget-AdBlock widget-instance-StickyFooterAd">
        <div class="js-adBlock-parent-wrap adblock-parent-wrap">
    <div class="adBlockStickyFooter-wrap js-adBlockStickyFooter hide">
        <div id="adBlockStickyFooter" class="js-adblock at-adblock" data-lazy-load-margin="150">
            <script>
                googletag.cmd.push(function () {
                    googletag.display('adBlockStickyFooter');
                });
            </script>
        </div>
            <div class="advertisement-text at-adblock js-adblock-advertisement-text hide">Advertisement</div>
                    <button type="button" class="close-footer-ad js-close-footer-ad">
                <span class="screenreader-text">close advertisement</span>
            </button>
    </div>
</div> 
    </div>

    </div>

    <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
            <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div class="js-adBlock-parent-wrap adblock-parent-wrap">
    <div class="adBlockFooter-wrap js-adBlockFooter hide">
        <div id="adBlockFooter" class="js-adblock at-adblock js-adblock-lazy-loading" data-lazy-load-margin="150">
            <script>
                googletag.cmd.push(function () {
                    googletag.display('adBlockFooter');
                });
            </script>
        </div>
            <div class="advertisement-text at-adblock js-adblock-advertisement-text hide">Advertisement</div>
            </div>
</div> 
    </div>

    </div>

<div class="journal-footer journal-bg">
    <div class="center-inner-row">

<div class="journal-footer-menu">

    <ul>
<li class="link-1">
    <a href="https://academic.oup.com/bioinformatics/pages/About">About Bioinformatics</a>
</li> <li class="link-2">
    <a href="https://academic.oup.com/bioinformatics/pages/Editorial_Board">Editorial Board</a>
</li> <li class="link-3">
    <a href="https://academic.oup.com/bioinformatics/pages/General_Instructions">Author Guidelines</a>
</li> <li class="link-4">
    <a href="https://www.facebook.com/OUPAcademic" data-google-interstitial="false">Facebook</a>
</li> <li class="link-5">
    <a href="https://twitter.com/oxfordjournals" data-google-interstitial="false">X (formerly Twitter)</a>
</li> </ul><ul><li class="link-1">
    <a href="https://academic.oup.com/bioinformatics/subscribe" data-google-interstitial="false">Purchase</a>
</li> <li class="link-2">
    <a href="http://www.oxfordjournals.org/en/library-recommendation-form.html" data-google-interstitial="false">Recommend to your Library</a>
</li> <li class="link-3">
    <a href="https://academic.oup.com/advertising-and-corporate-services/pages/bioinformatics-media-kit" data-google-interstitial="false">Advertising and Corporate Services</a>
</li> <li class="link-4">
    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?t730=&amp;t732=470053&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=" data-google-interstitial="false">Journals Career Network</a>
</li> 
    </ul>


</div><!-- /.journal-footer-menu -->
        <div class="journal-footer-affiliations">
            <!-- <h3>Affiliations</h3> -->
<a href="https://academic.oup.com/bioinformatics" target="">
    <img id="footer-logo-Bioinformatics" class="journal-footer-affiliations-logo" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/bioinformatics_f1-205396838.svg" alt="Bioinformatics">
</a>                        </div><!-- /.journal-footer-affiliations -->

        <div class="journal-footer-colophon">
            <ul>
<li>Online ISSN 1367-4811</li>                                <li>Copyright © 2025 Oxford University Press</li>
            </ul>
        </div><!-- /.journal-footer-colophon -->


    </div><!-- /.center-inner-row -->
</div><!-- /.journal-footer --> 
    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        



    <input type="hidden" class="SelfServeContentId" value="GlobalFooter_Links">
    <input type="hidden" class="SelfServeVersionId" value="0">

<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li><a href="https://academic.oup.com/pages/about-oxford-academic" data-google-interstitial="false">About Oxford Academic</a></li>
    <li><a href="https://academic.oup.com/pages/about-oxford-academic/publish-journals-with-us" data-google-interstitial="false">Publish journals with us</a></li>
    <li><a href="https://academic.oup.com/pages/about-oxford-academic/our-university-press-partners" data-google-interstitial="false">University press partners</a></li>
    <li><a href="https://academic.oup.com/pages/what-we-publish" data-google-interstitial="false">What we publish</a></li>
    <li><a href="https://academic.oup.com/pages/new-features" data-google-interstitial="false">New features</a>&nbsp;</li>
</ul>
</div>
<div class="global-footer selfservelinks">
<ul>
    <li><a href="https://academic.oup.com/pages/for-authors" data-google-interstitial="false">For authors</a></li>
    <li><a href="https://academic.oup.com/pages/open-research/open-access" data-google-interstitial="false">Open access</a></li>
    <li><a href="https://academic.oup.com/pages/purchasing" data-google-interstitial="false">Purchasing</a></li>
    <li><a href="https://academic.oup.com/pages/institutional-account-management" data-google-interstitial="false">Institutional account management</a></li>
    <li><a href="https://academic.oup.com/pages/purchasing/rights-and-permissions" data-google-interstitial="false">Rights and permissions</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<ul>
    <li><a href="https://academic.oup.com/pages/get-help-with-access" data-google-interstitial="false">Get help with access</a></li>
    <li><a href="https://academic.oup.com/pages/about-oxford-academic/accessibility" data-google-interstitial="false">Accessibility</a></li>
    <li><a href="https://academic.oup.com/pages/contact-us" data-google-interstitial="false">Contact us</a></li>
    <li><a href="https://academic.oup.com/pages/advertising" data-google-interstitial="false">Advertising</a></li>
    <li><a href="https://academic.oup.com/pages/media-enquiries" data-google-interstitial="false">Media enquiries</a></li>
</ul>
</div>
<div class="global-footer selfservelinks global-footer-external">
<ul>
    <li><a href="https://global.oup.com/" data-google-interstitial="false">Oxford University Press</a></li>
    <li><a href="https://corp.oup.com/press-office" data-google-interstitial="false">News</a></li>
    <li><a href="https://languages.oup.com/" data-google-interstitial="false">Oxford Languages</a></li>
    <li><a href="https://www.ox.ac.uk/" data-google-interstitial="false">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. 
It furthers the University's objective of excellence in research, 
scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/oup-logo_002.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright © 2025 Oxford University Press</li>
    <li><a href="#openCookiePreferences" class="optanon-show-settings">Cookie settings</a></li>
    <li><a href="https://global.oup.com/cookiepolicy" data-google-interstitial="false">Cookie policy</a></li>
    <li><a href="https://global.oup.com/privacy" data-google-interstitial="false">Privacy policy</a></li>
    <li><a href="https://academic.oup.com/pages/legal-notice" data-google-interstitial="false">Legal notice</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
    .sf-facet-list .sf-facet label,
    .sf-facet-list .taxonomy-label-wrap label {
    font-size: 0.9375rem;
    }
    .issue-pagination-wrap .pagination-container {
    float: right;
    }
</style>



 
    </div>

        </div>
    </div>
    <div class="ss-ui-only">

    </div>

        </section>
</div>






    



    <div class="widget widget-SiteWideModals widget-instance-SiteWideModals">
        <div id="revealModal" class="reveal-modal" data-reveal="">
    <div id="revealContent"></div>
    <a class="close-reveal-modal" href="javascript:;" data-google-interstitial="false"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>

<div id="globalModalContainer" class="modal-global-container">
    <div id="globalModalContent">
        <div class="js-globalModalPlaceholder"></div>
    </div>
    <a class="close-modal js-close-modal" href="javascript:;" data-google-interstitial="false"><i class="icon-general-close"><span class="screenreader-text">Close</span></i></a>
</div>
<div id="globalModalOverlay" class="modal-overlay js-modal-overlay"></div>

<div id="NeedSubscription" class="reveal-modal small" data-reveal="">
    <div class="subscription-needed">
        <h5>This Feature Is Available To Subscribers Only</h5>
        <p><a href="https://academic.oup.com/sign-in" data-google-interstitial="false">Sign In</a> or <a href="https://academic.oup.com/my-account/register?siteId=5139&amp;returnUrl=%2fbioinformatics%2farticle%2f41%2f8%2fbtaf449%2f8225722" data-google-interstitial="false">Create an Account</a></p>
    </div>
    <a class="close-reveal-modal" href="javascript:;" data-google-interstitial="false"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>

<div id="noAccessReveal" class="reveal-modal tiny" data-reveal="">
    <p>This PDF is available to Subscribers Only</p>
    <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal="" data-google-interstitial="false"><span>View Article Abstract &amp; Purchase Options</span></a>
    <div class="issue-purchase-modal">
        <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
    </div>
    <a class="close-reveal-modal" href="javascript:;" data-google-interstitial="false"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
 
    </div>

    



<script type="text/javascript">
    MathJax = {
        tex: {
            inlineMath: [['|$', '$|'], ['\\(', '\\)']]
        }
    };
</script>
<script id="MathJax-script" async="" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/tex-mml-chtml.js"></script>


    <!-- CookiePro Default Categories -->
    <!-- When the Cookie Compliance code loads, if cookies for the associated group have consent...
         it will dynamically change the tag to: script type=text/JavaScript...
         the code inside the tags will then be recognized and run as normal. -->














    <script>
        var NTPT_PGEXTRA = 'event_type=full-text\u0026discipline_ot_level_1=Science and Mathematics\u0026discipline_ot_level_2=Biological Sciences\u0026supplier_tag=SC_Journals\u0026object_type=Article\u0026taxonomy=taxId%3a39%7ctaxLabel%3aAcademicSubjects%7cnodeId%3aSCI01060%7cnodeLabel%3aBioinformatics+and+Computational+Biology%7cnodeLevel%3a3\u0026siteid=bioinformatics\u0026authzrequired=false\u0026doi=10.1093/bioinformatics/btaf449';
    </script>
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script type="text/javascript" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/ntpagetag.js"></script>
        <noscript>
            <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ" />
        </noscript>










    <script type="text/javascript" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/site.min.js"></script>


    
    
    
    <script type="text/javascript" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/chartist.min.js"></script>

    <script type="text/javascript" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/article-page.min.js"></script>



    

    
    




    <div class="ad-banner js-ad-riser ad-banner-riser">
    <div class="widget widget-AdBlock widget-instance-RiserAd">
        
 
    </div>

    </div>



                <script src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/scm.proxylogging.js"></script>

    <div class="end-of-page-js"></div>



<script type="text/javascript" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/btaf449"></script><script type="text/javascript" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/btaf449"></script><script type="text/javascript" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/btaf449"></script><script type="text/javascript" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/btaf449"></script><div data-reveal="math-jax-load" id="math-jax-modal" class="reveal-modal" style="display: none;"></div><iframe style="display: none; position: absolute; top: -999px; left: -999px;" id="ps_kejiz" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/a_002.htm"></iframe><iframe style="display: none; position: absolute; top: -999px; left: -999px;" id="ps_sriv" src="Leveraging%20large%20language%20models%20for%20literature-driven%20prioritization%20of%20protein%20binding%20pockets%20_%20Bioinformatics%20_%20Oxford%20Academic_files/a_002.htm"></iframe></body></html>